\section{Discussion}

Disease relapse remains a major barrier in treating \gls{ball} \cite{formanMythSecondRemission2013,liewOutcomesAdultPatients2012,hungerAcuteLymphoblasticLeukemia2015}.
While the genetic origins of relapse have been characterized, epigenetic aberrations underlying relapse have been less well-studied.
In this work, we investigated the epigenetic and transcriptomic changes of 5 \gls{ball} patients over the course of relapse to identify non-genetic changes in tumours that may lead to relapse.
\gls{dname} is more highly correlated with disease stage than RNA or chromatin accessibility and changes to \gls{dname} are antithetical to \gls{dname} changes seen in normal B-cell differentiation.
While most \gls{dname} changes are patient-specific, a small number of recurrent changes indicate a more stem-like state at relapse.
In some cases, these stem-like \gls{dname} profiles are present at diagnosis, indicating that subclones defined by \gls{dname} may also contribute to \gls{ball} relapse.

Both genetic and epigenetic aberrations in tumours play important roles in determining disease relapse \cite{nordlundGenomewideSignaturesDifferential2013,leeEpigeneticRemodelingBcell2015}.
Previous reports highlight the frequency that epigenetic regulators are mutated in \gls{ball} \cite{maRiseFallSubclones2015} and leukemias more generally, such as \emph{DNMT3A}, \emph{TET2}, \emph{IDH1}, and \emph{IDH2} in \gls{aml} \cite{kishtagariDriverMutationsAcute2020,papaemmanuilGenomicClassificationPrognosis2016,leyDNMT3AMutationsAcute2010} and \emph{CHD2}, \emph{HIST1H1E}, and \emph{ZMYM3} in \gls{cll} \cite{billotDeregulationAiolosExpression2011,landauChronicLymphocyticLeukemia2013,landauEvolutionImpactSubclonal2013}.
These findings demonstrate that epigenetic modifications, in conjunction with genetic aberrations, discriminate disease outcomes and can share and evolutionary trajectory in cancer.
However, it is not the case that genetic and epigenetic states always behave similarly.
In this study we found both \gls{dx} and \gls{dri} \glspl{pdx} samples that share \gls{dname} profiles with the \gls{rel} tumours, suggesting that \gls{dname} states can vary independently of mutations.
Moreover, the differences between \gls{pdx} methylomes derived from the same primary sample demonstrates that subpopulations of cells can have differing \gls{dname} states while sharing mutations.
This decoupling between genome and epigenome has been observed in other tumours, such as pediatric ependymomas, where recurrent \gls{dname} profiles were found in the absence of recurrent mutations and was associated with outcome \cite{mackEpigenomicAlterationsDefine2014,pajtlerMolecularClassificationEpendymal2015}, and glioblastoma, where stem cells are characterized by widespread changes in chromatin accessibility \cite{guilhamonSinglecellChromatinAccessibility2021} and histone modifications \cite{liauAdaptiveChromatinRemodeling2017}.
These studies highlight the role of epigenetic plasticity and intra-tumour heterogeneity in cancers \cite{flavahanEpigeneticPlasticityHallmarks2017}.
With similar results found in chronic leukemias that are linked to disease outcome \cite{pastoreCorruptedCoordinationEpigenetic2019,landauLocallyDisorderedMethylation2014,gaitiEpigeneticEvolutionLineage2019,namIntegratingGeneticNongenetic2021}, it is likely that epigenetic plasticity and heterogeneity are also key factors in therapeutic response and relapse.
Taken together, these results suggest that the epigenome can provide mechanisms independent of genetic aberrations, to respond and adapt to therapies.
This complexity of disease response will need to be addressed to design treatment regimens for patients with an increased propensity towards relapse.

\begin{enumerate}
  \item future directions for investigation
  \begin{enumerate}
    \item identifying important molecular changes over relapse may be easier by starting with patient-oriented analyses
    \item Most \glspl{dmr} were identified in intergenic regions, suggesting that targeted or gene-centric epigenomic assays are unlikely to discover epigenetic changes, and that genome-wide approaches should be prioritized
    \item effect of targeted \gls{dname} of promoter regions for important stem genes on engraftment
    \item effect of demethylating agents on relapsed \gls{ball} patients
    \item combination therapy of demethylating agents with chemotherapy to reduce the potential outgrowth of relapse-fated subclones
  \end{enumerate}
\end{enumerate}