\section{Methods}

\subsection{Patient selection and sample collection}

Cord blood pooling and isolation

B-ALL sample collection and cell sorting

\subsection{\Glsentrylong{pdx} generation and \glsentrylong{lda}}

\subsection{Human cell isolation from \glsentrylongpl{pdx}}

\subsection{Primary and \glsentrylong{pdx} sample sequencing}

\subsubsection{\Glsentrylong{rnaseq}}

\subsubsection{\Glsentrylong{mecapseq}}

\subsubsection{\Glsentrylong{atacseq}}

\subsection{Sequencing data analysis}

\subsubsection{Differential gene expression analysis}

The methods are described in \cite[REF][]{dobsonRelapseFatedLatentDiagnosis2020}.
Briefly, \gls{rnaseq} reads were aligned against the GRCh38 reference human genome with STAR (v2.5.2b) \cite{dobinSTARUltrafastUniversal2013} and annotated with the Ensembl reference (v90).
Default parameter were used with the following exceptions: chimeric segments were screened with a minimum size of 12 bp, junction overlap of 12 bp, and maximum segment reads gap of 3 bp; splice junction overlap of 10 bp; maximum gap between aligned mates of 100 000 bp; maximum aligned intron of 100 000; and alignSJstitchMismatchNmax of 5 1 5 5.
Transcript counts were obtained with HTSeq (v0.7.2) \cite{andersHTSeqPythonFramework2015}.
Data was library size normalized using the RLE method, followed by a variance stabilizing transformation using DESeq2 (v1.22.1) \cite{loveModeratedEstimationFold2014}.
Principal component analysis plots were generated on a per sample basis using the top 1,000 variable genes.
For downstream analysis, the mean expression of each sample clone condition was used.
For per-patient analyses, differentially expressed genes were identified between disease stage and clone status using DESeq2.
Genes with an \gls{fdr} $< 0.05$ and absolute $\log_2(\text{fold change}) > 1$ were considered significant.

\subsubsection{Identification of accessible chromatin peaks}

\Gls{atacseq} reads were aligned against the GRCh38 reference human genome with Bowtie2 (v2.3.4) \cite{langmeadFastGappedreadAlignment2012} with default parameters.
Accessible peaks were identified with MACS2 (v2.1.2) \cite{zhangModelbasedAnalysisChIPSeq2008} with the following command:

\begin{lstlisting}[basicstyle=\ttfamily]
  macs2 callpeak -f BED -g hs --keep-dup all -B --SPMR --nomodel --shift -75 --extsize 150 -p 0.01 --call-summits -n {sample_name} -t {input_bam}
\end{lstlisting}

A catalogue of peaks from all samples was collected with a custom R script.
\Gls{atacseq} signal was mapped from each sample to this catalogue using Bedtools \cite{quinlanBEDToolsSwissArmyTool2014} for downstream analysis.

\subsubsection{Bisulfite sequencing pre-processing}

Sequencing read qualities were assessed with FastQC (v0.11.8) \cite{simonandrewsFastQCQualityControl2010}.
Low quality bases were trimmed with Trim Galore! (v0.6.3) \cite{felixkruegerTrimGalore2012} with the following command:

\begin{lstlisting}[basicstyle=\ttfamily]
  trim_galore --gzip -q 30 --fastqc_args '-o TrimGalore' {sample_mate1} {sample_mate2}
\end{lstlisting}

Trimmed reads were aligned to the GRCh38 reference human genome with Bismark (v0.22.1) \cite{kruegerDNAMethylomeAnalysis2012} with default parameters.
Duplicates were removed from the resulting alignment file with the following command:

\begin{lstlisting}[basicstyle=\ttfamily]
  deduplicate_bismark -p --bam {input_bam}
\end{lstlisting}

The deduplicated BAM file was sorted by position with sambamba (v0.7.0) \cite{tarasovSambambaFastProcessing2015}.
$M$-biases were calculated with MethylDackel (v0.4.0) \cite{ryanMethylDackel2019}, and methylation $\beta$ values were extracted from the BAM files with the following command:

\begin{lstlisting}[basicstyle=\ttfamily]
  MethylDackel extract --mergeContext --OT 3,124,3,124 --OB 3,124,3,124 {ref_genome} {dedup_sorted_bam}
\end{lstlisting}

Both $M$ and $\beta$ values were for each \gls{cpg} were used in downstream analyses.

\subsubsection{Similarity network fusion}

Preprocessed data from each sample was collected with the following features: normalized gene expression abundance for all genes, chromatin accessibility signal within previously identified accessible peaks, and mean $\beta$ value for all \glspl{cpg} listed in the manifest for targeted bisulfite sequencing kit.
These features and sample labels were processed with the SNFtool R package \cite{wangSimilarityNetworkFusion2014} to perform the similarity network fusion analysis.
Graphs were constructed for all samples deriving from a single patient where each node is a sample and each edge is weighted according to the determined similarity between the samples.
Edges whose weights were below specific thresholds were removed from the graph.
The threshold weight for the fused graph was 0.05.
Similar graphs were constructed using the individual components for each sample (e.g. using just the similarity in \gls{rnaseq} data), and the component graphs were compared to the fused graph, to compare the importance of each feature.
Threshold weights for these individual graphs were determined to be $6 \times 10^{-5}$ for \gls{dname}, $4 \times 10^{-4}$ for gene expression, and $2 \times 10^{-4}$ for chromatin accessibility.

\subsubsection{\Glsentrylong{dmr} identification}

\Glspl{dmr} were identified using the dmrseq R package (v1.3.8) \cite{korthauerDetectionAccurateFalse2018} with an absolute filtering cutoff value of 0.05 and using the sequencing batch as an adjustment covariate.
Normal samples from all donors were compared pairwise based on their sorted cell type.
\Gls{ball} samples were compared by their designated disease stage (Dx, DRI, or Rel), and were compared both across all patients (e.g. all Dx samples against all Rel samples), or within a single patient (e.g. all Dx samples from Patient 1 against all Rel samples from Patient 1).
A multiple testing correction with the \gls{fdr} method was performed \cite{benjaminiControllingFalseDiscovery1995}.
Regions with an \gls{fdr} $< 0.1$ were determined to be significant.

\subsubsection{Gene ontology enrichment analysis}

Gene ontology enrichment analysis was performed using the PANTHER classification system (database version 2019-10-08) \cite{miLargescaleGeneFunction2013}.
Gene symbols for the genes whose promoter regions contained the recurrently hyper-methylated regions in all \gls{ball} patient samples were supplied, with the entire human genome as the background.
An over-representation Fisher test for biological processes was performed with an \gls{fdr} correction.
Biological processes at the top of the hierarchy with an \gls{fdr} $< 0.05$ were determined to be significant.
