\section{Results}

\subsection{Multiomic integration of \glsentryshort{ball} relapse patients links \glsentrylong{dname} to relapse status}

To investigate the molecular landscape of \gls{ball} relapse, we profiled gene expression, chromatin accessibility, and \gls{dname} of 3 adult and 2 pediatric \gls{ball} patients at both \gls{dx} and \gls{rel} with bulk \gls{rnaseq}, \gls{atacseq}, and \gls{mecapseq}, respectively (\Cref{fig:BALL_fig1}a).
These patients' tumours contained $\ge 90$ \% leukemic blasts at diagnosis and were previously profiled using \gls{wes} to identify the mutation burden of leukemic driver mutations \cite{dobsonRelapseFatedLatentDiagnosis2020} (see \Cref{tab:BALL_clinical}; patient numbers used in this study match those from \cite[REF][]{dobsonRelapseFatedLatentDiagnosis2020}).
Matching mutation profiles between \gls{dx} and \gls{rel} samples allowed for the identification of \gls{dri} samples, which are cells present at diagnosis that harbour mutations found at relapse, indicating that these cells are relapse-fated.
Comprehensive datasets containing \gls{rnaseq}, \gls{atacseq}, and \gls{mecapseq} were produced for 3 patients, with 2 patients lacking \gls{rnaseq} data due to source constraints.
While expression, chromatin accessibility, and \gls{dname} are each critical for determining cell phenotype and its role in relapse, we sought to investigate the importance of each dataset in an agnostic manner.
To achieve this, similarity scores were calculated between all samples using \gls{snf} \cite{wangSimilarityNetworkFusion2014}.
For each patient, similarity scores between all samples derived from that patient (both primary and \gls{pdx}) were calculated, and weighted graphs to cluster samples together were constructed (see \Cref{sec:BALL_snf}).
This was done for each individual data type, as well as for a fused network comprised of information by considering all data types simultaneously.
To determine the importance of each data type, samples were labelled by their disease stage (\gls{dx}, \gls{dri}, or \gls{rel}; \Cref{fig:BALL_fig1}b).
For all 3 patients with complete molecular datasets, the combined networks clustered samples based on disease stage more clearly than each individual dataset (\Cref{fig:BALL_fig1}b).
This suggests that disease stages can be more clearly identified from multiple molecular components together than a single component alone \cite{wangSimilarityNetworkFusion2014}.
The graphs produced from \gls{dname} data more clearly cluster samples by disease stage than gene expression or chromatin accessibility across all patients, suggesting that \gls{dname} may be a clearer marker of relapse.
Taken together, we find that \gls{ball} disease stage can be identified through non-genetic molecular measurements and that \gls{dname} is mostly closely linked to relapse than gene expression and chromatin accessibility.

\newfigure{chapter5/Figure1.png}{Experimental design and data integration}{\textbf{a.} Experimental design of samples used in this study. Normal samples were obtained from cord blood pools and sorted into various hematopoietic cell types. B-ALL patients who experienced relapse has sorted leukemic blasts collected at \gls{dx} and \gls{rel}. Based on the mutation profiles from \cite[REF][]{dobsonRelapseFatedLatentDiagnosis2020} some \gls{dx} samples are labelled as \gls{dri}. \textbf{b.} Individual and fused networks of samples from three patients with complete multiomic profiling. Nodes represent individual samples (either primary or \gls{pdx}), edges represent similarities between the connected samples.}{fig:BALL_fig1}

\subsection{Widespread loss of \glsentrylong{dname} over normal B-cell differentiation}

Given the strong correlation between \gls{dname} signal and disease state, we focused on \gls{dname} changes over \gls{ball} relapse.
To understand the dynamic changes to \gls{dname} that happen over the course of \gls{ball} relapse, we first looked to the hematopoietic hierarchy and \gls{dname} changes over normal B-cell differentiation.
Using normal cord blood pools, sorted into B-cells and multiple B-progenitor cell types, we performed \gls{mecapseq} on 8 pools separated into 4 cell types: \glspl{hsc} and \glspl{mpp}; \glspl{lmpp} and \glspl{mlp}; \glspl{eprob}, \glspl{preprob}, and \glspl{prob} (collectively labelled as \gls{prob}); and B-cells (\Cref{fig:BALL_fig2}a; see \Cref{tab:BALL_markers}).
Using pairwise comparisons between these cell types, we identified 540 \glspl{dmr} over the course of B-cell differentiation from \glspl{hsc} (\Cref{fig:BALL_fig2}b).
Significant changes to \gls{dname} occurred in 62 regions from \gls{hsc}-\gls{mpp} to \gls{lmpp}-\gls{mlp}, 312 regions from \gls{lmpp}-\gls{mlp} to \gls{prob}, and 166 regions from \gls{prob} to fully differentiated B-cells (\Cref{fig:BALL_fig2}c).
While roughly equal numbers of loci gained and lost \gls{dname} in the transition from \glspl{hsc}-\glspl{mpp} to \glspl{lmpp}-\glspl{mlp}, after lymphoid commitment, nearly all regions lost \gls{dname} in later differentiation transitions (\Cref{fig:3D_figs2}c).
Overall, 500 (92.6 \%) of \glspl{dmr} identified were loci that became hypomethylated over differentiation.
These changes are in agreement with earlier studies profiling \gls{dname} changes over B-cell differentiation using the Illumina 450K arrays \cite{leeGlobalDNAMethylation2012,leeEpigeneticRemodelingBcell2015,nordlundGenomewideSignaturesDifferential2013}, and provide an expanded set of \glspl{dmr} with which to track differentiation.
Notably, no \gls{dmr} identified in an earlier transition was found as differentially methylated in a later transition.
Regions with altered \gls{dname} in one cell type persisted for all downstream cell type transitions.
This suggests that \gls{dname} at these loci can be used as a marker of differentiation.
In summary, we find that normal \glspl{hsc} permanently change \gls{dname} over the course of differentiation, predominantly by losing \gls{dname}.

\newfigure{chapter5/Figure2.png}{Widespread loss of \Glsentrylong{dname} over B-cell differentiation}{\textbf{a.} Schematic of the hematopoietic hierarchy and the grouping of B-cell progenitors into the groups isolated in this study. \textbf{b.} Heatmap of \glspl{dmr} identified between B lineage cell types. Columns are samples ordered by cell type and rows are \glspl{dmr} identified in at least one pairwise comparison between cell types (dmrseq, \gls{fdr} $< 0.1$). \textbf{c.} Bar plot of \glspl{dmr} classified by which step in differentiation they were identified as significantly changed.}{fig:BALL_fig2}

\subsection{Recurrent \glsentrylong{dname} changes identify stem cell pathways in relapse}

With the predominant loss of \gls{dname} established in the normal setting, we identified \glspl{dmr} between \gls{dx} and \gls{rel} primary and \gls{pdx} \gls{ball} samples.
When considering all patients together and grouping by disease stage, we found no \glspl{dmr} remained statistically significant after multiple testing corrections.
This result conflicted with previous observations about \gls{dname} changes in \gls{ball} relapse \cite{leeEpigeneticRemodelingBcell2015,nordlundGenomewideSignaturesDifferential2013} as well as the earlier \gls{snf} analysis.
We hypothesized that \gls{dname} changes in across patients was heterogeneous, which limited the ability to detect significant changes.
Using a patient-oriented approach, we identified \glspl{dmr} between \gls{dx} and \gls{rel} for each patient, separately, to track changes over each patient's relapse trajectory.
This identified 25 761 \glspl{dmr} across the cohort of (range 98 - 15 296, median 7 426, $\delta \beta \ge 20$ \%, \gls{fdr} $< 0.1$, \Cref{fig:BALL_fig3}a).
Unlike the process of normal differentiation, most \glspl{dmr} were hypermethylated at relapse (\Cref{fig:BALL_fig2}b).
18 610 (72.2 \%) \glspl{dmr} were specific to a single patient and did not overlap \glspl{dmr} from others (\Cref{fig:BALL_fig3}b, left), as expected from the lack of significant \glspl{dmr} from the cohort-oriented analysis.
Notably, the 9 recurrently \glspl{dmr} in all 5 patients are all in the promoter regions of the following genes: \emph{BARHL2}, \emph{CYP26B1}, \emph{EBF3}, \emph{EN2}, \emph{GDNF}, \emph{HMGA2}, \emph{NKX2-2}, \emph{NR2F2}, and \emph{PAX6}.
Using \gls{go} analysis, we find that these genes with nearby recurrent differential methylation are positively associated with differentiation, with the most statistically significant pathway being cell fate determination (\Cref{fig:BALL_fig3}c).
For these genes, we find that the promoter regions become hypermethylated at \gls{ball} relapse (\Cref{fig:BALL_fig3}d).
Some genes, like \emph{CYP26B1}, have multiple short \glspl{dmr} in the promoter and one gains \gls{dname} while the other loses \gls{dname} at relapse, but all these promoters gain \gls{dname} overall.
Given the association between hypermethylation in promoter regions and decreased expression \cite{jonesFunctionsDNAMethylation2012}, these results suggest that these genes are under-expressed at relapse.
Taken together, we find that the changes to \gls{dname} over the course of \gls{ball} relapse is antithetical to the changes seen over normal B-cell differentiation, and that recurrent \gls{dname} changes suggest that \gls{ball} relapse reverts to a more de-differentiated, stem-like \gls{dname} state.

\newfigure{chapter5/Figure3.png}{Recurrent relapse \glspl{dmr} are associated with cell fate decision processes}{\textbf{a.} Heatmaps of \glspl{dmr} identified between \gls{dx} and \gls{rel} samples within each patient. \textbf{b.} Upset plot showing the shared \glspl{dmr} between patients. \Glspl{dmr} in the left highlighted block are unique to a single patient, whereas \glspl{dmr} in the right highlighted block are recurrent changes across all 5 relapse patients. These \glspl{dmr} are in the promoter regions of the callout genes listed. \textbf{c.} \Gls{go} analysis of genes with recurrently hypermethylated promoters in \gls{rel} \gls{ball} samples. The red dashed line indicates the \gls{fdr} threshold of 0.05. \textbf{d.} Pairwise \gls{dname} changes in each patient at the recurrently hypermethylated loci show increased methylation in all patients.}{fig:BALL_fig3}

\subsection{Relapse \glsentrylong{dname} profiles are present at diagnosis in some patients}

Relapse-fated subpopulations of cells present at diagnosis were detected in these patients by their mutations \cite{dobsonRelapseFatedLatentDiagnosis2020}.
Yet some \gls{dx} samples harboured similar \gls{dname} profiles to the \gls{rel} samples (e.g. column 2 for Patient 7 and column 6 for Patient 9 in \Cref{fig:BALL_fig3}a).
We hypothesized whether these same populations could be detected by their \gls{dname} profile at diagnosis.
By identifying \glspl{dmr} across all three disease stages (\gls{dx}, \gls{dri}, and \gls{rel}), we identified a median of 2 784 \glspl{dmr} between disease stages across each patient (range 376 - 4 098; \Cref{fig:BALL_fig4}).
There is heterogeneity in \gls{dname} profiles across samples derived from the same patient, and even within the same disease stage (\Cref{fig:BALL_fig4}a).
The heterogeneity within disease stages resulted identifying \glspl{dmr} specific to the \gls{dri} samples that were shared between \gls{dx} and \gls{rel} samples, even when some \gls{dri} samples showed similar methylation rates (e.g. leftmost \gls{dri} sample for Patient 4).
Based on which disease stage the \glspl{dmr} were identified in, each gls{dmr} was classified as \gls{dx}-specific (shared between \gls{dri} and \gls{rel}), \gls{dri}-specific (shared between \gls{dx} and \gls{rel}), \gls{rel}-specific (shared between \gls{dx} and \gls{dri}), or unique (significantly differentially methylated in all three stages).
All patients harboured \gls{rel}-specific \glspl{dmr}, and a majority of \glspl{dmr} in total were detected in Patients 1, 4, and 6 (range 71.6 \%, 100 \%, and 100 \%, respectively; \Cref{fig:BALL_fig4}b).
For these three patients, a majority of the relapse-fated cells shared the \gls{dname} profile of the neighbouring cells, suggesting that these \gls{dname} changes occurred after mutation.
Patients 1, 7, and 9, \gls{dri}-specific \glspl{dmr} were found, suggesting that the \gls{dname} profile of cells at diagnosis is not necessarily linked to their mutation status.
Further, 41.1 \% and 90.1 \% of \glspl{dmr} were found to be \gls{dx}-specific for Patients 7 and 9, respectively (\Cref{fig:BALL_fig4}b).
Patient 1 also harboured 356 (10.6 \%) \gls{dx}-specific \glspl{dmr}.
This suggests that some \gls{dname} changes are linked to the mutation status of relapse-fated cells.
Taken together, these results suggest that relapse-fated \gls{dname} profiles can be detected at diagnosis, but that the trajectory of \gls{dname} changes over the course of relapse is heterogeneous across patients.

\newfigure{chapter5/Figure4.png}{Subpopulations present at diagnosis can harbour relapse-like \gls{dname} profiles}{\textbf{a.} Heatmaps of expanded \glspl{dmr} identified in \gls{dri} \gls{pdx} samples. \textbf{b.} Bar plot showing the number of \glspl{dmr} classified by which disease stage it is specific to. "\Gls{dx} Specific" \glspl{dmr} have shared \gls{dname} between \gls{dri} and \gls{rel} samples. "\Gls{dri} Specific" \glspl{dmr} have shared \gls{dname} between \gls{dx} and \gls{rel} samples. "\Gls{rel} Specific" \glspl{dmr} have shared \gls{dname} between \gls{dx} and \gls{dri} samples. "Unique" \glspl{dmr} are regions that have significantly different \gls{dname} at each stage.}{fig:BALL_fig4}
