\chapter{Discussion \& Future Directions}
\label{chap:discussion}

Each of the previous chapters have presented a story interrogating multiple components of the chromatin architecture, how they interact with each other, and the plethora of computational and experimental methods required to unravel this architecture.
\Cref{chap:FOXA1} identifies and validates \glspl{cre} of the \emph{FOXA1} gene, a critical \gls{tf} that regulates \gls{pca} development and regulates \gls{ar} expression to control disease progression.
\Cref{chap:3D} expands on these ideas to investigate how the three-dimensional genome organization impacts gene regulatory networks and how genetic aberrations can alter this organization to promote oncogenesis.
\Cref{chap:JS} develops a mathematical and computational framework to reduce uncertainty about how individual aberrations in chromatin architecture impact gene expression.
Finally, \Cref{chap:BALL} identifies the strong relationship between genetic and epigenetic profiles in \gls{ball} relapse and investigates how \gls{dname} changes and revision to a more stem-like chromatin state may underlie disease relapse.
Together, the work presented in this thesis demonstrates that different components of the chromatin architecture, the genome, molecular chromatin modifications, and three-dimensional organization, can all individually contribute to cancer development and progression.
Moreover, this thesis demonstrates that aberrations in these components work together to drive disease.
These multiple components of the chromatin architecture need to be studied in tandem to understand the origins of cancer and how to develop curative treatments for it.

\section{Implications of non-coding \glsentrylongpl{snv} targeting a single gene}

In \Cref{chap:FOXA1}, I used gene essentiality screening data from multiple cell lines to prioritize the \emph{FOXA1} \gls{tf} as a critical factor across \gls{pca} cell lines.
I also made use of the concept that \glspl{snv} converge on \glspl{cre} of important genes in a given tumour type to predict how these mutations may impact candidate \glspl{cre} for the \emph{FOXA1} gene.
\emph{FOXA1} is also an important \gls{tf} in breast cancers \Cref{fig:FOXA1_figs3}.
Similar investigations into the impact of \glspl{snv} in breast tumours may identify the impact of aberrations to the \glspl{cre} of \emph{FOXA1}.
Identifying important genes in this manner is not limited to \emph{FOXA1} and breast and prostate tumours.
Critical genes may be identified in other cancer types using \gls{crispr} screens or \glspl{mpra}.
Similarly \glspl{snv} are not the only chromatin aberrations that can affect \gls{tf} binding or gene regulation.
Other chromatin aberrations may accumulate in \glspl{cre} of important genes in a similar fashion.
Complex \glspl{sv}, changes in \gls{dname}, or histone modifications may only need to accumulate in the set of \glspl{cre} for a given gene, rather than be recurrent in a single element, to affect its expression.
Interpreting chromatin aberrations in cancer in light of this plexus-based approach may aid in identifying driver events for cancer by aggregating previously unrelated events together.
These approaches are not limited to prostate tumours and can serve as a starting point to identify important genes in other cancers, more generally.

\section{Implications of three-dimensional organization and enhancer hijacking in \glsentrylong{pca}}

In \Cref{chap:3D}, my co-authors optimized a low-input Hi-C method to interrogate genome organization in cryo-preserved prostate tissue slides.
I then demonstrated that this could produce a high quality Hi-C library and helped produce that largest collection of genome organization data in prostate tumours to date.
This technological step forward opens the door for profiling the three-dimensional genome in cancer patients without relying in cell lines or other models, and may be a critical step in moving personalized medicine forward.
We add to existing evidence that \glspl{sv} can, but rarely, alter 3D structure in disease \cite{ghavi-helmHighlyRearrangedChromosomes2019,oudelaarRelationshipGenomeStructure2020,despangFunctionalDissectionSox92019,williamsonDevelopmentallyRegulatedShh2019,dixonIntegrativeDetectionAnalysis2018,akdemirDisruptionChromatinFolding2020,liPatternsSomaticStructural2020,iyyankiSubtypeassociatedEpigenomicLandscape2021}.
Elucidating when and how \glspl{sv} impact genome organization, then, is still an area that requires investigation.
Developments in statistical methods, such as those discussed in \Cref{chap:JS}, may help identify the effects of individual, non-recurrent \glspl{sv}.
Subclonality of \glspl{sv} may interfere with the ability to detect rearranged domains in bulk Hi-C measurements.
Thus, developments in high throughput sequencing and microscopy measurements in single cells, such as ORCA \cite{mateoVisualizingDNAFolding2019} and STORM \cite{batesStochasticOpticalReconstruction2013}, as well as organoid or explant models that recapitulate the chromatin state of the original tumour, may help in identifying the effect of such events \cite{}.
This work also adds to our ability to detect chromatin interactions between promoters and enhancers in patient samples, allowing for better characterization of gene regulatory networks for each and every gene.
Given the benefits of plexus-based approaches to interpreting aberrations in the chromatin architecture, this work serves as a foundation on which to integrate gene regulatory networks with chromatin aberrations in cancers more generally.
This foundation can be extended to studying the evolution of these networks, their genome organization, and their resiliency between species or over time as tumours respond to therapeutic interventions \cite{}.

\section{Implications of \glsentrylong{dname} changes in relapse}

\Cref{chap:BALL} identifies \gls{dname} as an epigenetic marker that can mirror that mutational profile of \gls{ball} cells.
The \gls{dname} changes observed over the course of \gls{ball} relapse have the potential to become a biomarker predicting relapse, although variation in \gls{dname} changes across patients necessitates larger sample sizes before recurrent events may be robustly identified.
Our patient-oriented approach to identify recurrent changes to \gls{dname} boosted our discovery of recurrent \glspl{dmr} over a cohort-oriented approach, and thus may be a beneficial strategy for similar studies.
Inter-tumour heterogeneity in tumours is a well-studied phenomenon \cite{marusykTumorHeterogeneityCauses2010,sottorivaCatchMyDrift2017,huangGeneticNongeneticInstability2013,mcgranahanBiologicalTherapeuticImpact2015,landauChronicLymphocyticLeukemia2013,ben-davidGeneticTranscriptionalEvolution2018,carterEpigeneticBasisCellular2021}, so patient-oriented discovery approaches may be advantageous when assessing molecular trajectories of relapse and therapeutic response, as well.
Changes in \gls{dname} can be detected through blood draws \cite{peterDynamicsCellfreeDNA2020,shenSensitiveTumourDetection2018,nassiriDetectionDiscriminationIntracranial2020} and offers the potential for a non-invasive biomarker to predict relapse in \gls{ball} patients.
Given the widespread hypermethylation observed at relapse in all patients, it is possible that \gls{ball} patients undergoing continuation/maintenance therapy may additionally benefit from demethylating agents such as 5-aza-cytidine and 5-aza-2'-deoxycytidine to prevent relapse.
Finally, the widespread hypermethylation of \gls{ball} at relapse reflects revision of malignant blasts to a more stem-like phenotype, a characteristic typically observed in other leukemias such as \gls{aml} \cite{krivtsovMLLTranslocationsHistone2007,liClinicalImplicationsGenomewide2017,kresoEvolutionCancerStem2014,shlushIdentificationPreleukaemicHaematopoietic2014,shlushTracingOriginsRelapse2017}.
This suggests that the role of leukemic stem cells and the impact that the cell-of-origin has on therapeutic response, should be prioritized in future research of \gls{ball} relapse.

\section{Limitations}

The main limitations in the studies presented here stem from cancer models, cohort sizes, and correlative measurements.
In \Cref{chap:FOXA1}, we used essentiality data from \gls{pca} cell lines that serve as models of primary prostate tumours.
Further, our dissection of \glspl{cre} relied on encyclopedias of DNA elements derived from cell lines, as did our validation experiments for assessing the impact of mutations seen in primary prostate tumours.
Cell line models are not necessarily representative of primary tumours observed in patients \cite{}.
Moreover, different cell lines may have different regulatory networks, as recent studies of chromatin organization and \gls{cre} contacts in \gls{pca} cell lines have highlighted \cite{ahmedCRISPRiScreensReveal2021}.
This separation between the models we study and the patients we aim to serve may prohibit translation of discoveries from our models to the clinic.

In \Cref{chap:3D}, we were able to profile the genome organization of 12 \gls{pca} patients and 5 benign prostate samples.
Similarly, in \Cref{chap:BALL}, we used \glspl{pdx} originating from 5 patients with \gls{ball}.
In light of the large degree of inter-patient variation seen in tumours from the same tissue, observations in these small sample sizes may not generalize to patients with these diseases as a whole.
In both cases, we used patient-matched data to corroborate findings from other components of the chromatin architecture, where possible, but validation experiments for some observations remain difficult.
Using \gls{crispr} technologies to knock in \glspl{snv} or inactivate individual \glspl{cre} is possible, but replicating complex \glspl{sv} or selectively (de)methylating \glspl{cre} to validate some findings remains technically challenging \cite{nakamuraCRISPRTechnologiesPrecise2021,pickar-oliverNextGenerationCRISPR2019,wangEngineering3DGenome2021}.

\section{Summary and concluding remarks}

Diagnosis, treatment, and the foundational understanding of cancer has been revolutionized by high throughput sequencing technologies and the ability to detect and interpret genetic aberrations in tumours.
When combined with information about \glspl{cre}, the essentiality of genes in multiple cell types, and the three-dimensional genome organization, genetic aberrations can provide a significant amount of explanatory power for the hallmarks of cancer and what causes them.
But genetic aberrations are not the only mechanism cancer cells can use to arrive at these hallmarks to drive disease.
Identifying aberrations across multiple components of the chromatin architecture can uncover complex mechanisms through which cancer cells turn off the expression of tumour suppressor genes, activate oncogenes, or activate pathways.
Interrogating the role of mutations in a gene's set of \glspl{cre} (\Cref{chap:FOXA1}), identifying stable regions of genome organization and shifting chromatin interactions over oncogenesis (\Cref{chap:3D}), and comparing genetic and epigenetic states over disease progression (\Cref{chap:BALL}) are all important approaches to better understand the aberrations, origins, and trajectories of cancers in patients.
This multi-pronged approach requires computational, statistical, and molecular, methods, optimized for low-input samples and small cohort sizes (\Cref{chap:3D,chap:JS}), and methodological developments in these areas are still required.
This thesis focuses on \gls{pca} and \gls{ball}, but the methodologies employed here are not restricted to a single disease.
Proof-of-concept studies here can be extended to all cancers, or any other diseases caused by aberrations to the chromatin architecture.
