\chapter{Introduction}
\label{chap:intro}

Cancer is one of the largest causes of death worldwide, ranking in the top ten most frequent causes in over 150 countries and most frequent in over 40 \cite{brayGlobalCancerStatistics2018}.
Disease treatment is complicated by the fact that cancers are a myriad of diseases with unique origins, symptoms, and treatment options, often related to the cell of origin \cite{gilbertsonMappingCancerOrigins2011}.
However, numerous hallmarks of cancers have emerged over the last 50 years to provide understanding about what biological aberrations cause tumours to initiate, how they develop over time, and how they respond to therapeutic interventions \cite{hanahanHallmarksCancer2000,hanahanHallmarksCancerNext2011,flavahanEpigeneticPlasticityHallmarks2017,pavlovaEmergingHallmarksCancer2016} (\Cref{fig:intro_hallmarks}).

\newflexfigure{chapter1/hallmarks.png}{The hallmarks of cancer}{Adapted from \cite[REF][]{hanahanHallmarksCancerNext2011}.}{fig:intro_hallmarks}{0.5\textwidth}

Many of these hallmarks of cancer can be achieved through aberrations to the genome and the molecular machinery that enables cells to function normally \cite{garrawayLessonsCancerGenome2013}.
For example, genome instability can be achieved by inhibiting DNA repair machinery, as is observed with abnormalities in \emph{MLH1} and \emph{MSH2} repair genes in colorectal cancers \cite{lengauerGeneticInstabilitiesHuman1998} or mutations to \emph{BRCA1}, \emph{BRCA2}, and \emph{ATM} genes in \gls{pca} \cite{abeshouseMolecularTaxonomyPrimary2015}.
Similarly, replicative immortality can be achieved through telomere elongation by over-expression of the \emph{TERT} gene \cite{vinagreFrequencyTERTPromoter2013}.
Mutations to the \emph{TERT} promoter, resulting in its over-expression, were first identified in melanomas \cite{huangHighlyRecurrentTERT2013,hornTERTPromoterMutations2013}, but have since been further identified in bladder, thyroid, and brain cancers \cite{vinagreFrequencyTERTPromoter2013,nagarajanRecurrentEpimutationsActivate2014,sternMutationTERTPromoter2015}.
But while cancer has long been viewed as a disease of the genome \cite{hanahanHallmarksCancer2000,garrawayLessonsCancerGenome2013}, there are many avenues cells can take to arrive these hallmarks resulting from aberrations of how genes are expressed inside the cell nucleus.

\section{Normal chromatin architecture in mammalian cells}

Genes, encoded as DNA, are expressed by being transcribed into \gls{mrna} and subsequently translated into proteins in the process known as the Central Dogma of molecular biology \cite{albertsMolecularBiologyCell2015} (\Cref{fig:intro_dogma}a).
The transcription of genes into \gls{mrna} requires RNA polymerase to bind at \glspl{tss} within DNA elements found at the beginning of genes, termed promoters \cite{goodrichUnexpectedRolesCore2010}.
Promoters are one example of a class of DNA elements, termed \glspl{cre} because of their roles in regulating the expression of genes on the same strand of DNA.
The recruitment of RNA polymerase is aided by a special class of proteins, termed \glspl{tf}, that can bind at DNA sequences either close to a gene's promoter, or far from it at other \glspl{cre} such as enhancers and insulators \cite{schoenfelderLongrangeEnhancerPromoter2019,spitzTranscriptionFactorsEnhancer2012,ongEnhancerFunctionNew2011,anderssonDeterminantsEnhancerPromoter2020,gasznerInsulatorsExploitingTranscriptional2006,oudelaarRelationshipGenomeStructure2020} (\Cref{fig:intro_dogma}b).
Together, the binding of \glspl{tf} to the DNA at specific \glspl{cre} is fundamental for to initiating transcription and expressing genes.

\subsection{DNA elements and features regulating transcription}

The ability of \glspl{tf} to bind at specific \glspl{cre} is dependent on multiple features of the DNA.
Many \glspl{tf} bind to DNA at specific sequences, termed motifs \cite{farnhamInsightsGenomicProfiling2009,spitzTranscriptionFactorsEnhancer2012}.
The structural protein \gls{ctcf} has a well-defined motif and binds to this sequence at thousands of locations across the human genome \cite{kimAnalysisVertebrateInsulator2007,dixonTopologicalDomainsMammalian2012}.
Mutations to the sequence motif can alter \gls{ctcf}'s binding affinity for DNA, as is the case with many \glspl{tf} \cite{kasowskiVariationTranscriptionFactor2010,mauranoWidespreadSitedependentBuffering2012,mauranoLargescaleIdentificationSequence2015}.
Relying on more than just the genetic sequence, \Gls{ctcf} is also an example of a \gls{tf} that is sensitive epigenetic features such as \gls{dname}, the addition of a methyl group to DNA nucleotides \cite{mauranoRoleDNAMethylation2015,wangWidespreadPlasticityCTCF2012,wiehleDNAMethylationEmbryonic2019,xuNascentDNAMethylome2018,vinerModelingMethylsensitiveTranscription2016}, as are DNA methyltransferases DNMT1, DNMT3A, and DNMT3B \cite{gollEukaryoticCytosineMethyltransferases2005,listerHumanDNAMethylomes2009}.
\Gls{tf} binding to DNA can also be affected by the presence of other proteins at binding sites.
\Glspl{tf} can bind in a combinatorial manner at the same location \cite{farnhamInsightsGenomicProfiling2009,ongEnhancerFunctionNew2011,spitzTranscriptionFactorsEnhancer2012} or be blocked from binding altogether by the presence of nucleosomes, protein complexes that DNA winds around to make it compact in three-dimensional space \cite{henikoffNucleosomeDestabilizationEpigenetic2008,jiangNucleosomePositioningGene2009}.
The collection of DNA, nucleosomes, DNA-bound transcription factors, and chemical modifications is defined as the chromatin, and the presence and density of nucleosomes, as well as DNA coiling, make certain segments of the chromatin more or less accessible for \gls{tf} binding (euchromatin and heterochromatin, respectively).
This can affect normal cellular behaviour such as cell-type-specific gene expression \cite{vierstraGlobalReferenceMapping2020,cusanovichSingleCellAtlasVivo2018} and DNA damage repair in inaccessible regions \cite{polakCelloforiginChromatinOrganization2015}.
Thus, both genetic and epigenetic chromatin features affect how \glspl{tf} can bind and regulate transcription.

In addition to \gls{tf} binding, transcription regulation depends on the ability of \glspl{cre} to localize together in three-dimensional space across large genomic distances \cite{zhuTranscriptionFactorsReaders2016,fureyChIPSeqNew2012,carterEpigeneticBasisCellular2021} (\Cref{fig:intro_dogma}c).
Localization of \glspl{cre} tens to thousands of \glspl{bp} apart form focal interactions is aided by the formation of \glspl{tad}, domains of chromatin whose boundaries are linked by structural proteins, including \gls{ctcf} and cohesin \cite{zhouChartingHistoneModifications2011,dekker3DGenomeModerator2016,finnMolecularBasisBiological2019,oudelaarRelationshipGenomeStructure2020}.
In addition to \glspl{tad} which can range in size from $10^4 - 10^6$ \gls{bp}, chromatin is also organized into active or inactive compartments (A and B compartments, respectively) that range in size from $10^5 - 10^6$ \gls{bp} \cite{lieberman-aidenComprehensiveMappingLongRange2009,rao3DMapHuman2014,oudelaarRelationshipGenomeStructure2020,mirnyTwoMajorMechanisms2019}.
These two modes of chromatin organization facilitate the proper localization of \glspl{cre} and \glspl{tf} at the right time.
While \glspl{tad} and compartments are largely conserved across cell types \cite{dixonTopologicalDomainsMammalian2012,stergachisConservationTransactingCircuitry2014,berthelotComplexityConservationRegulatory2017}, focal chromatin interactions can differ up to 45 \% between cell types, providing a further mechanism to change chromatin state \cite{rao3DMapHuman2014}.
Different chromatin states enable cells with the same DNA sequence to express genes differently \cite{spurrellTiesThatBind2016,buenrostroSinglecellChromatinAccessibility2015,leeTranscriptionalRegulationIts2013,ongEnhancerFunctionNew2011,schoenfelderLongrangeEnhancerPromoter2019,zhouChartingHistoneModifications2011}, and thus identifying the repertoire of \glspl{cre}, chromatin interactions, \glspl{tad}, and compartments are vital in determining the regulation of genes in various cell types.

\newfigure{chapter1/transcription-dogma.png}{The basics of gene expression inside the nucleus}{\textbf{a.} The central dogma of molecular biology. Adapted from \cite[REF][]{albertsMolecularBiologyCell2015}. \textbf{b.} Schematic of the transcription machinery to initiate transcription. Adapted from \cite[REF][]{albertsMolecularBiologyCell2015}. \textbf{c.} The scale of chromatin interactions across length scales. Adapted from \cite[REF][]{finnMolecularBasisBiological2019}.}{fig:intro_dogma}

\subsection{Methods for identifying DNA elements and chromatin interactions}

High throughput sequencing protocols have enabled the characterization of functional elements from across the genome and rely on a similar concept to do so.
This concept is to take a molecular feature of interest, be it an RNA transcript or nucleosome position, associate it with a short fragment of DNA, sequence these DNA fragments, and map it to the reference genome to identify where the original molecules came from (\Cref{fig:intro_sequencing}).
\Gls{rnaseq} methods reverse transcribed RNA into DNA that map back to individual genes, with the abundance of fragments indicating how much the gene is expressed \cite{conesaSurveyBestPractices2016}.
Protein binding sites and histone post-translational modifications can be identified by fragmenting DNA around antibodies that bind to these proteins with techniques like \gls{chipseq} and \gls{cutrun} \cite{robertsonGenomewideProfilesSTAT12007,baileyPracticalGuidelinesComprehensive2013,skeneTargetedSituGenomewide2018}.
Accessible and inaccessible chromatin can be assessed by the chromatin's propensity to be cut by enzymes like DNase I, Tn5 transposase, and micrococcal nuclease in \gls{dnaseseq}, \gls{atacseq}, and \gls{mnaseseq} protocols, respectively \cite{boyleHighResolutionMappingCharacterization2008,buenrostroTranspositionNativeChromatin2013,buenrostroATACseqMethodAssaying2015,corcesImprovedATACseqProtocol2017,schonesDynamicRegulationNucleosome2008}.
\Gls{dname} can be measured with bisulfite-sequencing assays \cite{lairdPrinciplesChallengesGenomewide2010}, and distal chromatin interactions can be identified with \gls{3c} and \gls{3c}-based methods such as Hi-C \cite{dekkerCapturingChromosomeConformation2002,liebermanaidenComprehensiveMappingLongRange2009,dixonTopologicalDomainsMammalian2012,noraSpatialPartitioningRegulatory2012,rao3DMapHuman2014}.
Using these genome-wide sequencing techniques across a variety of human cell lines and tissues, the ENCODE Project aimed to catalogue all biochemically functional elements in the human genome to better understand all the ways genes are expressed and how they are regulated in different cell types \cite{birneyIdentificationAnalysisFunctional2007,mooreExpandedEncyclopaediasDNA2020}.
While these measurements help in identifying candidate \glspl{cre} and important regions of the genome, determining their function and which target genes they regulate is a further complicating problem.

\newfigure{chapter1/sequencing-techniques.png}{Characterizing functional DNA elements with high throughput sequencing}{}{fig:intro_sequencing}


% start here

\section{Aberrations to chromatin architecture in cancer}

\subsection{Genetic aberrations in cancer}

\subsubsection{Various types of mutations}

\glspl{snv}, \glspl{cnv}, \glspl{sv}

\subsubsection{Coding and non-coding mutations}

\subsubsection{Impact of coding and non-coding mutations}

\subsection{Non-genetic aberrations in cancer}

\subsubsection{\Glsentrylong{dname} aberrations}

\subsubsection{Histone modifications}

\subsubsection{Chromatin accessibility and nucleosome positioning}

\subsubsection{Enhancer hijacking and chromatin organization aberrations}

\section{Interactions between chromatin architecture components in prostate cancer and leukemia}

\section{Dissertation structure}

I begin with \Cref{chap:FOXA1} by exploring the \emph{cis}-regulatory landscape of \gls{pca} and delineating the \glspl{cre} of the prostate oncogene \emph{FOXA1}.
I demonstrate the essentiality of \emph{FOXA1} for prostate tumours, identify putative \glspl{cre} based on integration of multiomic datasets in \gls{pca} cell lines, and assess the functional impact of recurrent \gls{pca} \glspl{snv} on \emph{FOXA1} expression and \gls{tf} binding.

With the \emph{cis}-regulatory network of \emph{FOXA1} established in \gls{pca}, I attempt to construct the \emph{cis}-regulatory landscape genome-wide in \gls{pca} with \gls{3c} mapping in \Cref{chap:3D}.
Using Hi-C, I characterize the three-dimensional chromatin organization of \gls{pca} and investigate changes to this structure over oncogenesis, and explore the relationship between chromatin organization, \glspl{sv}, and \gls{cre} hijacking.

In assessing the impact of \glspl{sv} on chromatin organization, I uncovered a statistical problem stemming from the lack of recurrent \glspl{sv} across \gls{pca} patients, leading to unbalanced experimental comparisons.
To address this problem, I developed a statistical method for reducing error in gene expression fold-change estimates from unbalanced experimental designs in \Cref{chap:JS} and characterize the method.

Given the shared importance of mutations to \glspl{tf} and epigenetic enzymes in prostate cancer and leukemias, in \Cref{chap:BALL} I explore the epigenetic landscape of \gls{ball} and its relapse after treatment.
I characterize molecular changes to \gls{ball} tumours over the course of disease relapse and identify important changes to \gls{dname} that indicate the reversion to a stem-like phenotype, often present in a subpopulation of cells at diagnosis.

Together, this dissertation investigates the multiple layers of the chromatin architecture that contribute to oncogenesis and cancer progression.
I demonstrate that aberrations to the genome, epigenome, and three-dimensional organization of chromatin play important roles individually, and together, in the orchestration of the disease.
