\chapter{Introduction}
\label{chap:intro}

Cancer is one of the largest causes of death worldwide, ranking in the top ten most frequent causes in over 150 countries and most frequent in over 40 countries \cite{brayGlobalCancerStatistics2018}.
Disease treatment is complicated by the fact that cancers are a myriad of diseases with unique origins, symptoms, and treatment options, often related to the cell of origin \cite{gilbertsonMappingCancerOrigins2011}.
However, numerous hallmarks of cancers have emerged over the last 50 years to provide understanding about what biological aberrations cause tumours to initiate, how they develop over time, and how they respond to therapeutic interventions \cite{hanahanHallmarksCancer2000,hanahanHallmarksCancerNext2011,flavahanEpigeneticPlasticityHallmarks2017,pavlovaEmergingHallmarksCancer2016} (\Cref{fig:intro_hallmarks}).

\newflexfigure{chapter1/hallmarks.png}{The hallmarks of cancer}{Adapted from \cite[REF][]{hanahanHallmarksCancerNext2011}.}{fig:intro_hallmarks}{0.5\textwidth}

Many of these hallmarks of cancer can be achieved through aberrations to the genome and the molecular machinery that enables cells to function normally \cite{garrawayLessonsCancerGenome2013}.
For example, genome instability can be achieved by inhibiting \gls{dna} repair machinery, as is observed with abnormalities in \emph{MLH1} and \emph{MSH2} repair genes in colorectal cancers \cite{lengauerGeneticInstabilitiesHuman1998} or mutations to \emph{BRCA1}, \emph{BRCA2}, and \emph{ATM} genes in \gls{pca} \cite{abeshouseMolecularTaxonomyPrimary2015}.
Similarly, replicative immortality can be achieved through telomere elongation by over-expression of the \emph{TERT} gene \cite{vinagreFrequencyTERTPromoter2013}.
Mutations to the \emph{TERT} promoter, resulting in its over-expression, were first identified in melanomas \cite{huangHighlyRecurrentTERT2013,hornTERTPromoterMutations2013}, but have since been further identified in bladder, thyroid, and brain cancers \cite{vinagreFrequencyTERTPromoter2013,nagarajanRecurrentEpimutationsActivate2014,sternMutationTERTPromoter2015}.
But while cancer has long been viewed as a disease of the genome \cite{hanahanHallmarksCancer2000,garrawayLessonsCancerGenome2013}, there are many avenues cells can take to arrive these hallmarks resulting from aberrations of how genes are expressed inside the cell nucleus.

\section{Normal chromatin architecture in mammalian cells}

Genes, encoded as \gls{dna}, are expressed by being transcribed into \gls{rna} and subsequently translated into proteins in the process known as the central dogma of molecular biology \cite{albertsMolecularBiologyCell2015} (\Cref{fig:intro_dogma}a).
The transcription of genes into \gls{mrna} requires \gls{rna} polymerase to bind at \glspl{tss} within \gls{dna} elements found at the beginning of genes, termed promoters \cite{goodrichUnexpectedRolesCore2010}.
Promoters are one example of a class of \gls{dna} elements, termed \glspl{cre} because of their roles in regulating the expression of genes on the same strand of \gls{dna}.
The recruitment of \gls{rna} polymerase is aided by a special class of proteins, termed \glspl{tf}, that can bind at \gls{dna} sequences either close to a gene's promoter, or far from it at other \glspl{cre} such as enhancers and insulators \cite{schoenfelderLongrangeEnhancerPromoter2019,spitzTranscriptionFactorsEnhancer2012,ongEnhancerFunctionNew2011,anderssonDeterminantsEnhancerPromoter2020,gasznerInsulatorsExploitingTranscriptional2006,oudelaarRelationshipGenomeStructure2020} (\Cref{fig:intro_dogma}b).
Together, the binding of \glspl{tf} to the \gls{dna} at specific \glspl{cre} is fundamental for to initiating transcription and expressing genes.

\subsection{\Glsentryshort{dna} elements and features regulating transcription}

The ability of \glspl{tf} to bind at specific \glspl{cre} is dependent on multiple features of the \gls{dna}.
Many \glspl{tf} bind to \gls{dna} at specific sequences, termed motifs \cite{farnhamInsightsGenomicProfiling2009,spitzTranscriptionFactorsEnhancer2012}.
Finding the locations of a given motif in the genome is often the first step in determining the cistrome of a \gls{tf}, the set of all sites and \glspl{cre} a \gls{tf} binds to \emph{in vivo} \cite{liuCistromeIntegrativePlatform2011,lupienCistromicsHormonedependentCancer2009}.
The structural protein \gls{ctcf} has a well-defined motif and binds to this sequence at thousands of locations across the human genome \cite{kimAnalysisVertebrateInsulator2007,dixonTopologicalDomainsMammalian2012}.
Mutations to the sequence motif can alter \gls{ctcf}'s binding affinity for \gls{dna}, as is the case with many \glspl{tf} \cite{kasowskiVariationTranscriptionFactor2010,mauranoWidespreadSitedependentBuffering2012,mauranoLargescaleIdentificationSequence2015}.
Relying on more than just the genetic sequence, \Gls{ctcf} is also an example of a \gls{tf} that is sensitive epigenetic features such as \gls{dname}, the addition of a methyl group to \gls{dna} nucleotides \cite{mauranoRoleDNAMethylation2015,wangWidespreadPlasticityCTCF2012,wiehleDNAMethylationEmbryonic2019,xuNascentDNAMethylome2018,vinerModelingMethylsensitiveTranscription2016}, as are \gls{dna} methyltransferases DNMT1, DNMT3A, and DNMT3B \cite{gollEukaryoticCytosineMethyltransferases2005,listerHumanDNAMethylomes2009}.
\Gls{tf} binding to \gls{dna} can also be affected by the presence of other proteins at binding sites.
\Glspl{tf} can bind in a combinatorial manner at the same location \cite{farnhamInsightsGenomicProfiling2009,ongEnhancerFunctionNew2011,spitzTranscriptionFactorsEnhancer2012} or be blocked from binding altogether by the presence of nucleosomes, protein complexes that \gls{dna} winds around to make it compact in three-dimensional space \cite{henikoffNucleosomeDestabilizationEpigenetic2008,jiangNucleosomePositioningGene2009}.
The collection of \gls{dna}, nucleosomes, \gls{dna}-bound \glspl{tf}, and chemical modifications is defined as the chromatin, and the presence and density of nucleosomes, as well as \gls{dna} coiling, make certain segments of the chromatin more or less accessible for \gls{tf} binding (euchromatin and heterochromatin, respectively).
This can affect normal cellular behaviour such as cell-type-specific gene expression \cite{vierstraGlobalReferenceMapping2020,cusanovichSingleCellAtlasVivo2018} and \gls{dna} damage repair in inaccessible regions \cite{polakCelloforiginChromatinOrganization2015}.
Thus, both genetic and epigenetic chromatin features affect how \glspl{tf} can bind and regulate transcription.

In addition to \gls{tf} binding, transcription regulation depends on the ability of \glspl{cre} to localize together in three-dimensional space across large genomic distances \cite{zhuTranscriptionFactorsReaders2016,fureyChIPSeqNew2012,carterEpigeneticBasisCellular2021} (\Cref{fig:intro_dogma}c).
Localization of \glspl{cre} that are tens of thousands of \glspl{bp} apart form focal interactions is aided by the formation of \glspl{tad}, domains of chromatin whose boundaries are linked by structural proteins, including \gls{ctcf} and cohesin \cite{zhouChartingHistoneModifications2011,dekker3DGenomeModerator2016,finnMolecularBasisBiological2019,oudelaarRelationshipGenomeStructure2020}.
In addition to \glspl{tad} which can range in size from $10^4 - 10^6$ \gls{bp}, chromatin is also organized into active or inactive compartments (A and B compartments, respectively) that range in size from $10^5 - 10^6$ \gls{bp} \cite{liebermanaidenComprehensiveMappingLongRange2009,rao3DMapHuman2014,oudelaarRelationshipGenomeStructure2020,mirnyTwoMajorMechanisms2019}.
These two modes of chromatin organization facilitate the proper localization of \glspl{cre} and \glspl{tf} at the right time.
While \glspl{tad} and compartments are largely conserved across cell types \cite{dixonTopologicalDomainsMammalian2012,stergachisConservationTransactingCircuitry2014,berthelotComplexityConservationRegulatory2017}, focal chromatin interactions can differ up to 45 \% between cell types, providing a further mechanism to change chromatin state \cite{rao3DMapHuman2014}.
Different chromatin states enable cells with the same \gls{dna} sequence to express genes differently \cite{spurrellTiesThatBind2016,buenrostroSinglecellChromatinAccessibility2015,leeTranscriptionalRegulationIts2013,ongEnhancerFunctionNew2011,schoenfelderLongrangeEnhancerPromoter2019,zhouChartingHistoneModifications2011}, and thus identifying the repertoire of \glspl{cre}, chromatin interactions, \glspl{tad}, and compartments are vital in determining the regulation of genes in various cell types.

\newfigure{chapter1/transcription-dogma.png}{The basics of gene expression inside the nucleus}{\textbf{a.} The central dogma of molecular biology. Adapted from \cite[REF][]{albertsMolecularBiologyCell2015}. \textbf{b.} Schematic of the transcription machinery to initiate transcription. Adapted from \cite[REF][]{albertsMolecularBiologyCell2015}. \textbf{c.} The scale of chromatin interactions across length scales. Adapted from \cite[REF][]{finnMolecularBasisBiological2019}.}{fig:intro_dogma}

\subsection{Methods for identifying \Glsentryshort{dna} elements and chromatin interactions}

High throughput sequencing protocols have enabled the characterization of functional elements from across the genome and rely on a similar concept to do so.
This concept is to take a molecular feature of interest, be it an \gls{rna} transcript or nucleosome position, associate it with a short fragment of \gls{dna}, sequence these \gls{dna} fragments, and map it to the reference genome to identify where the original molecules came from (\Cref{fig:intro_sequencing}).
\Gls{rnaseq} methods reverse transcribed \gls{rna} into \gls{dna} that map back to individual genes, with the abundance of fragments indicating how much the gene is expressed \cite{conesaSurveyBestPractices2016}.
Protein binding sites and histone post-translational modifications can be identified by fragmenting \gls{dna} around antibodies that bind to these proteins with techniques like \gls{chipseq} and \gls{cutrun} \cite{robertsonGenomewideProfilesSTAT12007,baileyPracticalGuidelinesComprehensive2013,skeneTargetedSituGenomewide2018}.
Accessible and inaccessible chromatin can be assessed by the chromatin's propensity to be cut by enzymes like DNase I, Tn5 transposase, and micrococcal nuclease in \gls{dnaseseq}, \gls{atacseq}, and \gls{mnaseseq} protocols, respectively \cite{boyleHighResolutionMappingCharacterization2008,buenrostroTranspositionNativeChromatin2013,buenrostroATACseqMethodAssaying2015,corcesImprovedATACseqProtocol2017,schonesDynamicRegulationNucleosome2008}.
\Gls{dname} can be measured with bisulfite-sequencing assays \cite{lairdPrinciplesChallengesGenomewide2010}, and distal chromatin interactions can be identified with \gls{3c} and \gls{3c}-based methods such as Hi-C \cite{dekkerCapturingChromosomeConformation2002,liebermanaidenComprehensiveMappingLongRange2009,dixonTopologicalDomainsMammalian2012,noraSpatialPartitioningRegulatory2012,rao3DMapHuman2014}.
Yet while these measurements help in identifying candidate \glspl{cre} and important regions of the genome, determining their function and which target genes they regulate is a further complicating problem.

\newfigure{chapter1/sequencing-techniques.png}{Characterizing functional \gls{dna} elements with high throughput sequencing}{}{fig:intro_sequencing}

Varying chromatin states across cell types means that multiple measurements across multiple cell types are necessary to understand the breadth of functions a single \gls{cre} may have.
In 2007, the ENCODE Project aimed to catalogue all biochemically functional elements in the human genome to better understand all the ways genes are expressed and how they are regulated in different cell types \cite{birneyIdentificationAnalysisFunctional2007,mooreExpandedEncyclopaediasDNA2020}.
Using these genome-wide sequencing techniques across a variety of human cell lines and tissues, the ENCODE Project has since catalogued nearly $10^6$ candidate \glspl{cre}, comprising nearly 8 \% of the human genome \cite{mooreExpandedEncyclopaediasDNA2020}.
Interpreting this data requires computational methods to correlate and interpret measurements across samples.
Genome segmentation methods such as ChromHMM \cite{ernstChromHMMAutomatingChromatinstate2012} and Segway \cite{hoffmanUnsupervisedPatternDiscovery2012,chanSegwayGaussianMixture2018}  classify genomic regions according to their predicted function which can be validated with \emph{in vitro} or \emph{in vivo} experiments.
Many techniques for experimental validation, including \gls{crispr}-Cas9, \gls{sirna}, and \gls{shrna}, can interfere with candidate \glspl{cre} by deleting them from the genome, preventing \glspl{tf} from binding to the chromatin, or preventing translation of \gls{mrna} transcripts into proteins \cite{zhouEmergenceNoncodingCancer2016,gasperiniComprehensiveCatalogueValidated2020}.
These same techniques can also be used to screen for candidate \glspl{cre} themselves, through \glspl{mpra} and \gls{crispr} screens \cite{gasperiniComprehensiveCatalogueValidated2020}, necessitating their own suite of statistical and software tools for analyzing observations.
Altogether, a collection of experimental and computational techniques enable the cataloguing and interpretation of thousands of \glspl{cre} and chromatin interactions across many cell types.
These catalogues facilitate understanding how genes are expressed within the complex chromatin architecture in normal cells and, importantly, how aberrations to this architecture can result in disease.

\section{Aberrations to chromatin architecture in cancer}

\subsection{Genetic aberrations in cancer}

Discovery of genetic mutations of oncogenes in tumours nearly 50 years ago spurred the widespread characterization of genetic aberrations in cancers \cite{croceOncogenesCancer2008,baileyComprehensiveCharacterizationCancer2018,weinsteinCancerGenomeAtlas2013,PancancerAnalysisWhole2020}.
These mutations occur within genic regions that code for proteins, but more than 98 \% of somatic mutations acquired in tumours are found in non-coding regions \cite{khuranaRoleNoncodingSequence2016}.
\Glspl{snv}, \glspl{cnv}, and \glspl{sv} are found throughout the genome, and interpreting the impact of these mutations on cancer is an active area of research \cite{mooreExpandedEncyclopaediasDNA2020,rheinbayAnalysesNoncodingSomatic2020,PancancerAnalysisWhole2020,zhangHighcoverageWholegenomeAnalysis2020}.
Analysis of recurrent somatic mutations in tumours led to the identification of \emph{TP53} as a tumour suppressor gene \cite{hollsteinP53MutationsHuman1991}, the frequently mutated \emph{SPOP} gene to help define a molecular subtype of prostate tumours \cite{barbieriExomeSequencingIdentifies2012}, and the interpretation of recurrent rearrangements of the proto-oncogene \emph{MYC} in multiple cancers \cite{meyerReflecting25Years2008}.
The impact of a mutation can also be predicted by identify overlapping regulatory elements or \gls{tf} binding sites \cite{mauranoWidespreadSitedependentBuffering2012,cowper-sal*lariBreastCancerRisk2012,kronEnhancerAlterationsCancer2014}.
Grouping \glspl{cre} by their putative target genes led to the identification of the \emph{ESR1} gene as having its gene regulatory network recurrently mutated in \textapprox 10 \% of breast cancers, resulting in its over-expression, despite the gene itself being mutated in \textapprox 1 \% of breast cancers \cite{baileyNoncodingSomaticInherited2016}.
Similarly, the binding sites of the \emph{FOXA1}, \emph{HOXB13}, \emph{AR}, and \emph{SOX9} \glspl{tf} are enriched with mutations affecting their binding affinities \cite{mazrooeiCistromePartitioningReveals2019} and recurrent amplifications of enhancers near the \emph{AR} and \emph{FOXA1} genes are associated with increased rates of metastasis \cite{quigleyGenomicHallmarksStructural2018,paroliaDistinctStructuralClasses2019}.
Furthermore, mutations that do not directly target gene bodies or \glspl{cre} can lead to oncogene over-expression.
Multiple non-coding \glspl{sv} in pediatric medulloblastoma patients were found to bring the \emph{GFI1} and \emph{GFI1B} oncogenes proximal to enhancer clusters, causing the oncogenes to become aberrantly regulated by this enhancer cluster \cite{northcottEnhancerHijackingActivates2014}.
This mechanism of enhancer hijacking has also been observed in developmental diseases \cite{lupianezDisruptionsTopologicalChromatin2015,allouNoncodingDeletionsIdentify2021}.
While this is not an exhaustive list, it is clear that genetic aberrations are abundant in cancers and that integrating genetic information with other components of the chromatin architecture can help identify driver events that promote oncogenesis or aggressive disease.

Mutations to DNA methyltransferases and chromatin remodelling proteins are common in cancers, and the impact of these mutations can be observed in their chromatin state.
The \gls{idh} enzymes \emph{IDH1}, \emph{IDH2}, and the \gls{tet}  enzymes \emph{TET1} and \emph{TET2} are frequently mutated in cancers, most often in leukemias and gliomas \cite{pirozziImplicationsIDHMutations2021,imDNMT3AIDHMutations2014,issaAcuteMyeloidLeukemia2021,molenaarWildtypeMutatedIDH12018,shihRoleMutationsEpigenetic2012}.
These mutations often affect the \gls{dname} profiles of tumours and differentiation programs \cite{pirozziImplicationsIDHMutations2021}, such as loss of enhancer hydroxymethylation and germinal centre hyperplasia in \gls{dlbcl} \cite{dominguezTET2DeficiencyCauses2018}.
Similarly, mutations to the \emph{EZH2} gene in leukemias can affect the ability of the EZH2 protein to write the H3K27me3 histone mark \cite{plassMutationsRegulatorsEpigenome2013,nikoloskiSomaticMutationsHistone2010,ernstInactivatingMutationsHistone2010,morinSomaticMutationsAltering2010} and \emph{EZH2} over-expression is associated with poor survival in \gls{pca} \cite{varamballyPolycombGroupProtein2002,xuEZH2OncogenicActivity2012,minOncogeneTumorSuppressor2010,kimTargetingEZH2Cancer2016}.
Together, these findings show that genetic aberrations to genes regulating other aspects of the chromatin architecture are abundant in multiple cancers and can drive specific programs in tumours.
These programs can, in turn, affect progression of the disease and treatment strategies for patients.
Importantly, the impact of these mutations is dependent on the function of the affected protein or \gls{cre}, which varies between different cancers.
Thus, understanding how non-genetic aberrations affect tumours can be a vital step in understanding the impact of genetic aberrations.

\subsection{Non-genetic aberrations in cancer}

Non-genetic aberrations to chromatin have long been recognized as important factors in cancer development and progression \cite{jonesCancerepigeneticsComesAge1999,jonesFundamentalRoleEpigenetic2002}.
Methylation of gene promoters is associated with reduced gene expression and loss of \gls{dname} (hypomethylation) across the genome and focal increases of \gls{dname} (hypermethylation) have been found across numerous cancers \cite{jonesFundamentalRoleEpigenetic2002,feinbergHistoryCancerEpigenetics2004}.
Importantly, these changes in \gls{dname} can be found in the absence of mutations targeting DNA methyltransferases.
Analysis of \textapprox 200 metastatic \gls{pca} patients with matching \gls{wgs}, \gls{rnaseq}, and \gls{wgbs} identified a subtype of tumours with a distinct \gls{dname} profile \cite{zhaoDNAMethylationLandscape2020}.
Ependymomas have also been found to display distinct \gls{dname} profiles in the absence of recurrent mutations across patients \cite{mackEpigenomicAlterationsDefine2014} along with \gls{aml}, \gls{all}, glioblastoma, and colorectal, liver, pancreatic, and ovarian cancers \cite{issaCpGIslandMethylator2004}.
Notably, treatment of cancer cells with demethylating agents such as 5-aza-cytidine and 5-aza-2$^\prime$-deoxycytidine for use in patients with \gls{aml} and \gls{mds} have shown to significantly increase survival times, demonstrating the clinical relevance of epigenetic marks in treatment strategies \cite{schmelzInductionGeneExpression2005,azadFutureEpigeneticTherapy2013,kellyEpigeneticModificationsTherapeutic2010}.
Though many causal mechanisms relating \gls{dname} to cancer phenotype are lacking, the impact of \gls{dname} on \gls{tf} binding has been well-demonstrated.
Variable \gls{ctcf} binding across human cell lines has been shown to vary with \gls{dname} levels, which can affect genome organization \cite{wangWidespreadPlasticityCTCF2012,mauranoRoleDNAMethylation2015}.
In gastrointestinal cancer, \gls{ctcf} binding sites are hypermethylated \emph{SDH}-deficient tumours, resulting in widespread loss of \gls{ctcf} and increased contact between the \emph{FGF3} and \emph{FGF4} oncogenes and a nearby enhancer cluster \cite{flavahanAlteredChromosomalTopology2019}.
Moreover, aberrant contact of \emph{FGF3} and \emph{FGF4} is concomitant with increased H3K27ac modifications, further demonstrating the increased regulation and expression of the oncogenes.
Disruptions of \gls{ctcf} binding sites at \gls{tad} boundaries, resulting in aberrant regulation has also been found in T-cell \gls{all}, leading to over-expression of the \emph{TAL1} and \emph{LMO2} oncogenes \cite{hniszActivationProtooncogenesDisruption2016}.
Both of these cases mimic the enhancer hijacking mechanism without the need for nearby genetic mutations.
Together, these results show the importance of \gls{dname} on three-dimensional genome organization and \gls{tf} binding, and genetic and non-genetic aberrations can be observed in chromatin contacts and histone modifications.

The affect of chromatin variants on gene regulation extends beyond \gls{dname}.
Cell type differences in nucleosome occupancy can lead to increased rates of mutation across the genome \cite{pichSomaticGermlineMutation2018}.
Similarly, \gls{tf} binding can affect the ability of DNA damage repair complexes to perform local nucleotide excision repair \cite{sabarinathanNucleotideExcisionRepair2016,gonzalez_perezLocalDeterminantsMutational2019}.
Thus, cell type differences in chromatin state can influence the frequency and location of DNA damage, which may describe some differences in recurrent mutations across cancer types.
Many computational techniques have been developed in an attempt to prioritize the roles of different components of the chromatin architecture.
One method, called \gls{snf}, integrates multiple chromatin measurements together to construct a mathematical graph whereby multiple samples cluster together if they share properties across multiple components \cite{wangSimilarityNetworkFusion2014}.
Many similar methods exist that use machine learning-oriented and biology-oriented techniques to integrate multiple data types together to provide a comprehensive view of the chromatin architecture \cite{rappoportMultiomicMultiviewClustering2018}.
Taken together, these papers demonstrate the effect of differences in normal cell chromatin architecture on cancer and the multiple computational and experimental methods required to unravel these relationships.

Overall, these non-genetic aberrations of chromatin can be found across multiple cancer types.
But we will continue to focus on two seemingly different cancer types that both display complex relationships between different components of the chromatin architecture: \gls{pca} and \gls{ball}.

\section{Chromatin architecture of \glsentrylong{pca} and \glsentrylong{ball}}

\subsection{Prostate cancer}

\subsubsection{Diagnosis, treatment, and risk factors}

\Gls{pca} is the second most commonly diagnosed cancer in men globally, with an estimated 23 300 men being diagnosed with the disease in Canada in 2020 \cite{brayGlobalCancerStatistics2018,brennerProjectedEstimatesCancer2020}.
Diagnosis typically begins with the detection of \gls{psa} in the blood, followed by a digital rectal exam for an enlarged prostate and a core needle biopsy to rule out benign prostate hyperplasia \cite{rebelloProstateCancer2021}.
Once diagnosed, patients are typically grouped into one of several risk categories based on factors including \gls{psa} levels, histopathological assessment (i.e. Gleason or \gls{isup} scores), and medical imaging to detect for distal metastases (\gls{tnm} staging)\cite{rebelloProstateCancer2021}.
\Gls{pca} patients assessed to have a low mortality risk often undergo active surveillance to monitor for changes in the disease that pose a risk to the patient.
Patients with high mortality risks often undergo one of multiple treatment regimens, including surgery, androgen deprivation therapy, chemotherapy, and radiotherapy \cite{rebelloProstateCancer2021}.
While \textapprox 93 \% of men with localized \gls{pca} survive, \textapprox 70 \% of patients with metastatic disease will die within 5 years \cite{hahnMetastaticCastrationSensitiveProstate2018,SEERProstateCancer}, accounting for \textapprox 10 \% of all cancer deaths in men \cite{brennerProjectedEstimatesCancer2020}.
This highlights the need for accurate risk assessment at diagnosis and knowledge of what aberrations lead to aggressive, metastatic disease.

Risk of developing \gls{pca} is associated with age and the median age at diagnosis is 66 years old \cite{rawlaEpidemiologyProstateCancer2019}.
While developing \gls{pca} at a young age is rare, younger men who are diagnosed typically have a more aggressive disease and relatively poorer survival rates \cite{SEERProstateCancer}.
In addition to age, genetic ancestry is a risk factors for developing the disease.
Men of African ancestry are \textapprox 1.6 times more likely to be diagnosed with \gls{pca} than men of western European ancestry, who in turn are \textapprox 2 times more likely than men of Asian ancestry \cite{SEERProstateCancer,smithAfricanAmericanProstateCancer2017,dalleraChangingIncidenceMetastatic2019}.
Men of different ancestries also tend to accumulate different sets of mutations in their tumours.
For example, \textapprox 50 \% of men of western European ancestry harbour a fusion of an ETS gene family member \cite{fraserGenomicHallmarksLocalized2017}, whereas only \textapprox 10 \% of men of Asian ancestry harboured a similar mutation \cite{liGenomicEpigenomicAtlas2020}.
Inherited germline mutations are also a risk factor for \gls{pca}, as men with \emph{BRCA1} and \emph{BRCA2} mutations are \textapprox 2 times more likely to develop \gls{pca} than those without.
Studies identifying these risks demonstrate that familial history, in addition to age and genetic ancestry, are important factors for developing \gls{pca}.

\subsubsection{Chromatin aberrations in \glsentrylong{pca}}

Large cohort studies of prostate tumours have identified numerous driver mutations for the disease.
% do I need all these mutations? should I elaborate on their function?
These driver mutations include, but are not limited to, coding mutations to the \emph{BRCA1}, \emph{BRCA2}, \emph{CHD1}, \emph{IDH1}, \emph{MYC}, \emph{NKX3-1}, \emph{PTEN}, \emph{RB1}, \emph{SPOP}, and \emph{TP53} genes, as well as ETS, FOX, HOX, \emph{KLK}, and KMT factors \cite{fraserGenomicHallmarksLocalized2017,pcf/su2cinternationalprostatecancerdreamteamLongTailOncogenic2018,abeshouseMolecularTaxonomyPrimary2015}.
% feels like a bit of a shoehorn
ETS factor mutations, such as the \gls{t2e} fusion, can lead to a globally \emph{cis}-regulatory landscape, affecting \gls{tf} binding genome-wide and \emph{NOTCH} signalling \cite{kronTMPRSS2ERGFusion2017}.
Metastatic tumours are enriched for amplifications to the \emph{FOXA1} and \gls{ar} genes compared to primary tumours, as well as mutations targeting epigenetic regulators, such as \glspl{kmt} \cite{grassoMutationalLandscapeLethal2012,robinsonIntegrativeClinicalGenomics2015,quigleyGenomicHallmarksStructural2018}.
Over-expression of \gls{ar} is associated with castration resistance, reducing the effectiveness of androgen deprivation therapies \cite{quigleyGenomicHallmarksStructural2018,daskivichRecentProgressHormonal2006}.
% weird phrasing, might want to fix up
Importantly, \emph{FOXA1} is a pioneer \gls{tf} that regulates \gls{ar} expression, and over-expression of \emph{FOXA1} is also more frequently found in metastatic than primary tumours \cite{tengPioneerProstateCancer2021}.
Together, these two genes, their \glspl{cre}, and their cistromes constitute important regions of chromatin that impact the progression of low-risk, localized \gls{pca} into high-risk metastatic \gls{pca}.

\subsection{\Glsentrylong{ball}}

\subsubsection{Diagnosis, treatment, and risk factors}

Leukemia is the 15th most commonly diagnosed cancer globally, with an estimated 6 900 individuals being diagnosed with the disease in Canada in 2020 \cite{brayGlobalCancerStatistics2018,brennerProjectedEstimatesCancer2020}.
Leukemias, generally, result from an overgrowth of undifferentiated blast cells that do not exhibit the same behaviours as fully differentiated cells in the hematopoietic hierarchy \cite{quigleyGenomicHallmarksStructural2018}.
\Gls{ball} is an acute clonal expansion of primitive cells restricted to the lymphoid hematopoietic lineage of B-cells and primarily occurs in children \cite{hungerAcuteLymphoblasticLeukemia2015}.
Currently, overall survival of pediatric \gls{ball} is \textapprox 90 \% \cite{hungerAcuteLymphoblasticLeukemia2015}, yet disease relapse after treatment still occurs in 10 - 15 \% of patients \cite{inabaAcuteLymphoblasticLeukaemia2013,heikampNextGenerationEvaluationTreatment2018}.
Diagnosis of \gls{ball} typically begins with the detection of over-abundant lymphoblasts by microscopy and immunophenotypic assessment of cell surface markers indicating lineage commitment and developmental stage \cite{inabaAcuteLymphoblasticLeukaemia2013}.
After diagnosis, mortality risk is assessed based on factors including age and white blood cell counts.
Patients under 2 or over 10 years of age have worse prognoses than patients of other ages, as do patients with $\ge 50 \times 10^3$ cells / mL \cite{hungerAcuteLymphoblasticLeukemia2015,inabaAcuteLymphoblasticLeukaemia2013}.
Newly diagnosed patients typically undergo remission-induction therapy, intensification/consolidation therapy, and continuation/maintenance therapy over the span of 2 years \cite{inabaAcuteLymphoblasticLeukaemia2013}.
Risk factors for developing the disease include sex, genetic ancestry, and chromosomal rearrangements, with men, African or Hispanic ancestry, and Down's syndrome all associated with an increased risk \cite{inabaAcuteLymphoblasticLeukaemia2013,hungerAcuteLymphoblasticLeukemia2015}.
Risk factors for disease relapse remain elusive; however, karyotpying and high throughput sequencing technologies are helping to identify new biomarkers.

\subsubsection{Chromatin aberrations in \glsentrylong{ball}}

\Gls{ball} is commonly classified according to the presence of recurrent mutations.
Hyperploidy and the presence of the fusion of the \emph{ETV6} and \emph{RUNX1} genes are associated with favourable outcomes, whereas hypoploidy with $<$ 44 chromosomes, fusion of the \emph{BCR} and \emph{ABL1} genes, and mutations affecting the \emph{PAX5}, \emph{EBF1}, \emph{KMT2A}, \emph{CRLF2}, and \emph{IKZF1} genes are all associated with poorer outcomes \cite{inabaAcuteLymphoblasticLeukaemia2013,hungerAcuteLymphoblasticLeukemia2015}.
Many of these affected genes regulate B-cell development, such as \emph{PAX5} \cite{liuPax5LossImposes2014,dangPAX5TumorSuppressor2015,mullighanGenomewideAnalysisGenetic2007}, \emph{IKZF1} \cite{mullighanGenomewideAnalysisGenetic2007}, and \emph{EBF1} \cite{bollerDefiningCellChromatin2018,nuttTranscriptionalRegulationCell2007}.
Similarly, \emph{KMT2A} and \emph{CREBBP} are histone writers, depositing methyl groups to the histone H3 lysine 4 residue and acetyl groups to the histone H3 lysine 56 residue, respectively \cite{slanyMLLFusionProteins2020,krivtsovMLLTranslocationsHistone2007,raoHijackedCancerKMT22015,parkPHD3DomainMLL2010,liStructuralBasisActivity2016,dasBindingHistoneChaperone2014}.
Mutations in these genes are enriched in relapse \cite{hungerAcuteLymphoblasticLeukemia2015,mullighanGenomicAnalysisClonal2008}, suggesting that not only do epigenetic regulators play a key role in oncogenesis, but that they also promote relapse.

Aberrant changes to \gls{dname} may also play a role in \gls{ball} relapse.
\Gls{dname} has been shown to change across B-cell differentiation, with \glspl{dmr} found in the cistromes of \glspl{tf} that regulate differentiation, including \emph{EBF1} and \emph{PAX5} \cite{leeGlobalDNAMethylation2012}.
Additionally, the \gls{dname} profile of \gls{ball} cells differ at thousands of loci across the genome, compared to normal B-cells, primarily in bivalent \glspl{cre} and promoter regions \cite{leeEpigeneticRemodelingBcell2015,nordlundGenomewideSignaturesDifferential2013}.
These findings suggest that aberrant \gls{dname} pattern in \gls{ball} may be affecting B-cell differentiation through \gls{tf} binding.
Moreover, hypomethylation of the \emph{IL2RA} gene is associated with a worse prognosis, as is aberrant \gls{dname} in the presence of \emph{E2A}-\emph{PBX1} or \emph{KMT2A} fusions \cite{gengIntegrativeEpigenomicAnalysis2012}.
This suggests that specific \gls{dname} changes may cooperate with mutated epigenetic regulators to promote aggressive disease that is more likely to relapse after treatment.
Overall, numerous genetic and epigenetic alterations in primary \gls{ball} and relapsed \gls{ball} suggest that multiple chromatin aberrations impact the development and progression of this disease.

While cellular phenotypes and treatment strategies for \gls{pca} and \gls{ball} do not resemble each other,  \gls{pca} oncogenesis, \gls{pca} metastases, and \gls{ball} relapse all harbour aberrations to different components of the chromatin architecture that interact with each other.
Thus, to mitigate, or even prevent, these processes from occurring, this thesis investigates mutations targeting \glspl{cre} of important \glspl{tf}, the relationship between three-dimensional genome organization and \glspl{sv}, and the effect of \gls{dname} changes over the course of relapse.

\section{Thesis structure}

I begin with \Cref{chap:FOXA1} by exploring the \emph{cis}-regulatory landscape of \gls{pca} and delineating the \glspl{cre} of the prostate oncogene \emph{FOXA1}.
I demonstrate the essentiality of \emph{FOXA1} for prostate tumours, identify putative \glspl{cre} based on integration of multiomic datasets in \gls{pca} cell lines, and assess the functional impact of recurrent \gls{pca} \glspl{snv} on \emph{FOXA1} expression and \gls{tf} binding.

With the \emph{cis}-regulatory network of \emph{FOXA1} established in \gls{pca}, I attempt to construct the \emph{cis}-regulatory landscape genome-wide in \gls{pca} with \gls{3c} mapping in \Cref{chap:3D}.
Using Hi-C, I characterize the three-dimensional chromatin organization of \gls{pca} and investigate changes to this structure over oncogenesis, and explore the relationship between chromatin organization, \glspl{sv}, and \gls{cre} hijacking.

In assessing the impact of \glspl{sv} on chromatin organization, I uncovered a statistical problem stemming from the lack of recurrent \glspl{sv} across \gls{pca} patients, leading to unbalanced experimental comparisons.
To address this problem, I developed a statistical method for reducing error in gene expression fold-change estimates from unbalanced experimental designs in \Cref{chap:JS} and characterize the method.

Given the shared importance of mutations to \glspl{tf} and epigenetic enzymes in prostate cancer and leukemias, in \Cref{chap:BALL} I explore the epigenetic landscape of \gls{ball} and its relapse after treatment.
I characterize molecular changes to \gls{ball} tumours over the course of disease relapse and identify important changes to \gls{dname} that indicate the reversion to a stem-like phenotype, often present in a subpopulation of cells at diagnosis.

Together, this thesis investigates the multiple layers of the chromatin architecture that contribute to oncogenesis and cancer progression.
I demonstrate that aberrations to the genome, epigenome, and three-dimensional organization of chromatin play important roles individually, and together, in the orchestration of the disease.
