\chapter{Discussion \& Future Directions}
\label{chap:discussion}

Each of the previous chapters have presented a story interrogating multiple components of the chromatin architecture, how they interact with each other, and the plethora of computational and experimental methods required to unravel this architecture.
\Cref{chap:FOXA1} identifies and validates \glspl{cre} of the \emph{FOXA1} gene, a critical \gls{tf} that regulates \gls{pca} development and regulates \gls{ar} expression to control disease progression.
\Cref{chap:3D} expands on these ideas to investigate how the three-dimensional genome organization impacts gene regulatory networks and how genetic aberrations can alter this organization to promote oncogenesis.
\Cref{chap:JS} develops a mathematical and computational framework to reduce uncertainty about how individual aberrations in chromatin architecture impact gene expression.
Finally, \Cref{chap:BALL} identifies the strong relationship between genetic and epigenetic profiles in \gls{ball} relapse and investigates how \gls{dname} changes and revision to a more stem-like chromatin state may underlie disease relapse.
Together, the work presented in this thesis demonstrates that different components of the chromatin architecture, the genome, molecular chromatin modifications, and three-dimensional organization, can all individually contribute to cancer development and progression.
Moreover, this thesis demonstrates that aberrations in these components work together to drive disease.
These multiple components of the chromatin architecture need to be studied in tandem to understand the origins of cancer and how to develop curative treatments for it.

\section{Implications of non-coding \glsentrylongpl{snv} targeting a single gene}

In \Cref{chap:FOXA1}, I used gene essentiality screening data from multiple cell lines to prioritize the \emph{FOXA1} \gls{tf} as a critical factor across \gls{pca} cell lines.
I also made use of the concept that \glspl{snv} converge on \glspl{cre} of important genes in a given tumour type to predict how these mutations may impact candidate \glspl{cre} for the \emph{FOXA1} gene.
\emph{FOXA1} is also an important \gls{tf} in breast cancers \Cref{fig:FOXA1_figs3}.
Similar investigations into the impact of \glspl{snv} in breast tumours may identify the impact of aberrations to the \glspl{cre} of \emph{FOXA1}.
Identifying important genes in this manner is not limited to \emph{FOXA1} and breast and prostate tumours.
Critical genes may be identified in other cancer types using \gls{crispr} screens or \glspl{mpra}.
Similarly \glspl{snv} are not the only chromatin aberrations that can affect \gls{tf} binding or gene regulation.
Other chromatin aberrations may accumulate in \glspl{cre} of important genes in a similar fashion.
Complex \glspl{sv}, changes in \gls{dname}, or histone modifications may only need to accumulate in the set of \glspl{cre} for a given gene, rather than be recurrent in a single element, to affect its expression.
Interpreting chromatin aberrations in cancer in light of this plexus-based approach may aid in identifying driver events for cancer by aggregating previously unrelated events together.
These approaches are not limited to prostate tumours and can serve as a starting point to identify important genes in other cancers, more generally.

\section{Implications of three-dimensional organization and enhancer hijacking in \glsentrylong{pca}}

In \Cref{chap:3D}, my co-authors optimized a low-input Hi-C method to interrogate genome organization in cryo-preserved prostate tissue slides.
I then demonstrated that this could produce a high quality Hi-C library and helped produce that largest collection of genome organization data in prostate tumours to date.
This technological step forward opens the door for profiling the three-dimensional genome in cancer patients without relying in cell lines or other models, and may be a critical step in moving personalized medicine forward.
We add to existing evidence that \glspl{sv} can, but rarely, alter 3D structure in disease \cite{ghavi-helmHighlyRearrangedChromosomes2019,oudelaarRelationshipGenomeStructure2020,despangFunctionalDissectionSox92019,williamsonDevelopmentallyRegulatedShh2019,dixonIntegrativeDetectionAnalysis2018,akdemirDisruptionChromatinFolding2020,liPatternsSomaticStructural2020,iyyankiSubtypeassociatedEpigenomicLandscape2021}.
Elucidating when and how \glspl{sv} impact genome organization, then, is still an area that requires investigation.
Developments in statistical methods, such as those discussed in \Cref{chap:JS}, may help identify the effects of individual, non-recurrent \glspl{sv}.
Subclonality of \glspl{sv} may interfere with the ability to detect rearranged domains in bulk Hi-C measurements.
Thus, developments in high throughput sequencing and microscopy measurements in single cells, such as ORCA \cite{mateoVisualizingDNAFolding2019} and STORM \cite{batesStochasticOpticalReconstruction2013}, as well as organoid or explant models that recapitulate the chromatin state of the original tumour, may help in identifying the effect of such events \cite{}.
This work also adds to our ability to detect chromatin interactions between promoters and enhancers in patient samples, allowing for better characterization of gene regulatory networks for each and every gene.
Given the benefits of plexus-based approaches to interpreting aberrations in the chromatin architecture, this work serves as a foundation on which to integrate gene regulatory networks with chromatin aberrations in cancers more generally.
This foundation can be extended to studying the evolution of these networks, their genome organization, and their resiliency between species or over time as tumours respond to therapeutic interventions \cite{}.

\section{Implications of \glsentrylong{dname} changes in relapse}

\Cref{chap:BALL} identifies \gls{dname} as an epigenetic marker that can mirror that mutational profile of \gls{ball} cells.
The \gls{dname} changes observed over the course of \gls{ball} relapse have the potential to become a biomarker predicting relapse, although variation in \gls{dname} changes across patients necessitates larger sample sizes before recurrent events may be robustly identified.
Our patient-oriented approach to identify recurrent changes to \gls{dname} boosted our discovery of recurrent \glspl{dmr} over a cohort-oriented approach, and thus may be a beneficial strategy for similar studies.
Inter-tumour heterogeneity in tumours is a well-studied phenomenon \cite{marusykTumorHeterogeneityCauses2010,sottorivaCatchMyDrift2017,huangGeneticNongeneticInstability2013,mcgranahanBiologicalTherapeuticImpact2015,landauChronicLymphocyticLeukemia2013,ben-davidGeneticTranscriptionalEvolution2018,carterEpigeneticBasisCellular2021}, so patient-oriented discovery approaches may be advantageous when assessing molecular trajectories of relapse and therapeutic response, as well.
Changes in \gls{dname} can be detected through blood draws \cite{peterDynamicsCellfreeDNA2020,shenSensitiveTumourDetection2018,nassiriDetectionDiscriminationIntracranial2020} and offers the potential for a non-invasive biomarker to predict relapse in \gls{ball} patients.
Given the widespread hypermethylation observed at relapse in all patients, it is possible that \gls{ball} patients undergoing continuation/maintenance therapy may additionally benefit from demethylating agents such as 5-aza-cytidine and 5-aza-2'-deoxycytidine to prevent relapse.
Finally, the widespread hypermethylation of \gls{ball} at relapse reflects revision of malignant blasts to a more stem-like phenotype, a characteristic typically observed in other leukemias such as \gls{aml} \cite{krivtsovMLLTranslocationsHistone2007,liClinicalImplicationsGenomewide2017,kresoEvolutionCancerStem2014,shlushIdentificationPreleukaemicHaematopoietic2014,shlushTracingOriginsRelapse2017}.
This suggests that the role of leukemic stem cells and the impact that the cell-of-origin has on therapeutic response, should be prioritized in future research of \gls{ball} relapse.

\section{Summary and concluding remarks}

\begin{itemize}
  \item work does not focus on a single disease, should extend this type of analysis to all cancers, since they all appear to harbour aberrations affecting multiple components of the chromatin architecture
  \item multi-pronged approach of computational, statistical, and molecular, and microscopy methods optimized for low-input samples targeting the set of DNA elements and their relationships to each other in individual patients to develop personalized medicines and treat cancer at its origins in the chromatin
\end{itemize}
