\chapter{Introduction}
\label{chap:intro}

Cancer is one of the largest causes of death worldwide, ranking in the top ten most frequent causes in over 150 countries and most frequent in over 40 \cite{brayGlobalCancerStatistics2018}.
Disease treatment is complicated by the fact that cancers are a myriad of diseases with unique origins, symptoms, and treatment options, often related to the cell of origin \cite{gilbertsonMappingCancerOrigins2011}.
However, numerous hallmarks of cancers have emerged over the last 50 years to provide understanding about what biological aberrations cause tumours to initiate, how they develop over time, and how they respond to therapeutic interventions \cite{hanahanHallmarksCancer2000,hanahanHallmarksCancerNext2011,flavahanEpigeneticPlasticityHallmarks2017,pavlovaEmergingHallmarksCancer2016} (\Cref{fig:intro_hallmarks}).

\newflexfigure{chapter1/hallmarks.png}{The hallmarks of cancer}{Adapted from \cite[REF][]{hanahanHallmarksCancerNext2011}.}{fig:intro_hallmarks}{0.5\textwidth}

Many of these hallmarks of cancer can be achieved through aberrations to the genome and the molecular machinery that enables cells to function normally \cite{garrawayLessonsCancerGenome2013}.
For example, genome instability can be achieved by inhibiting \gls{dna} repair machinery, as is observed with abnormalities in \emph{MLH1} and \emph{MSH2} repair genes in colorectal cancers \cite{lengauerGeneticInstabilitiesHuman1998} or mutations to \emph{BRCA1}, \emph{BRCA2}, and \emph{ATM} genes in \gls{pca} \cite{abeshouseMolecularTaxonomyPrimary2015}.
Similarly, replicative immortality can be achieved through telomere elongation by over-expression of the \emph{TERT} gene \cite{vinagreFrequencyTERTPromoter2013}.
Mutations to the \emph{TERT} promoter, resulting in its over-expression, were first identified in melanomas \cite{huangHighlyRecurrentTERT2013,hornTERTPromoterMutations2013}, but have since been further identified in bladder, thyroid, and brain cancers \cite{vinagreFrequencyTERTPromoter2013,nagarajanRecurrentEpimutationsActivate2014,sternMutationTERTPromoter2015}.
But while cancer has long been viewed as a disease of the genome \cite{hanahanHallmarksCancer2000,garrawayLessonsCancerGenome2013}, there are many avenues cells can take to arrive these hallmarks resulting from aberrations of how genes are expressed inside the cell nucleus.

\section{Normal chromatin architecture in mammalian cells}

Genes, encoded as \gls{dna}, are expressed by being transcribed into \gls{rna} and subsequently translated into proteins in the process known as the Central Dogma of molecular biology \cite{albertsMolecularBiologyCell2015} (\Cref{fig:intro_dogma}a).
The transcription of genes into \gls{mrna} requires \gls{rna} polymerase to bind at \glspl{tss} within \gls{dna} elements found at the beginning of genes, termed promoters \cite{goodrichUnexpectedRolesCore2010}.
Promoters are one example of a class of \gls{dna} elements, termed \glspl{cre} because of their roles in regulating the expression of genes on the same strand of \gls{dna}.
The recruitment of \gls{rna} polymerase is aided by a special class of proteins, termed \glspl{tf}, that can bind at \gls{dna} sequences either close to a gene's promoter, or far from it at other \glspl{cre} such as enhancers and insulators \cite{schoenfelderLongrangeEnhancerPromoter2019,spitzTranscriptionFactorsEnhancer2012,ongEnhancerFunctionNew2011,anderssonDeterminantsEnhancerPromoter2020,gasznerInsulatorsExploitingTranscriptional2006,oudelaarRelationshipGenomeStructure2020} (\Cref{fig:intro_dogma}b).
Together, the binding of \glspl{tf} to the \gls{dna} at specific \glspl{cre} is fundamental for to initiating transcription and expressing genes.

\subsection{\Glsentryshort{dna} elements and features regulating transcription}

The ability of \glspl{tf} to bind at specific \glspl{cre} is dependent on multiple features of the \gls{dna}.
Many \glspl{tf} bind to \gls{dna} at specific sequences, termed motifs \cite{farnhamInsightsGenomicProfiling2009,spitzTranscriptionFactorsEnhancer2012}.
The structural protein \gls{ctcf} has a well-defined motif and binds to this sequence at thousands of locations across the human genome \cite{kimAnalysisVertebrateInsulator2007,dixonTopologicalDomainsMammalian2012}.
Mutations to the sequence motif can alter \gls{ctcf}'s binding affinity for \gls{dna}, as is the case with many \glspl{tf} \cite{kasowskiVariationTranscriptionFactor2010,mauranoWidespreadSitedependentBuffering2012,mauranoLargescaleIdentificationSequence2015}.
Relying on more than just the genetic sequence, \Gls{ctcf} is also an example of a \gls{tf} that is sensitive epigenetic features such as \gls{dname}, the addition of a methyl group to \gls{dna} nucleotides \cite{mauranoRoleDNAMethylation2015,wangWidespreadPlasticityCTCF2012,wiehleDNAMethylationEmbryonic2019,xuNascentDNAMethylome2018,vinerModelingMethylsensitiveTranscription2016}, as are \gls{dna} methyltransferases DNMT1, DNMT3A, and DNMT3B \cite{gollEukaryoticCytosineMethyltransferases2005,listerHumanDNAMethylomes2009}.
\Gls{tf} binding to \gls{dna} can also be affected by the presence of other proteins at binding sites.
\Glspl{tf} can bind in a combinatorial manner at the same location \cite{farnhamInsightsGenomicProfiling2009,ongEnhancerFunctionNew2011,spitzTranscriptionFactorsEnhancer2012} or be blocked from binding altogether by the presence of nucleosomes, protein complexes that \gls{dna} winds around to make it compact in three-dimensional space \cite{henikoffNucleosomeDestabilizationEpigenetic2008,jiangNucleosomePositioningGene2009}.
The collection of \gls{dna}, nucleosomes, \gls{dna}-bound transcription factors, and chemical modifications is defined as the chromatin, and the presence and density of nucleosomes, as well as \gls{dna} coiling, make certain segments of the chromatin more or less accessible for \gls{tf} binding (euchromatin and heterochromatin, respectively).
This can affect normal cellular behaviour such as cell-type-specific gene expression \cite{vierstraGlobalReferenceMapping2020,cusanovichSingleCellAtlasVivo2018} and \gls{dna} damage repair in inaccessible regions \cite{polakCelloforiginChromatinOrganization2015}.
Thus, both genetic and epigenetic chromatin features affect how \glspl{tf} can bind and regulate transcription.

In addition to \gls{tf} binding, transcription regulation depends on the ability of \glspl{cre} to localize together in three-dimensional space across large genomic distances \cite{zhuTranscriptionFactorsReaders2016,fureyChIPSeqNew2012,carterEpigeneticBasisCellular2021} (\Cref{fig:intro_dogma}c).
Localization of \glspl{cre} tens to thousands of \glspl{bp} apart form focal interactions is aided by the formation of \glspl{tad}, domains of chromatin whose boundaries are linked by structural proteins, including \gls{ctcf} and cohesin \cite{zhouChartingHistoneModifications2011,dekker3DGenomeModerator2016,finnMolecularBasisBiological2019,oudelaarRelationshipGenomeStructure2020}.
In addition to \glspl{tad} which can range in size from $10^4 - 10^6$ \gls{bp}, chromatin is also organized into active or inactive compartments (A and B compartments, respectively) that range in size from $10^5 - 10^6$ \gls{bp} \cite{liebermanaidenComprehensiveMappingLongRange2009,rao3DMapHuman2014,oudelaarRelationshipGenomeStructure2020,mirnyTwoMajorMechanisms2019}.
These two modes of chromatin organization facilitate the proper localization of \glspl{cre} and \glspl{tf} at the right time.
While \glspl{tad} and compartments are largely conserved across cell types \cite{dixonTopologicalDomainsMammalian2012,stergachisConservationTransactingCircuitry2014,berthelotComplexityConservationRegulatory2017}, focal chromatin interactions can differ up to 45 \% between cell types, providing a further mechanism to change chromatin state \cite{rao3DMapHuman2014}.
Different chromatin states enable cells with the same \gls{dna} sequence to express genes differently \cite{spurrellTiesThatBind2016,buenrostroSinglecellChromatinAccessibility2015,leeTranscriptionalRegulationIts2013,ongEnhancerFunctionNew2011,schoenfelderLongrangeEnhancerPromoter2019,zhouChartingHistoneModifications2011}, and thus identifying the repertoire of \glspl{cre}, chromatin interactions, \glspl{tad}, and compartments are vital in determining the regulation of genes in various cell types.

\newfigure{chapter1/transcription-dogma.png}{The basics of gene expression inside the nucleus}{\textbf{a.} The central dogma of molecular biology. Adapted from \cite[REF][]{albertsMolecularBiologyCell2015}. \textbf{b.} Schematic of the transcription machinery to initiate transcription. Adapted from \cite[REF][]{albertsMolecularBiologyCell2015}. \textbf{c.} The scale of chromatin interactions across length scales. Adapted from \cite[REF][]{finnMolecularBasisBiological2019}.}{fig:intro_dogma}

\subsection{Methods for identifying \Glsentryshort{dna} elements and chromatin interactions}

High throughput sequencing protocols have enabled the characterization of functional elements from across the genome and rely on a similar concept to do so.
This concept is to take a molecular feature of interest, be it an \gls{rna} transcript or nucleosome position, associate it with a short fragment of \gls{dna}, sequence these \gls{dna} fragments, and map it to the reference genome to identify where the original molecules came from (\Cref{fig:intro_sequencing}).
\Gls{rnaseq} methods reverse transcribed \gls{rna} into \gls{dna} that map back to individual genes, with the abundance of fragments indicating how much the gene is expressed \cite{conesaSurveyBestPractices2016}.
Protein binding sites and histone post-translational modifications can be identified by fragmenting \gls{dna} around antibodies that bind to these proteins with techniques like \gls{chipseq} and \gls{cutrun} \cite{robertsonGenomewideProfilesSTAT12007,baileyPracticalGuidelinesComprehensive2013,skeneTargetedSituGenomewide2018}.
Accessible and inaccessible chromatin can be assessed by the chromatin's propensity to be cut by enzymes like DNase I, Tn5 transposase, and micrococcal nuclease in \gls{dnaseseq}, \gls{atacseq}, and \gls{mnaseseq} protocols, respectively \cite{boyleHighResolutionMappingCharacterization2008,buenrostroTranspositionNativeChromatin2013,buenrostroATACseqMethodAssaying2015,corcesImprovedATACseqProtocol2017,schonesDynamicRegulationNucleosome2008}.
\Gls{dname} can be measured with bisulfite-sequencing assays \cite{lairdPrinciplesChallengesGenomewide2010}, and distal chromatin interactions can be identified with \gls{3c} and \gls{3c}-based methods such as Hi-C \cite{dekkerCapturingChromosomeConformation2002,liebermanaidenComprehensiveMappingLongRange2009,dixonTopologicalDomainsMammalian2012,noraSpatialPartitioningRegulatory2012,rao3DMapHuman2014}.
Yet while these measurements help in identifying candidate \glspl{cre} and important regions of the genome, determining their function and which target genes they regulate is a further complicating problem.

\newfigure{chapter1/sequencing-techniques.png}{Characterizing functional \gls{dna} elements with high throughput sequencing}{}{fig:intro_sequencing}

Varying chromatin states across cell types means that multiple measurements across multiple cell types are necessary to understand the breadth of functions a single \gls{cre} may have.
In 2007, the ENCODE Project aimed to catalogue all biochemically functional elements in the human genome to better understand all the ways genes are expressed and how they are regulated in different cell types \cite{birneyIdentificationAnalysisFunctional2007,mooreExpandedEncyclopaediasDNA2020}.
Using these genome-wide sequencing techniques across a variety of human cell lines and tissues, the ENCODE Project has since catalogued nearly $10^6$ candidate \glspl{cre}, comprising nearly 8 \% of the human genome \cite{mooreExpandedEncyclopaediasDNA2020}.
Interpreting this data requires computational methods to correlate and interpret measurements across samples.
Genome segmentation methods such as ChromHMM \cite{ernstChromHMMAutomatingChromatinstate2012} and Segway \cite{hoffmanUnsupervisedPatternDiscovery2012,chanSegwayGaussianMixture2018}  classify genomic regions according to their predicted function which can be validated with \emph{in vitro} or \emph{in vivo} experiments.
Many techniques for experimental validation, including \gls{crispr}-Cas9, \gls{sirna}, and \gls{shrna}, can interfere with candidate \glspl{cre} by deleting them from the genome, preventing \glspl{tf} from binding to the chromatin, or preventing translation of \gls{mrna} transcripts into proteins \cite{zhouEmergenceNoncodingCancer2016,gasperiniComprehensiveCatalogueValidated2020}.
These same techniques can also be used to screen for candidate \glspl{cre} themselves, through \glspl{mpra} and \gls{crispr} screens \cite{gasperiniComprehensiveCatalogueValidated2020}, necessitating their own suite of statistical and software tools for analyzing observations.
Altogether, a collection of experimental and computational techniques enable the cataloguing and interpretation of thousands of \glspl{cre} and chromatin interactions across many cell types.
These catalogues facilitate understanding how genes are expressed within the complex chromatin architecture in normal cells and, importantly, how aberrations to this architecture can result in disease.

\section{Aberrations to chromatin architecture in cancer}

\subsection{Genetic aberrations in cancer}

Discovery of genetic mutations of oncogenes in tumours nearly 50 years ago spurred the widespread characterization of genetic aberrations in cancers \cite{croceOncogenesCancer2008,baileyComprehensiveCharacterizationCancer2018,weinsteinCancerGenomeAtlas2013,PancancerAnalysisWhole2020}.
These mutations occur within genic regions that code for proteins, but more than 98 \% of somatic mutations acquired in tumours are found in non-coding regions \cite{khuranaRoleNoncodingSequence2016}.
\Glspl{snv}, \glspl{cnv}, and \glspl{sv} are found throughout the genome, and interpreting the impact of these mutations on cancer is an active area of research \cite{mooreExpandedEncyclopaediasDNA2020,rheinbayAnalysesNoncodingSomatic2020,PancancerAnalysisWhole2020,zhangHighcoverageWholegenomeAnalysis2020}.
Analysis of recurrent somatic mutations in tumours led to the identification of \emph{TP53} as a tumour suppressor gene \cite{hollsteinP53MutationsHuman1991}, the frequently mutated \emph{SPOP} gene to help define a molecular subtype of prostate tumours \cite{barbieriExomeSequencingIdentifies2012}, and the interpretation of recurrent rearrangements of the proto-oncogene \emph{MYC} in multiple cancers \cite{meyerReflecting25Years2008}.
The impact of a mutation can also be predicted by identify overlapping regulatory elements or \gls{tf} binding sites \cite{mauranoWidespreadSitedependentBuffering2012,cowper-sal*lariBreastCancerRisk2012,kronEnhancerAlterationsCancer2014}.
Grouping \glspl{cre} by their putative target genes led to the identification of the \emph{ESR1} gene as having its gene regulatory network recurrently mutated in \textapprox 10 \% breast cancers, resulting in its over-expression, despite the gene itself being mutated in \textapprox 1 \% of breast cancers \cite{baileyNoncodingSomaticInherited2016}.
Similarly, the binding sites of the \emph{FOXA1}, \emph{HOXB13}, \emph{AR}, and \emph{SOX9} \glspl{tf} are enriched with mutations affecting their binding affinities \cite{mazrooeiCistromePartitioningReveals2019} and recurrent amplifications of enhancers near the \emph{AR} and \emph{FOXA1} genes are associated with increased rates of metastasis \cite{quigleyGenomicHallmarksStructural2018,paroliaDistinctStructuralClasses2019}.
Furthermore, mutations that do not directly target gene bodies or \glspl{cre} can lead to oncogene over-expression.
Multiple non-coding \glspl{sv} in pediatric medulloblastoma patients were found to bring the \emph{GFI1} and \emph{GFI1B} oncogenes proximal to enhancer clusters, causing the oncogenes to become aberrantly regulated by this enhancer cluster \cite{northcottEnhancerHijackingActivates2014}.
This mechanism of enhancer hijacking has also been observed in developmental diseases \cite{lupianezDisruptionsTopologicalChromatin2015,allouNoncodingDeletionsIdentify2021}.
While genetic aberrations are abundant in cancers, integrating genetic information with other components of the chromatin architecture can help identify driver events that promote oncogenesis or aggressive disease.

Mutations to DNA methyltransferases and chromatin remodelling proteins are common in cancers, and the impact of these mutations can be observed in their chromatin state.
The \gls{idh} enzymes \emph{IDH1}, \emph{IDH2}, and the \gls{tet}  enzymes \emph{TET1} and \emph{TET2} are frequently mutated in cancers, most often in leukemias and gliomas \cite{pirozziImplicationsIDHMutations2021,imDNMT3AIDHMutations2014,issaAcuteMyeloidLeukemia2021,molenaarWildtypeMutatedIDH12018,shihRoleMutationsEpigenetic2012}.
These mutations often affect the \gls{dname} profiles of tumours and differentiation programs \cite{pirozziImplicationsIDHMutations2021}, such as loss of enhancer hydroxymethylation and germinal centre hyperplasia in \gls{dlbcl} \cite{dominguezTET2DeficiencyCauses2018}.
Similarly, mutations to the \emph{EZH2} gene in leukemias can affect the ability of the EZH2 protein to write the H3K27me3 histone mark \cite{plassMutationsRegulatorsEpigenome2013,nikoloskiSomaticMutationsHistone2010,ernstInactivatingMutationsHistone2010,morinSomaticMutationsAltering2010} and \emph{EZH2} over-expression is associated with poor survival in \gls{pca} \cite{varamballyPolycombGroupProtein2002,xuEZH2OncogenicActivity2012,minOncogeneTumorSuppressor2010,kimTargetingEZH2Cancer2016}.
Together, these findings show that genetic aberrations to genes regulating other aspects of the chromatin architecture are abundant in multiple cancers and can drive specific programs in tumours.
These programs can, in turn, affect progression of the disease and treatment strategies for patients.
Importantly, the impact of these mutations is dependent on the function of the affected protein or \gls{cre}, which varies between different cancers.
Thus, understanding how non-genetic aberrations affect tumours can be a vital step in understanding the impact of genetic aberrations.

\subsection{Non-genetic aberrations in cancer}

\subsubsection{\Glsentrylong{dname} aberrations}

\subsubsection{Histone modifications}

\subsubsection{Chromatin accessibility and nucleosome positioning}

\subsubsection{Enhancer hijacking and chromatin organization aberrations}

\section{Interactions between chromatin architecture components in prostate cancer and leukemia}

\section{Dissertation structure}

I begin with \Cref{chap:FOXA1} by exploring the \emph{cis}-regulatory landscape of \gls{pca} and delineating the \glspl{cre} of the prostate oncogene \emph{FOXA1}.
I demonstrate the essentiality of \emph{FOXA1} for prostate tumours, identify putative \glspl{cre} based on integration of multiomic datasets in \gls{pca} cell lines, and assess the functional impact of recurrent \gls{pca} \glspl{snv} on \emph{FOXA1} expression and \gls{tf} binding.

With the \emph{cis}-regulatory network of \emph{FOXA1} established in \gls{pca}, I attempt to construct the \emph{cis}-regulatory landscape genome-wide in \gls{pca} with \gls{3c} mapping in \Cref{chap:3D}.
Using Hi-C, I characterize the three-dimensional chromatin organization of \gls{pca} and investigate changes to this structure over oncogenesis, and explore the relationship between chromatin organization, \glspl{sv}, and \gls{cre} hijacking.

In assessing the impact of \glspl{sv} on chromatin organization, I uncovered a statistical problem stemming from the lack of recurrent \glspl{sv} across \gls{pca} patients, leading to unbalanced experimental comparisons.
To address this problem, I developed a statistical method for reducing error in gene expression fold-change estimates from unbalanced experimental designs in \Cref{chap:JS} and characterize the method.

Given the shared importance of mutations to \glspl{tf} and epigenetic enzymes in prostate cancer and leukemias, in \Cref{chap:BALL} I explore the epigenetic landscape of \gls{ball} and its relapse after treatment.
I characterize molecular changes to \gls{ball} tumours over the course of disease relapse and identify important changes to \gls{dname} that indicate the reversion to a stem-like phenotype, often present in a subpopulation of cells at diagnosis.

Together, this dissertation investigates the multiple layers of the chromatin architecture that contribute to oncogenesis and cancer progression.
I demonstrate that aberrations to the genome, epigenome, and three-dimensional organization of chromatin play important roles individually, and together, in the orchestration of the disease.
