\section{Discussion}

Modern technologies and understanding of the epigenome allow the possibility of probing \glspl{cre} involved in regulating genes implicated in disease.
Despite FOXA1 being recurrently mutated \cite{abeshouseMolecularTaxonomyPrimary2015,fraserGenomicHallmarksLocalized2017,barbieriExomeSequencingIdentifies2012,grassoMutationalLandscapeLethal2012,robinsonIntegrativeClinicalGenomics2015} and playing potent oncogenic roles in \gls{pca} etiology \cite{paroliaDistinctStructuralClasses2019,adamsFOXA1MutationsAlter2019,gaoForkheadDomainMutations2019}, the \glspl{cre} involved in its transcriptional regulation are poorly understood.
Understanding how FOXA1 is expressed can provide a complementary strategy to antagonize FOXA1 in \gls{pca}.

We used the DHS profiled in \gls{pca} cells to identify putative FOXA1 \glspl{cre} through annotating these regions with five different histone modifications, TF binding sites and noncoding \glspl{snv} profiled in \gls{pca} cells and primary prostate tumours.
Our efforts identified and validated a set of six active \glspl{cre} involved in FOXA1 regulation, agreeing with a recent report where a subset of our \glspl{cre} map to loci suggested to be in contact with the FOXA1 promoter \cite{rhieHighresolution3DEpigenomic2019}.
The disruption of these six distal \glspl{cre} each significantly reduced FOXA1 mRNA levels, similar to what has been demonstrated for ESR1 in luminal breast cancer \cite{baileyNoncodingSomaticInherited2016}, MLH1 in Lynch syndrome \cite{liuDisruption35Kb2018}, MYC in lung adenocarcinoma and endometrial cancer \cite{zhangIdentificationFocallyAmplified2016}, and AR in mCRPC \cite{takedaSomaticallyAcquiredEnhancer2018,viswanathanStructuralAlterationsDriving2018}.
Through combinatorial deletion of two \glspl{cre}, FOXA1 mRNA levels were further reduced in comparison with single CRE deletions, raising the possibility of CRE additivity \cite{osterwalderEnhancerRedundancyProvides2018}.
The deletion of the FOXA1 plexus \glspl{cre} also significantly reduced \gls{pca} cell proliferation at levels comparable to what has been reported upon deletion of the amplified CRE upstream of the AR gene in mCRPC \cite{takedaSomaticallyAcquiredEnhancer2018}, suggestive of onco-\glspl{cre} as reported in lung \cite{zhangIdentificationFocallyAmplified2016} and prostate \cite{takedaSomaticallyAcquiredEnhancer2018} cancer.

More than 90\% of \glspl{snv} found in cancer map to the noncoding genome \cite{meltonRecurrentSomaticMutations2015,mazrooeiCistromePartitioningReveals2019} with a portion of these \glspl{snv} mapping to \glspl{cre} altering their transactivation potential \cite{baileyNoncodingSomaticInherited2016,zhangIntegrativeFunctionalGenomics2012,huangHighlyRecurrentTERT2013,hornTERTPromoterMutations2013} and/or downstream target gene expression \cite{zhouEmergenceNoncodingCancer2016,meltonRecurrentSomaticMutations2015,weinholdGenomewideAnalysisNoncoding2014}.
We extended this concept with \glspl{snv} identified from primary prostate tumours mapping to FOXA1 plexus \glspl{cre}.
We observed that a subset of these \glspl{snv} can alter transactivation potential by modulating the binding of specific \glspl{tf} whose cistromes are preferentially burdened by \glspl{snv} in primary \gls{pca} \cite{mazrooeiCistromePartitioningReveals2019}.
Our findings complement recent reports of \glspl{snv} found in the noncoding space of FOXA1 that could affect its expression \cite{annalaFrequentMutationFOXA12018,camcapstudygroupSequencingProstateCancers2018}.
The FOXA1 plexus \glspl{cre} we identified here are also reported to be target of structural variants in both the primary and metastatic settings \cite{paroliaDistinctStructuralClasses2019,quigleyGenomicHallmarksStructural2018}, including tandem duplication in \textapprox 14\% (14/101) mCRPC tumours over CRE2 \cite{quigleyGenomicHallmarksStructural2018}, amplification, duplication and translocation over CRE3, CRE4, and CRE5 \cite{paroliaDistinctStructuralClasses2019}.
Notably, the translocation and duplication defining the FOXMIND enhancer driving FOXA1 expression reported in primary and metastatic settings harbors the CRE3 element we characterized \cite{paroliaDistinctStructuralClasses2019}.
Collectively, these studies combined with our discoveries reveal the fundamental contribution of the FOXA1 plexus in \gls{pca} etiology.
As a whole, our findings in conjunction with recent reports suggest that \glspl{cre} involved in the transcriptional regulation of FOXA1 may be hijacked in prostate tumours through various types of genetic alterations.

Despite initial treatment responses from treating aggressive primary and metastatic \gls{pca} through castration to suppress AR signalling \cite{attardProstateCancer2016}, resistance ensues as 80\% of mCRPC tumours harbor either AR gene amplification, amplification of a CRE upstream of AR, or activating AR coding mutations \cite{robinsonIntegrativeClinicalGenomics2015,takedaSomaticallyAcquiredEnhancer2018,quigleyGenomicHallmarksStructural2018}.
Given the AR-dependent \cite{yangCurrentPerspectivesFOXA12015,pomerantzAndrogenReceptorCistrome2015} and AR-independent \cite{sunkelIntegrativeAnalysisIdentifies2017} oncogenic activity of FOXA1 in \gls{pca}, its inhibition is an appealing alternative therapeutic strategy.
Our dissection of the FOXA1 cis-regulatory landscape complement recent findings through revealing loci that are important for the regulation of FOXA1.
Theoretically, direct targeting of the \glspl{cre} regulating FOXA1 would down-regulate FOXA1 levels and could therefore serve as a valid alternative to antagonize its function.

Taken together, we identified FOXA1 \glspl{cre} targeted by \glspl{snv} that are capable of altering transactivation potential through the modulation of key \gls{pca} \glspl{tf}.
The study supports the importance of considering \glspl{cre} not only as lone occurrences but as a team that works together to regulate their target genes, particularly when considering the impact of genetic alterations.
As such, our work builds a bridge between the understanding of FOXA1 transcriptional regulation and new routes to FOXA1 inhibition.
Aligning with recent reports \cite{paroliaDistinctStructuralClasses2019,adamsFOXA1MutationsAlter2019,gaoForkheadDomainMutations2019}, our findings support the oncogenic nature of FOXA1 in \gls{pca}.
Gaining insight on the cis-regulatory plexuses of important genes such as FOXA1 in \gls{pca} may provide new avenues to inhibit other drivers across various cancer types to halt disease progression.
