\section{Results}

\subsection{\emph{FOXA1} is essential for \glsentrylong{pca} proliferation}

We interrogated \emph{FOXA1} expression levels across cancer types.
We find that \emph{FOXA1} \gls{mrna} is consistently the most abundant in prostate tumours compared to 25 other cancer types across patients (\Cref{fig:FOXA1_fig1}a), ranking in the 95th percentile for 492 of 497 prostate tumours profiled in \gls{tcga} (\Cref{fig:FOXA1_figs1}a).
Using the same dataset we also find that \emph{FOXA1} is the most highly expressed out of 41 other \gls{fox} factors in prostate tumours (\Cref{fig:FOXA1_figs1}b).
We next analyzed expression data from \gls{depmap} and observed \emph{FOXA1} to be most highly expressed in \gls{pca} cell lines compared to cell lines of other cancer types (\Cref{fig:FOXA1_figs2}a).
Amongst the eight \gls{pca} cell lines in the dataset (22Rv1, DU145, LNCaP, MDA-PCa-2B, NCI-H660, PrECLH, PC3, and VCaP), \emph{FOXA1} \gls{mrna} abundance is above the 90th percentile in all but one cell line (PrECLH) compared to the $>$ 56,000 protein coding and non-protein coding genes profiled (\Cref{fig:FOXA1_figs2}b).
These new results gained from the \gls{tcga} and \gls{depmap} validate previous understanding that \emph{FOXA1} is one of the highest expressed genes in \gls{pca} \cite{tsourlakisFOXA1ExpressionStrong2017}.

\newfigure{Fig1.png}{\emph{FOXA1} is highly expressed in \gls{pca} and essential for \gls{pca} cell proliferation.}{\textbf{a.} The \gls{mrna} expression of \emph{FOXA1} across tumour types ($n = 26$) from \gls{rnaseq} data of \gls{tcga}. \textbf{b.} \emph{FOXA1} essentiality mediated through \gls{rnai} across various cell lines ($n=707$) from \gls{depmap}. Gene essentiality scores are normalized $z$-scores. Higher scores indicate less essential, and lower scores indicate more essential for cell proliferation. $x$-axis indicate tissue of origin for each cell line tested. Each dot indicates one cell line. \textbf{c.} Gene essentiality mediated through \gls{rnai} across \gls{pca} cell lines ($n=8$) from \gls{depmap}. Each dot indicates one gene, red indicates \emph{FOXA1}. \textbf{d.} Representative Western blot against \emph{FOXA1} in LNCaP cells 5 days post-transfection of non-targeting \gls{sirna} and two independent \gls{sirna} targeting \emph{FOXA1}. \textbf{e.} Cell proliferation assay conducted in LNCaP cells upon \gls{sirna}-mediated knockdown of \emph{FOXA1} across 5 days. \textbf{f.} Cell proliferation assay conducted in VCaP cells upon \gls{sirna}-mediated knockdown of \emph{FOXA1} across 5 days. Error bars indicate $\pm$ s.d. $n=3$ independent experiments. Mann-Whitney U test, * $p < 0.05$, ** $p < 0.01$.}{fig:FOXA1_fig1}

Following up on \emph{FOXA1} \gls{mrna} expression levels, we interrogated the essentiality of \emph{FOXA1} for \gls{pca} cell growth.
\gls{rnai}-mediated essentiality screens compiled in \gls{depmap} show that \emph{FOXA1} lies in the 94th percentile across 6 of the 8 available \gls{pca} cell lines: 22Rv1, LNCaP, MDA PCa 2B, NCI-H660, PC3, and VCaP cells (\Cref{fig:FOXA1_fig1}b-c).
The median \gls{rnai}-mediated essentiality score for all prostate cell lines is significantly lower than all other cell lines, suggesting that \emph{FOXA1} is especially essential for \gls{pca} cell proliferation (permutation test, $p = 1 \times 10^{-6}$, see \Cref{sec:foxa1_methods}; \Cref{fig:FOXA1_figs3}a).
Growth assays in LNCaP and VCaP cells following \emph{FOXA1} knockdown using two independent \gls{sirna}s (\Cref{fig:FOXA1_fig1}d, \Cref{fig:FOXA1_figs3}b) show significant growth inhibition in LNCaP (\gls{sirna} \#1: 4-fold, \gls{sirna} \#2: 3.35-fold) and VCaP (\gls{sirna} \#1: 8.7-fold, \gls{sirna} \#2: 2-fold) cells five days post-transfection (Mann-Whitney U Test, $p<0.05$; \Cref{fig:FOXA1_fig1}e-f).
In accordance with previous reports, our results using essentiality datasets followed by knockdown validation reveals that \emph{FOXA1} is oncogenic and essential for \gls{pca} cell proliferation.

\subsection{Identifying putative \emph{FOXA1} \glsentryshortpl{cre}}

The interweaving of distal \glspl{cre} with target gene promoters establishes regulatory plexuses with some to be ascribed to specific genes \cite{sallariConvergenceDispersedRegulatory2016,baileyNoncodingSomaticInherited2016}.
Regulatory plexuses stem from chromatin interactions orchestrated by various factors including ZNF143, YY1, CTCF and the cohesin complex \cite{phillipsCTCFMasterWeaver2009,weintraubYY1StructuralRegulator2017,baileyZNF143ProvidesSequence2015}.
Motivated by the oncogenic role of \emph{FOXA1} in \gls{pca}, we investigated its regulatory plexus controlling its expression.
According to chromatin contact frequency maps generated from Hi-C assays performed in LNCaP \gls{pca} cells, \emph{FOXA1} lies in a 440 kbp \gls{tad} (chr14: $37720001-38160000$ $\pm$ 40 kbp adjusting for resolution; \Cref{fig:FOXA1_fig2}a).
By overlaying DNase-seq data from LNCaP \gls{pca} cells, there are a total of 123 putative \glspl{cre} reported as \glspl{dhs} that populate this \gls{tad} (\Cref{fig:FOXA1_fig2}a).
We next inferred the regulatory plexus of \emph{FOXA1} using the C3D method \cite{mehdiC3DToolPredict2019}.
C3D aggregates and draws correlation of \gls{dhs} signal intensities between the cell line of choice and the \gls{dhs} signal across all systems in a collection of cell lines and tissues \cite{mehdiC3DToolPredict2019}.
Anchoring our analysis to the \emph{FOXA1} promoter and using accessible chromatin regions defined in LNCaP \gls{pca} cells identified 55 putative \glspl{cre} to the \emph{FOXA1} regulatory plexus ($r > 0.7$; \Cref{fig:FOXA1_fig2}b).

\newfigure{Fig2.png}{Epigenetic annotation of 14q21.1 locus and identification of \emph{FOXA1} \glspl{cre}}{\textbf{a.} Overview of \emph{cis}-regulatory landscape surrounding \emph{FOXA1} on the 14q21.1 locus. H3K27ac signal track is the \gls{chipseq} signal overlay of 19 primary prostate tumours. LNCaP Hi-C depicts the \gls{tad} structure around \emph{FOXA1}. LNCaP signal values with peak calls below indicate the accessibility of the chromatin around \emph{FOXA1}. Mutations indicate \glspl{snv} identified in 200 primary prostate tumours. \textbf{b.} Functional annotation of putative \emph{FOXA1} \glspl{cre} using \gls{tf} and histone modification \gls{chipseq} conducted in primary tumours and \gls{pca} cell lines. Annotated in the matrix are all \gls{dhs} within the \gls{tad} and $\pm$ 40 kbp resolution left and right of the \gls{tad}. Putative \emph{FOXA1} \glspl{cre} targeted by noncoding \Glspl{snv} for downstream validation are boxed. Coordinates are 1-based in the GRCh37 reference genome.}{fig:FOXA1_fig2}

\subsection{Putative \emph{FOXA1} \glsentryshortpl{cre} harbour \glsentryshort{tf} binding sites and \glsentryshortpl{snv}}

To delineate the \glspl{cre} that could be actively involved in the transcriptional regulation of \emph{FOXA1}, we annotated the \gls{dhs} from LNCaP cells with available \gls{chipseq} data for histone modifications and \glspl{tf} conducted in LNCaP, 22Rv1, VCaP \gls{pca} cell lines and primary prostate tumours (\Cref{fig:FOXA1_fig2}b) \cite{pomerantzAndrogenReceptorCistrome2015,kronTMPRSS2ERGFusion2017}.
Close to 60\% (33/55) of the putative \emph{FOXA1} plexus \glspl{cre} are marked by H3K27ac profiled in primary prostate tumours \cite{kronTMPRSS2ERGFusion2017}, indicative of active \glspl{cre} in tumours (\Cref{fig:FOXA1_fig2}b) \cite{creyghtonHistoneH3K27acSeparates2010}.
Next, considering that noncoding \Glspl{snv} can target a set of \glspl{cre} that converge on the same target gene in cancer \cite{baileyNoncodingSomaticInherited2016}, we overlapped the somatic \Glspl{snv} called from \gls{wgs} across 200 primary prostate tumours to the 33 H3K27ac-marked \gls{dhs} predicted to regulate \emph{FOXA1} \cite{fraserGenomicHallmarksLocalized2017,espirituEvolutionaryLandscapeLocalized2018}.
This analysis identified 6 out of the 33 \gls{dhs} marked with H3K27ac (18.2\%) harboring one or more \glspl{snv} (10 total \glspl{snv} called from 9 tumours; \Cref{fig:FOXA1_fig2}b).
We observe that these 6 \glspl{cre} can be bound by multiple \glspl{tf} in \gls{pca} cells, including \emph{FOXA1}, \emph{\gls{ar}} and \emph{HOXB13} (\Cref{fig:FOXA1_fig2}b, \Cref{fig:FOXA1_figs4}).
The Hi-C data from the LNCaP \gls{pca} cells corroborates the C3D predictions as demonstrated by the elevated contact frequency between the region harboring the \emph{FOXA1} promoter and where the 6 \glspl{cre} are located, compared to other loci in the same \gls{tad} (\Cref{fig:FOXA1_fig3}a).
The 6 \glspl{cre} lie in intergenic or intronic regions (\Cref{fig:FOXA1_fig3}b-h).
Together, histone modifications, \gls{tf} binding sites and noncoding \glspl{snv} support that these 6 putative \glspl{cre} are active in primary \gls{pca}.
The Hi-C and C3D predictions suggest that they regulate \emph{FOXA1} expression.

\newfigure{Fig3.png}{Putative \glspl{cre} predicted to interact with \emph{FOXA1} promoter}{\textbf{a.} Hi-C contact matrix conducted in LNCaP cells indicating physical interactions between putative \emph{FOXA1} \glspl{cre} and the \emph{FOXA1} promoter. Hi-C resolution is 40 kbp. \textbf{b.} Epigenome annotations around the \emph{FOXA1} locus. The six putative \emph{FOXA1} \glspl{cre} are coloured in yellow. \textbf{c-h.} Zoom-in of each individual putative \emph{FOXA1} \gls{cre}. C3D value is the Pearson correlation of \gls{dhs} signal between LNCaP and the \gls{dhs} reference matrix. Coordinates are 1-based in the GRCh37 reference genome.}{fig:FOXA1_fig3}

\subsection{Disruption of \glsentryshortpl{cre} reduces \emph{FOXA1} \glsentryshort{mrna} expression}

We next assessed the role of \glspl{cre} toward \emph{FOXA1} expression using LNCaP and 22Rv1 clones stably expressing the \gls{wt} Cas9 protein (\Cref{fig:FOXA1_fig4}a-b).
\Glspl{grna} designed against the \emph{FOXA1} gene (exon 1 and intron 1) served as positive controls while an outside-\gls{tad} region (termed Chr14 (-)), a region on a different chromosome (the human \emph{AAVS1} safe-harbor site at the \emph{PPP1R12C} locus \cite{kronTMPRSS2ERGFusion2017,dekelverFunctionalGenomicsProteomics2010}), and three regions within the \gls{tad} predicted to be excluded from the \emph{FOXA1} plexus served as negative controls.
Individual deletion of the \emph{FOXA1} plexus \glspl{cre} through transient transfection of \glspl{grna} into the LNCaP cells (see \Cref{sec:FOXA1_methods}) led to significantly decreased \emph{FOXA1} \gls{mrna} expression ($\Delta$\gls{cre}1 \textapprox 29.3 $\pm$ 8.3\%, $\Delta$\gls{cre}2 \textapprox 40.1 $\pm$ 11.8\%, $\Delta$\gls{cre}3 \textapprox 30.6 $\pm$ 9.1\%, $\Delta$\gls{cre}4 \textapprox 23.6 $\pm$ 8.2\%, $\Delta$\gls{cre}5 \textapprox 25.3 $\pm$ 6.6\%, $\Delta$\gls{cre}6 \textapprox 24.5 $\pm$ 10.2\% and $\Delta$\emph{FOXA1} (exon 1 and intron 1) \textapprox 87.4 $\pm$ 8.8\% reduction relative to basal levels; \Cref{fig:FOXA1_fig4}c, \Cref{fig:FOXA1_figs5}a-f).
In contrast, deletion of several negative control regions within the same \gls{tad} did not significantly reduce \emph{FOXA1} \gls{mrna} level (\Cref{fig:FOXA1_fig4}c, \Cref{fig:FOXA1_figs5}g-i).
Similar results were observed in 22Rv1 \gls{pca} cells (\Cref{fig:FOXA1_fig4}d).
As each clone expressed Cas9 protein at different levels, there may be a difference between genome editing efficiencies between the clones.
We compared the CRISPR/Cas9 on-target genome editing efficiency across the five LNCaP cell line-derived clones with the relative \emph{FOXA1} \gls{mrna} levels, and indeed observe a significant inverse correlation across all \glspl{cre} (Pearson's correlation $r = 0.49, p < 0.005$; \Cref{fig:FOXA1_figs6}a) and agreeing trends for each individual \gls{cre} (\Cref{fig:FOXA1_figs6}b).

\newlongfigure{Fig4.png}{Functional dissection of putative \emph{FOXA1} \glspl{cre}}{\textbf{a.} Representative western blot probed against Cas9 in LNCaP clones ($n = 5$ clones) derived to stably express Cas9 protein upon blasticidin selection. \textbf{b.} Representative western blot probed against Cas9 in 22Rv1 clones ($n = 4$ clones) derived to stably express Cas9 protein upon blasticidin selection. \textbf{c.} \emph{FOXA1} \gls{mrna} expression normalized to housekeeping \emph{TBP} \gls{mrna} expression upon CRISPR/Cas9-mediated deletion of each \gls{cre} using LNCaP clones ($n = 5$ independent experiments, each dot represents an independent clone). \textbf{d.} \emph{FOXA1} \gls{mrna} expression normalized to housekeeping \emph{TBP} \gls{mrna} expression upon CRISPR/Cas9-mediated deletion of each \gls{cre} using 22Rv1 clones ($n = 4$ independent experiments, each dot represents an independent clone). \textbf{e.} Representative western blot probed against Cas9 in LNCaP clones ($n = 4$ clones) derived to stably express the dCas9-KRAB fusion protein upon blasticidin selection. \textbf{f.} Representative western blot probed against Cas9 in 22Rv1 clones ($n = 4$ clones) derived to stably express dCas9-KRAB fusion protein upon blasticidin selection. \textbf{g.} \emph{FOXA1} \gls{mrna} expression normalized to housekeeping \emph{TBP} \gls{mrna} expression upon dCas9-KRAB-mediated repression of each \gls{cre} using LNCaP clones ($n = 4$ independent experiments, each dot represents an independent clone). \textbf{h.} \emph{FOXA1} \gls{mrna} expression normalized to housekeeping \emph{TBP} \gls{mrna} expression upon dCas9-KRAB-mediated repression of each \gls{cre} using 22Rv1 clones ($n = 4$ independent experiments, each dot represents an independent clone). \emph{FOXA1} \gls{mrna} expression was normalized to basal \emph{FOXA1} expression prior to statistical testing. $\Delta$ indicates CRISPR/Cas9-mediated deletion, $i$ indicates dCas9-KRAB-mediated repression. Error bars indicate $\pm$ s.d. Student's $t$-test, n.s. not significant, * $p < 0.05$, ** $p < 0.01$, *** $p < 0.001$.}{fig:FOXA1_fig4}

Complementary to our findings using the \gls{wt} CRISPR/Cas9 system, we next generated four LNCaP and four 22Rv1 cell line-derived dCas9-KRAB fusion protein expressing clones (\Cref{fig:FOXA1_fig4}e-f).
Transient transfection of the same \glspl{grna} used in the \gls{wt} Cas9 experiments, targeting the six \emph{FOXA1} plexus \glspl{cre} into our dCas9-KRAB LNCaP clones significantly decreased \emph{FOXA1} expression relative to basal levels (i\gls{cre}1 \textapprox 24.6 $\pm$ 6.2\%, i\gls{cre}2 \textapprox 42.2 $\pm$ 10.8\%, i\gls{cre}3 \textapprox 25.3 $\pm$ 9.2\%, i\gls{cre}4 \textapprox 23.3 $\pm$ 4.3\%, i\gls{cre}5 \textapprox 30.2 $\pm$ 3.4\% and i\gls{cre}6 \textapprox 23.1 $\pm$ 8.1\% reduction).
Similarly, \glspl{grna} targeting the dCas9-KRAB fusion protein to \emph{FOXA1} decreased its expression (i\emph{FOXA1} \textapprox 81.6 $\pm$ 11.8\% reduction; Student's $t$-test, $p<0.05$, \Cref{fig:FOXA1_fig4}g).
Analogous results were also observed in our four clonal 22Rv1 dCas9-KRAB cell lines (Student's $t$-test, $p<0.05$, \Cref{fig:FOXA1_fig4}h).
Collectively, our results suggest that the six \glspl{cre} control \emph{FOXA1} expression.

We further assessed the regulatory activity of the six \emph{FOXA1} plexus \glspl{cre} by testing the consequent \gls{mrna} expression on other genes within the same \gls{tad}, namely \emph{MIPOL1} and \emph{TTC6}.
$\Delta$\gls{cre}1 and $\Delta$\gls{cre}2 significantly reduced \emph{MIPOL1} \gls{mrna} expression by \textapprox 38.4 $\pm$ 6.4\% and \textapprox 48.4 $\pm$ 9\%, respectively relative to basal levels, whereas deletion of the other four \glspl{cre} did not result in any significant \emph{MIPOL1} expression changes (Student's $t$-test, $p<0.05$, \Cref{fig:FOXA1_figs7}a).
On the other hand, deletion of \glspl{cre} each significantly reduced \emph{TTC6} \gls{mrna} expression relative to its basal levels ($\Delta$\gls{cre}1 \textapprox 52.9\% $\pm$ 6.4\%, $\Delta$\gls{cre}2 \textapprox 66 $\pm$ 11.3\%, $\Delta$\gls{cre}3 \textapprox 55.5 $\pm$ 12.8\%, $\Delta$\gls{cre}4 44.9 $\pm$ 10.6\%, $\Delta$\gls{cre}5 43.1 $\pm$ 11.9\% and $\Delta$\gls{cre}6 52.2 $\pm$ 7.3\% reduction (Student's $t$-test, $p<0.05$, \Cref{fig:FOXA1_figs7}b), in agreement with the fact that \emph{TTC6} shares its promoter with \emph{FOXA1} as both genes are transcribed on opposing strands (\Cref{fig:FOXA1_figs7}c).

Reduction in \emph{FOXA1} \gls{mrna} expression resulting from the deletion of \emph{FOXA1} plexus \glspl{cre} may also impact gene expression downstream of \emph{FOXA1}, we assessed the \gls{mrna} expression of several \emph{FOXA1} target genes, namely \emph{SNAI2}, \emph{ACPP}, and \emph{GRIN3A}.
Deletion of \glspl{cre} resulted in significant change in \emph{SNAI2} (up-regulation; $\Delta$\gls{cre}1 \textapprox 190\%, $\Delta$\gls{cre}2 \textapprox 162.8\%, $\Delta$\gls{cre}3 \textapprox 147.5\%, $\Delta$\gls{cre}4 \textapprox 133.3\%, $\Delta$\gls{cre}5 \textapprox 137.3\%, $\Delta$\gls{cre}6 \textapprox 120.8\%, $\Delta$\emph{FOXA1} \textapprox 266.7\%), \emph{ACPP} (down-regulation; $\Delta$\gls{cre}1 \textapprox 73.5\%, $\Delta$\gls{cre}2 \textapprox 62.5\%, $\Delta$\gls{cre}3 \textapprox 69.6\%, $\Delta$\gls{cre}4 \textapprox 75.6\%, $\Delta$\gls{cre}5 \textapprox 70.9\%, $\Delta$\gls{cre}6 \textapprox 74.6\%, $\Delta$\emph{FOXA1} \textapprox 52.2\%) and \emph{GRIN3A} expression (up-regulation; $\Delta$\gls{cre}1 \textapprox 138.2\%, $\Delta$\gls{cre}2 \textapprox 168.8\%, $\Delta$\gls{cre}3 \textapprox 144.6\%, $\Delta$\gls{cre}4 \textapprox 132.1\%, $\Delta$\gls{cre}5 \textapprox 131.4\%, $\Delta$\gls{cre}6 \textapprox 127\%, $\Delta$\emph{FOXA1} \textapprox 228\%; Student's $t$-test, $p < 0.05$, \Cref{fig:FOXA1_figs7}d-f).
Collectively, our results support the restriction of most \emph{FOXA1} plexus \glspl{cre} towards \emph{FOXA1} and its target genes.

\subsection{\emph{FOXA1} \glsentryshortpl{cre} collaborate to regulate its expression}

Expanding on the idea that multiple \glspl{cre} can converge to regulate the expression of a single target gene \cite{sallariConvergenceDispersedRegulatory2016,baileyNoncodingSomaticInherited2016,pennacchioEnhancersFiveEssential2013}, we asked whether the \glspl{cre} we identified collaboratively regulate \emph{FOXA1} \gls{mrna} expression.
Here, we applied a transient approach that delivers Cas9 protein:\gls{grna} as a ribonucleoprotein (RNP) complex formed prior to transfection that would avoid the heterogeneity of Cas9 protein expression across the \gls{pca} cell clones (see \Cref{sec:FOXA1_methods}).
We first validated this system through single \gls{cre} deletions, where we transiently transfected a set of \gls{grna} targeting the \gls{cre} of interest.
In accordance with data from our \gls{pca} cell clones stably expressing \gls{wt} Cas9 and dCas9-KRAB, individual \gls{cre} deletion resulted in a significant reduction in \emph{FOXA1} \gls{mrna} expression: ($\Delta$\gls{cre}1 \textapprox 29.3 $\pm$ 7.3\%, $\Delta$\gls{cre}2 \textapprox 36 $\pm$ 11.8\%, $\Delta$\gls{cre}3 \textapprox 30.6 $\pm$ 12.7\%, $\Delta$\gls{cre}4 \textapprox 24.5 $\pm$ 6.1\%, $\Delta$\gls{cre}5 \textapprox 23.7 $\pm$ 13.2\%, $\Delta$\gls{cre}6 \textapprox 26.8 $\pm$ 14.2\% and $\Delta$\emph{FOXA1} \textapprox 96.2 $\pm$ 1.4\% reduction; Student's $t$-test, $p < 0.05$, \Cref{fig:FOXA1_fig5}a, \Cref{fig:FOXA1_figs8}a-f).
Next for combinatorial deletions, we prioritized the \glspl{cre} that harbor more than 1 \gls{snv} (i.e. \gls{cre}1, \gls{cre}2, \gls{cre}4), and transiently transfected RNP complexes that target both \glspl{cre} in various combinations (i.e. \gls{cre}1 + \gls{cre}2, \gls{cre}1 + \gls{cre}4, \gls{cre}2 + \gls{cre}4), and assessed \emph{FOXA1} \gls{mrna} expression.
Compared to negative control regions, the combinatorial deletion of $\Delta$\gls{cre}1 + $\Delta$\gls{cre}2, $\Delta$\gls{cre}1 + $\Delta$\gls{cre}4, and $\Delta$\gls{cre}2 + $\Delta$\gls{cre}4 resulted in a significant \textapprox 48.5 $\pm$ 4.5\%, \textapprox 50.4 $\pm$ 2.9\% and \textapprox 45.2 $\pm$ 5.5\% reduction in \emph{FOXA1} \gls{mrna} expression, respectively (Student's $t$-test, $p<0.05$, \Cref{fig:FOXA1_fig5}b, \Cref{fig:FOXA1_figs9}a-f) a fold reduction greater than single \gls{cre} deletions (Student's $t$-test, \Cref{fig:FOXA1_figs10}, $p<0.05$).
These results together demonstrate that these \glspl{cre} collaboratively contribute to the establishment and regulation of \emph{FOXA1} expression in \gls{pca}.

\newfigure{Fig5.png}{\emph{FOXA1} \glspl{cre} collaborate to regulate its expression and are critical for \gls{pca} cell proliferation}{\textbf{a.} \emph{FOXA1} \gls{mrna} expression normalized to housekeeping \emph{TBP} \gls{mrna} expression upon transient transfection-based CRISPR/Cas9-mediated deletion of \gls{cre}1, \gls{cre}2, \gls{cre}4, and sequential deletion combinations ($n = 5$ independent experiments). Dots represent values from individual replicates. \textbf{b.} \emph{FOXA1} \gls{mrna} expression normalized to housekeeping \emph{TBP} \gls{mrna} expression upon bulk lentiviral-based CRISPR/Cas9-mediated deletion of each \gls{cre} in LNCaP cells ($n = 3$ independent experiments). \textbf{c.} Cell proliferation assay conducted after puromycin and blasticidin selection for LNCaP cells carrying deleted regions of interest. Data was based on cell counting 6 days after seeding post-selection ($n = 3$, representative of three independent experiments). \emph{FOXA1} \gls{mrna} expression upon deletion was normalized to basal \emph{FOXA1} expression prior to statistical testing. \emph{FOXA1} \gls{mrna} expression was normalized to the basal LNCaP \emph{FOXA1} expression prior to statistical testing. $\Delta$ indicates CRISPR/Cas9-mediated deletion. Error bars indicate $\pm$ s.d. Student's $t$-test, n.s. not significant, * $p < 0.05$, ** $p < 0.01$, *** $p < 0.001$.}{fig:FOXA1_fig5}

\subsection{Disruption of \emph{FOXA1} \glsentryshortpl{cre} reduces \glsentrylong{pca} cell growth}

As \emph{FOXA1} is essential for \gls{pca} growth (\Cref{fig:FOXA1_fig1}b-e), we next sought to assess the importance of the six \glspl{cre} in the \emph{FOXA1} plexus towards \gls{pca} cell growth.
We adapted a lentiviral-based approach that expressed both the Cas9 protein and two \gls{grna} that target each \gls{cre} for deletion (see \Cref{sec:foxa1_methods}).
Upon lentiviral transduction with subsequent selection, we separated LNCaP \gls{pca} cells for RNA, DNA and for cell proliferation.
We first tested the system by measuring \emph{FOXA1} \gls{mrna} expression, and independently observed significant reductions of \emph{FOXA1} \gls{mrna} expression ($\Delta$\gls{cre}1 \textapprox 18\%, $\Delta$\gls{cre}2 \textapprox 30\%, $\Delta$\gls{cre}3 \textapprox 15\%, $\Delta$\gls{cre}4 \textapprox 12\%, $\Delta$\gls{cre}5 \textapprox 35\%, $\Delta$\gls{cre}6 \textapprox 46\% and $\Delta$ \emph{FOXA1} (exon 1 and intron 1) \textapprox 48\% reduction (Student's $t$-test, $p<0.05$, \Cref{fig:FOXA1_fig5}c, \Cref{fig:FOXA1_figs11}a-f).
We then seeded these cells at equal density.
Six days post-seeding, we harvested the cells and observed a significant reduction in cell growth upon deleting any of the six \emph{FOXA1} plexus \glspl{cre} ($\Delta$\gls{cre}1 \textapprox 42\%, $\Delta$\gls{cre}2 \textapprox 28\%, $\Delta$\gls{cre}3 \textapprox 33\%, $\Delta$\gls{cre}4 \textapprox 27\%, $\Delta$\gls{cre}5 \textapprox 42\%, $\Delta$\gls{cre}6 \textapprox 44\% and $\Delta$ \emph{FOXA1} (exon 1 and intron 1) \textapprox 50\% reduction (Student's $t$-test, $p<0.05$, Fig 5d).
These results suggest that the six \emph{FOXA1} plexus contribute to \gls{pca} etiology, in agreement with their ability to regulate \emph{FOXA1} expression and the essentiality of this gene in \gls{pca} cell growth.

\subsection{\Glsentryshortpl{snv} mapping to \emph{FOXA1} \glsentryshortpl{cre} can alter their activity}

\Glspl{snv} can alter the transactivation potential of \glspl{cre} \cite{baileyNoncodingSomaticInherited2016,rheinbayRecurrentFunctionalRegulatory2017,zhangIntegrativeFunctionalGenomics2012,huangHighlyRecurrentTERT2013,hornTERTPromoterMutations2013,fuxmanbassHumanGeneCenteredTranscription2015,zhouEmergenceNoncodingCancer2016,feiginRecurrentNoncodingRegulatory2017,khuranaRoleNoncodingSequence2016,cowper-sal*lariBreastCancerRisk2012}.
In total, we found 10 \glspl{snv} called from 9 out of the 200 tumours that map to the six \emph{FOXA1} plexus \glspl{cre} (\Cref{fig:FOXA1_fig6}a).
To assess the impact of these noncoding \glspl{snv}, we conducted luciferase assays comparing differential reporter activity between the variant and the \gls{wt} allele of each \gls{cre} (\Cref{fig:FOXA1_fig6}b-k).
We found that the variant alleles of 6 of the 10 \glspl{snv} displayed significantly greater luciferase reporter activity when compared to the \gls{wt} alleles (Mann-Whitney U test, $p<0.05$).
Specifically, we observed the following fold-changes: chr14:37,887,005 A $>$ G (1.65-fold), chr14:37,904,343 A $>$ T (1.35-fold), chr14:37,905,854 A $>$ G (1.28-fold), chr14:37,906,009 T $>$ C (1.71-fold), chr14:38,036,543 A $>$ G (1.44-fold), chr14:38,055,269 C $>$ G (1.39-fold; \Cref{fig:FOXA1_fig6}b, d-h).
These results indicate that these \glspl{snv} can alter the transactivation potential of \emph{FOXA1} plexus \glspl{cre} in \gls{pca} cells.

\newfigure{Fig6.png}{A subset of noncoding \glspl{snv} mapping to the \emph{FOXA1} \glspl{cre} are gain-of-function}{\textbf{a.} Matrix showcasing the patients from the \gls{cpcgene} dataset that harbour \glspl{snv} at the \emph{FOXA1} \glspl{cre}, exons, introns, and the 3$^\prime$ \gls{utr} of \emph{FOXA1}. \textbf{b-k.} Luciferase assays are conducted in LNCaP cells. Bar plot showcases the mean firefly luciferase activity normalized by \emph{Renilla} luciferase activity in \glspl{rlu}. Error bars indicate $\pm$ s.d. $n = 5$ independent experiments for all \glspl{cre} except for chr14:38,127,842 T $>$ C where $n = 3$. Each diamond represents an independent experiment. Hypothesis testing done with Mann-Whitney U test. \textbf{l-q.} Allele-specific ChIP-qPCR conducted on plasmids carrying the \gls{wt} or variant sequence upon transient transfection in \gls{pca} cells. Data is presented as $\log_2$ fold-change of variant sequence upon comparison to \gls{wt} sequence ($n = 3$ independent experiments per ChIP). Hypothesis testing done with Student's $t$-test, n.s. not significant, * $p < 0.05$, ** $p < 0.01$, *** $p < 0.001$. Coordinates are 1-based in the GRCh37 reference genome.}{fig:FOXA1_fig6}

\subsection{\Glsentryshortpl{snv} mapping to \emph{FOXA1} \glsentryshortpl{cre} can modulate the binding of \glsentryshortpl{tf}}

We next assessed if the changes in transactivation potential induced by noncoding \glspl{snv} related to changes in \gls{tf} binding to \glspl{cre} by allele-specific ChIP-qPCR \cite{baileyNoncodingSomaticInherited2016,zhangIntegrativeFunctionalGenomics2012,cowper-sal*lariBreastCancerRisk2012} in LNCaP \gls{pca} cells.
We observed differential binding of FOXA1, AR, HOXB13, GATA2, and FOXP1 for the chr14:37887005 (A $>$ G) \gls{snv} found in \gls{cre}1; the chr14:37904343 (A $>$ T), chr14:37905854 (A $>$ G) and chr14:37906009 (T $>$ C) \glspl{snv} found in \gls{cre}2; and the chr14:38055269 (C $>$ G) \gls{snv} found in \gls{cre}4 (Student's $t$-test, $p<0.05$, \Cref{fig:FOXA1_fig6}l-p).
In contrast, SOX9 and NKX3.1 binding was unaffected by these \glspl{snv} (\Cref{fig:FOXA1_fig6}l-q).
Compared to the \gls{wt} sequence, chr14:37,887,005 A $>$ G significantly increased AR binding (1.31-fold increase), GATA2 binding (1.25-fold increase) and FOXP1 binding (1.23-fold increase); chr14:37,904,343 A $>$ T significant increased AR binding (1.30-fold increase), GATA2 (1.25-fold increase) and FOXP1 (1.33-fold increase); chr14:37,905,854 A $>$ G significantly increased FOXA1 binding (1.41-fold increase) and AR binding (1.33-fold increase); chr14:37,906,009 T $>$ C significantly increased the binding of FOXA1 (1.29-fold increase), AR (1.31-fold increase), HOXB13 (1.13-fold increase) and FOXP1 (1.25-fold increase); and chr14:38,055,269 C $>$ G significantly increased FOXA1 binding (1.20-fold increase).
Notably all six \glspl{snv} increased the binding of the \glspl{tf} known to bind at these \glspl{cre}.
In contrast, none of the \glspl{snv} significantly decreased the binding of these \glspl{tf}.
Our observations suggest that gain-of-function populate the \emph{FOXA1} plexus \glspl{cre}.
