\section{Results}

\subsection{\emph{FOXA1} is essential for prostate cancer proliferation}

We interrogated FOXA1 expression levels across cancer types.
We find that FOXA1 mRNA is consistently the most abundant in prostate tumours compared to 25 other cancer types across patients (\Cref{fig:FOXA1_fig1}a), ranking in the 95th percentile for 492 of 497 prostate tumours profiled in TCGA (\Cref{fig:FOXA1_figs1}a).
Using the same dataset we also find that FOXA1 is the most highly expressed out of 41 other Forkhead Box (FOX) factors in prostate tumours (\Cref{fig:FOXA1_figs1}b).
We next analyzed expression data from DEPMAP and observed FOXA1 to be most highly expressed in prostate cancer cell lines compared to cell lines of other cancer types (\Cref{fig:FOXA1_figs2}a).
Amongst the eight prostate cancer cell lines in the dataset (22Rv1, DU145, LNCaP, MDA-PCa-2B, NCI-H660, PrECLH, PC3, and VCaP), FOXA1 mRNA abundance is above the 90th percentile in all but one cell line (PrECLH) compared to the  $>$  56,000 protein coding and non-protein coding genes profiled (\Cref{fig:FOXA1_figs2}b).
These new results gained from the TCGA and DEPMAP validate previous understanding that FOXA1 is one of the highest expressed genes in prostate cancer \cite{tsourlakisFOXA1ExpressionStrong2017}.

\newfigure{Fig1.png}{FOXA1 is highly expressed in prostate cancer and essential for prostate cancer cell proliferation.}{\textbf{a.} The mRNA expression of FOXA1 across tumour types ($n = 26$) from RNA-seq data of TCGA. \textbf{b.} FOXA1 essentiality mediated through RNAi across various cell lines ($n=707$) from DEPMAP. Gene essentiality scores are normalized Z-scores. Higher scores indicate less essential, and lower scores indicate more essential for cell proliferation. X-axis indicate tissue of origin for each cell line tested. Each dot indicates one cell line. \textbf{c.} Gene essentiality mediated through RNAi across prostate cancer cell lines ($n=8$) from DEPMAP. Each dot indicates one gene, red indicates FOXA1. \textbf{d.} Representative Western blot against FOXA1 in LNCaP cells 5 days post-transfection of non-targeting siRNA and two independent siRNA targeting FOXA1. \textbf{e.} Cell proliferation assay conducted in LNCaP cells upon siRNA-mediated knockdown of FOXA1 across 5 days. \textbf{f.} Cell proliferation assay conducted in VCaP cells upon siRNA-mediated knockdown of FOXA1 across 5 days. Error bars indicate $\pm$ s.d. $n=3$ independent experiments. Mann-Whitney U test, * $p < 0.05$, ** $p < 0.01$.}{fig:FOXA1_fig1}

Following up on FOXA1 mRNA expression levels, we interrogated the essentiality of FOXA1 for prostate cancer cell growth.
RNAi-mediated essentiality screens compiled in DEPMAP show that FOXA1 lies in the 94th percentile across 6 of the 8 available prostate cancer cell lines: 22Rv1, LNCaP, MDA PCa 2B, NCI-H660, PC3, and VCaP cells (\Cref{fig:FOXA1_fig1}b-c).
The median RNAi-mediated essentiality score for all prostate cell lines is significantly lower than all other cell lines, suggesting that FOXA1 is especially essential for prostate cancer cell proliferation (permutation test, $p = 1 \cdot 10^{-6}$, see Methods) (\Cref{fig:FOXA1_figs3}a).
Growth assays in LNCaP and VCaP cells following FOXA1 knockdown using two independent siRNAs (\Cref{fig:FOXA1_fig1}d, \Cref{fig:FOXA1_figs3}b) show significant growth inhibition in LNCaP (siRNA \#1: 4-fold, siRNA \#2: 3.35-fold) and VCaP (siRNA \#1: 8.7-fold, siRNA \#2: 2-fold) cells five days post-transfection (Mann-Whitney U Test, $p<0.05$; \Cref{fig:FOXA1_fig1}e-f).
In accordance with previous reports, our results using essentiality datasets followed by knockdown validation reveals that FOXA1 is oncogenic and essential for prostate cancer cell proliferation.

\subsection{Identifying putative \emph{FOXA1} CREs}

The interweaving of distal CREs with target gene promoters establishes regulatory plexuses with some to be ascribed to specific genes \cite{sallariConvergenceDispersedRegulatory2016,baileyNoncodingSomaticInherited2016}.
Regulatory plexuses stem from chromatin interactions orchestrated by various factors including ZNF143, YY1, CTCF and the cohesin complex \cite{phillipsCTCFMasterWeaver2009,weintraubYY1StructuralRegulator2017,baileyZNF143ProvidesSequence2015}.
Motivated by the oncogenic role of FOXA1 in prostate cancer, we investigated its regulatory plexus controlling its expression.
According to chromatin contact frequency maps generated from Hi-C assays performed in LNCaP prostate cancer cells, FOXA1 lies in a 440 kbp TAD (chr14: 37720002-38160000 $\pm$ 40 kbp adjusting for resolution) (\Cref{fig:FOXA1_fig2}a).
By overlaying DNase-seq data from LNCaP prostate cancer cells, there are a total of 123 putative CREs reported as DNase I Hypersensitive Sites (DHS) that populate this TAD (\Cref{fig:FOXA1_fig2}a).
We next inferred the regulatory plexus of FOXA1 using the Cross Cell-Type Correlation based on DNA accessibility (C3D) method \cite{mehdiC3DToolPredict2019}.
C3D aggregates and draws correlation of DHS signal intensities between the cell line of choice and the DHS signal across all systems in the database \cite{mehdiC3DToolPredict2019}.
Anchoring our analysis to the FOXA1 promoter and using accessible chromatin regions defined in LNCaP prostate cancer cells identified 55 putative CREs to the FOXA1 regulatory plexus ($r > 0.7$) (\Cref{fig:FOXA1_fig2}b).

\newfigure{Fig2.png}{Epigenetic annotation of 14q21.1 locus and identification of \emph{FOXA1} CREs.}{\textbf{a.} Overview of cis-regulatory landscape surrounding FOXA1 on the 14q21.1 locus. H3K27ac signal track is the ChIP-seq signal overlay of 19 primary prostate tumours. LNCaP Hi-C depicts the TAD structure around FOXA1. Mutations indicate SNVs identified in 200 primary prostate tumours. \textbf{b.} Functional annotation of putative FOXA1 CREs using transcription factor and histone modification ChIP-seq conducted in primary tumours and prostate cancer cell lines. Annotated in the matrix are all DHS within the TAD and $\pm$ 40 kbp resolution left and right of the TAD. Putative FOXA1 CREs targeted by noncoding SNVs for downstream validation are boxed.}{fig:FOXA1_fig2}

\subsection{Putative \emph{FOXA1} CREs harbour TF binding sites and SNVs}

To delineate the CREs that could be actively involved in the transcriptional regulation of FOXA1, we annotated the DHS with available ChIP-seq data for histone modifications and TFs conducted in LNCaP, 22Rv1, VCaP prostate cancer cell lines and primary prostate tumours (\Cref{fig:FOXA1_fig2}b) \cite{pomerantzAndrogenReceptorCistrome2015,kronTMPRSS2ERGFusion2017}.
Close to 60\% (33/55) of the putative FOXA1 plexus CREs are positively marked by H3K27ac profiled in primary prostate tumours \cite{kronTMPRSS2ERGFusion2017}, indicative of active CREs in tumours (\Cref{fig:FOXA1_fig2}b) \cite{creyghtonHistoneH3K27acSeparates2010}.
Next, considering that noncoding SNVs can target a set of CREs that converge on the same target gene in cancer \cite{baileyNoncodingSomaticInherited2016}, we overlapped the somatic SNVs called from the whole-genome sequencing across 200 primary prostate tumours to the 33 H3K27ac-marked DHS predicted to regulate FOXA1 \cite{fraserGenomicHallmarksLocalized2017,espirituEvolutionaryLandscapeLocalized2018}.
This analysis identified 6 out of the 33 DHS marked with H3K27ac (18.2\%) harboring one or more SNV(s) (10 total SNVs called from 9 tumours) (\Cref{fig:FOXA1_fig2}b).
We observe that these 6 CREs can be bound by multiple TFs in prostate cancer cells, including FOXA1, AR and HOXB13 (\Cref{fig:FOXA1_fig2}b, \Cref{fig:FOXA1_figs4}).
The Hi-C data from the LNCaP prostate cancer cells corroborates the C3D predictions as demonstrated by the elevated contact frequency between the region harboring the FOXA1 promoter and where the 6 CREs are located, compared to other loci in the same TAD (\Cref{fig:FOXA1_fig3}a).
The 6 CREs lie in intergenic or intronic regions (\Cref{fig:FOXA1_fig3}b-h).
Together, histone modifications, TF binding sites and noncoding SNVs support that these 6 putative CREs are active in primary prostate cancer.
The Hi-C and C3D predictions suggest that they regulate FOXA1 expression.

\newfigure{Fig3.png}{Putative CREs predicted to interact with \emph{FOXA1} promoter}{\textbf{a.} Hi-C conducted in LNCaP cells indicating physical interactions between putative \emph{FOXA1} CREs and the \emph{FOXA1} promoter. Hi-C resolution is 40 kbp. \textbf{b.} The six putative \emph{FOXA1} CREs are coloured in yellow. \textbf{c-h.} Zoom-in of each individual putative \emph{FOXA1} CRE. C3D value is the Pearson correlation of DHS signal between LNCaP and the DHS reference matrix.}{fig:FOXA1_fig3}

\subsection{Disruption of CREs reduces \emph{FOXA1} mRNA expression}

We next assessed the role of CREs toward FOXA1 expression using LNCaP and 22Rv1 clones stably expressing the wild-type Cas9 protein (\Cref{fig:FOXA1_fig4}a-b).
Guide RNAs (gRNAs) designed against the FOXA1 gene (exon 1 and intron 1) served as positive controls while an outside-TAD region (i.e termed Chr14 (-)), a region on a different chromosome (the human AAVS1 safe-harbor site at the PPP1R12C locus \cite{kronTMPRSS2ERGFusion2017,dekelverFunctionalGenomicsProteomics2010}), and three regions within the TAD predicted to be excluded from the FOXA1 plexus served as negative controls.
Individual deletion of the FOXA1 plexus CREs through transient transfection of gRNAs into the LNCaP cells (See Methods) led to significantly decreased FOXA1 mRNA expression ($\Delta$ CRE1 \\textapprox 29.3 $\pm$ 8.3\%, $\Delta$ CRE2 \\textapprox 40.1 $\pm$ 11.8\%, $\Delta$ CRE 3 \\textapprox 30.6 $\pm$ 9.1\%, $\Delta$ CRE4 \\textapprox 23.6 $\pm$ 8.2\%, $\Delta$ CRE5 \\textapprox 25.3 $\pm$ 6.6\%, $\Delta$ CRE6 \\textapprox 24.5 $\pm$ 10.2\% and $\Delta$ FOXA1 (exon 1 and intron 1) \\textapprox 87.4 $\pm$ 8.8\% reduction relative to basal levels) (\Cref{fig:FOXA1_fig4}c, \Cref{fig:FOXA1_figs5}a-f).
In contrast, deletion of several negative control regions within the same TAD did not significantly reduce FOXA1 mRNA level (\Cref{fig:FOXA1_fig4}c, \Cref{fig:FOXA1_figs5}g-i).
Similar results were observed in 22Rv1 prostate cancer cells (\Cref{fig:FOXA1_fig4}d).
As each clone expressed Cas9 protein at different levels, there may be a difference between genome editing efficiencies between the clones.
We compared the CRISPR/Cas9 on-target genome editing efficiency across the five LNCaP cell line-derived clones with the relative FOXA1 mRNA levels, and indeed observe a significant inverse correlation across all CREs (Pearson's correlation $r = 0.49, p < 0.005$) (\Cref{fig:FOXA1_figs6}a) and agreeing trends for each individual CRE (\Cref{fig:FOXA1_figs6}b).

\newlongfigure{Fig4.png}{Functional dissection of putative \emph{FOXA1} CREs}{\textbf{a.} Representative western blot probed against Cas9 in LNCaP clones ($n = 5$ clones) derived to stably express Cas9 protein upon blasticidin selection. \textbf{b.} Representative western blot probed against Cas9 in 22Rv1 clones ($n = 4$ clones) derived to stably express Cas9 protein upon blasticidin selection. \textbf{c.} \emph{FOXA1} mRNA expression normalized to housekeeping \emph{TBP} mRNA expression upon CRISPR/Cas9-mediated deletion of each CRE using LNCaP clones ($n = 5$ independent experiments, each dot represents an independent clone). \textbf{d.} \emph{FOXA1} mRNA expression normalized to housekeeping \emph{TBP} mRNA expression upon CRISPR/Cas9-mediated deletion of each CRE using 22Rv1 clones ($n = 4$ independent experiments, each dot represents an independent clone). \textbf{e.} Representative western blot probed against Cas9 in LNCaP clones ($n = 4$ clones) derived to stably express the dCas9-KRAB fusion protein upon blasticidin selection. \textbf{f.} Representative western blot probed against Cas9 in 22Rv1 clones ($n = 4$ clones) derived to stably express dCas9-KRAB fusion protein upon blasticidin selection. \textbf{g.} \emph{FOXA1} mRNA expression normalized to housekeeping \emph{TBP} mRNA expression upon dCas9-KRAB-mediated repression of each CRE using LNCaP clones ($n = 4$ independent experiments, each dot represents an independent clone). \textbf{h.} \emph{FOXA1} mRNA expression normalized to housekeeping \emph{TBP} mRNA expression upon dCas9-KRAB-mediated repression of each CRE using 22Rv1 clones ($n = 4$ independent experiments, each dot represents an independent clone). \emph{FOXA1} mRNA expression was normalized to basal \emph{FOXA1} expression prior to statistical testing. $\Delta$ indicates CRISPR/Cas9-mediated deletion, $i$ indicates dCas9-KRAB-mediated repression. Error bars indicate $\pm$ s.d. Student's t test, n.s. not significant, * $p < 0.05$, ** $p < 0.01$, *** $p < 0.001$.}{fig:FOXA1_fig4}

Complementary to our findings using the wild-type CRISPR/Cas9 system, we next generated four LNCaP and four 22Rv1 cell line-derived dCas9-KRAB fusion protein expressing clones (\Cref{fig:FOXA1_fig4}e-f).
Transient transfection of the same gRNAs used in the wild-type Cas9 experiments, targeting the six FOXA1 plexus CREs into our dCas9-KRAB LNCaP clones significantly decreased FOXA1 expression relative to basal levels (iCRE1 \\textapprox 24.6 $\pm$ 6.2\%, iCRE2 \\textapprox 42.2 $\pm$ 10.8\%, iCRE3 \\textapprox 25.3 $\pm$ 9.2\%, iCRE4 \\textapprox 23.3 $\pm$ 4.3\%, iCRE5 \\textapprox 30.2 $\pm$ 3.4\% and iCRE6 \\textapprox 23.1 $\pm$ 8.1\% reduction).
Similarly, gRNAs targeting the dCas9-KRAB fusion protein to FOXA1 decreased its expression (iFOXA1 \\textapprox 81.6 $\pm$ 11.8\% reduction; Student's t-test, $p<0.05$, \Cref{fig:FOXA1_fig4}g).
Analogous results were also observed in our four clonal 22Rv1 dCas9-KRAB cell lines (Student's t-test, $p<0.05$, \Cref{fig:FOXA1_fig4}h).
Collectively, our results suggest that the six CREs control FOXA1 expression.

We further assessed the regulatory activity of the six FOXA1 plexus CREs by testing the consequent mRNA expression on other genes within the same TAD, namely MIPOL1 and TTC6.
$\Delta$ CRE1 and $\Delta$ CRE2 significantly reduced MIPOL1 mRNA expression by \\textapprox 38.4 $\pm$ 6.4\% and \\textapprox 48.4 $\pm$ 9\%, respectively relative to basal levels, whereas deletion of the other four CREs did not result in any significant MIPOL1 expression changes (Student's t-test, $p<0.05$, \Cref{fig:FOXA1_figs7}a).
On the other hand, deletion of CREs each significantly reduced TTC6 mRNA expression relative to its basal levels ($\Delta$ CRE1 \\textapprox 52.9\% $\pm$ 6.4\%, $\Delta$ CRE2 \\textapprox 66 $\pm$ 11.3\%, $\Delta$ CRE3 \\textapprox 55.5 $\pm$ 12.8\%, $\Delta$ CRE4 44.9 $\pm$ 10.6\%, $\Delta$ CRE5 43.1 $\pm$ 11.9\% and $\Delta$ CRE6 52.2 $\pm$ 7.3\% reduction (Student's t-test, p<0.05, \Cref{fig:FOXA1_figs7}b), in agreement with the fact that TTC6 shares its promoter with FOXA1 as both genes are transcribed on opposing strands (\Cref{fig:FOXA1_figs7}c).

Reduction in FOXA1 mRNA expression resulting from the deletion of FOXA1 plexus CREs may also impact gene expression downstream of FOXA1, we assessed the mRNA expression of several FOXA1 target genes, namely SNAI2, ACPP, and GRIN3A.
Deletion of CREs resulted in significant change in SNAI2 (up-regulation; $\Delta$ CRE1 \\textapprox 190\%, $\Delta$ CRE2 \\textapprox 162.8\%, $\Delta$ CRE3 \\textapprox 147.5\%, $\Delta$ CRE4 \\textapprox 133.3\%, $\Delta$ CRE5 \\textapprox 137.3\%, $\Delta$ CRE6 \\textapprox 120.8\%, $\Delta$ FOXA1 \\textapprox 266.7\%), ACPP (down-regulation; $\Delta$ CRE1 \\textapprox 73.5\%, $\Delta$ CRE2 \\textapprox 62.5\%, $\Delta$ CRE3 \\textapprox 69.6\%, $\Delta$ CRE4 \\textapprox 75.6\%, $\Delta$ CRE5 \\textapprox 70.9\%, $\Delta$ CRE6 \\textapprox 74.6\%, $\Delta$ FOXA1 \\textapprox 52.2\%) and GRIN3A expression (up-regulation; $\Delta$ CRE1 \\textapprox 138.2\%, $\Delta$ CRE2 \\textapprox 168.8\%, $\Delta$ CRE3 \\textapprox 144.6\%, $\Delta$ CRE4 \\textapprox 132.1\%, $\Delta$ CRE5 \\textapprox 131.4\%, $\Delta$ CRE6 \\textapprox 127\%, $\Delta$ FOXA1 \\textapprox 228\%) (Student's t-test, $p < 0.05$, \Cref{fig:FOXA1_figs7}d-f).
Collectively, our results support the restriction of most FOXA1 plexus CREs towards FOXA1 and its target genes.

\subsection{\emph{FOXA1} CREs collaborate to regulate its expression}

Expanding on the idea that multiple CREs can converge to regulate the expression of a single target gene \cite{sallariConvergenceDispersedRegulatory2016,baileyNoncodingSomaticInherited2016,pennacchioEnhancersFiveEssential2013}, we asked whether the CREs we identified collaboratively regulate FOXA1 mRNA expression.
Here, we applied a transient approach that delivers Cas9 protein:gRNA as a ribonucleoprotein (RNP) complex formed prior to transfection that would avoid the heterogeneity of Cas9 protein expression across the prostate cancer cell clones (See Methods).
We first validated this system through single CRE deletions, where we transiently transfected a set of gRNA targeting the CRE of interest.
In accordance with data from our prostate cancer cell clones stably expressing wild-type Cas9 and dCas9-KRAB, individual CRE deletion resulted in a significant reduction in FOXA1 mRNA expression: ($\Delta$ CRE1 \\textapprox 29.3 $\pm$ 7.3\%, $\Delta$ CRE2 \\textapprox 36 $\pm$ 11.8\%, $\Delta$ CRE3 \\textapprox 30.6 $\pm$ 12.7\%, $\Delta$ CRE4 \\textapprox 24.5 $\pm$ 6.1\%, $\Delta$ CRE5 \\textapprox 23.7 $\pm$ 13.2\%, $\Delta$ CRE6 \\textapprox 26.8 $\pm$ 14.2\% and $\Delta$ FOXA1 \\textapprox 96.2 $\pm$ 1.4\% reduction (Student's t-test, $p < 0.05$, \Cref{fig:FOXA1_fig5}a, \Cref{fig:FOXA1_figs8}a-f).
Next for combinatorial deletions, we prioritized the CREs that harbor more than 1 SNV (i.e CRE1, CRE2, CRE4), and transiently transfected RNP complexes that target both CREs in various combinations (i.e CRE1 + CRE2, CRE1 + CRE4, CRE2 + CRE4), and assessed FOXA1 mRNA expression.
Compared to negative control regions, the combinatorial deletion of $\Delta$ CRE1 + $\Delta$ CRE2, $\Delta$ CRE1 + $\Delta$ CRE4, and $\Delta$ CRE2 + $\Delta$ CRE4 resulted in a significant \\textapprox 48.5 $\pm$ 4.5\%, \\textapprox 50.4 $\pm$ 2.9\% and \\textapprox 45.2 $\pm$ 5.5\% reduction in FOXA1 mRNA expression, respectively (Student's t-test, $p<0.05$, \Cref{fig:FOXA1_fig5}b, \Cref{fig:FOXA1_figs9}a-f) a fold reduction greater than single CRE deletions (Student's t-test, \Cref{fig:FOXA1_figs10}, $p<0.05$).
These results together demonstrate that these CREs collaboratively contribute to the establishment and regulation of FOXA1 expression in prostate cancer.

\newfigure{Fig5.png}{\emph{FOXA1} CREs collaborate to regulate its expression and are critical for prostate cancer cell proliferation.}{\textbf{a.} \emph{FOXA1} mRNA expression normalized to housekeeping \emph{TBP} mRNA expression upon transient transfection-based CRISPR/Cas9-mediated deletion of CRE1, CRE2, CRE4, and sequential deletion combinations ($n = 5$ independent experiments). \textbf{b.} \emph{FOXA1} mRNA expression normalized to housekeeping \emph{TBP} mRNA expression upon bulk lentiviral-based CRISPR/Cas9-mediated deletion of each CRE in LNCaP cells ($n = 3$ independent experiments). \textbf{c.} Cell proliferation assay conducted after puromycin and blasticidin selection for LNCaP cells carrying deleted regions of interest. Data was based on cell counting 6 days after seeding post-selection ($n = 3$, representative of three independent experiments). \emph{FOXA1} mRNA expression upon deletion was normalized to basal \emph{FOXA1} expression prior to statistical testing. \emph{FOXA1} mRNA expression was normalized to the basal LNCaP \emph{FOXA1} expression prior to statistical testing. $\Delta$ indicates CRISPR/Cas9-mediated deletion. Error bars indicate $\pm$ s.d. Student's t-test, n.s. not significant, * $p < 0.05$, ** $p < 0.01$, *** $p < 0.001$.}{fig:FOXA1_fig5}

\subsection{Disruption of \emph{FOXA1} CREs reduces prostate cancer cell growth}

As FOXA1 is essential for prostate cancer growth (\Cref{fig:FOXA1_fig1}b-e), we next sought to assess the importance of the six FOXA1 plexus CREs towards prostate cancer cell growth.
We adapted a lentiviral-based approach that expressed both the Cas9 protein and two gRNA that target each CRE for deletion (See Methods).
Upon lentiviral transduction with subsequent selection, we separated LNCaP prostate cancer cells for RNA, DNA and for cell proliferation.
We first tested the system by measuring FOXA1 mRNA expression, and independently observed significant reductions of FOXA1 mRNA expression ($\Delta$ CRE1 \\textapprox 18\%, $\Delta$ CRE2 \\textapprox 30\%, $\Delta$ CRE3 \\textapprox 15\%, $\Delta$ CRE4 \\textapprox 12\%, $\Delta$ CRE5 \\textapprox 35\%, $\Delta$ CRE6 \\textapprox 46\% and $\Delta$ FOXA1 (exon 1 and intron 1) \\textapprox 48\% reduction (Student's t-test, $p<0.05$, \Cref{fig:FOXA1_fig5}c, \Cref{fig:FOXA1_figs11}a-f).
We then seeded these cells at equal density.
Six days post-seeding, we harvested the cells and observed a significant reduction in cell growth upon deleting any of the six FOXA1 plexus CREs ($\Delta$ CRE1 \\textapprox 42\%, $\Delta$ CRE2 \\textapprox 28\%, $\Delta$ CRE3 \\textapprox 33\%, $\Delta$ CRE4 \\textapprox 27\%, $\Delta$ CRE5 \\textapprox 42\%, $\Delta$ CRE6 \\textapprox 44\% and $\Delta$ FOXA1 (exon 1 and intron 1) \\textapprox 50\% reduction (Student's t-test, $p<0.05$, Fig 5d).
These results suggest that the six FOXA1 plexus contribute to prostate cancer etiology, in agreement with their ability to regulate FOXA1 expression and the essentiality of this gene in prostate cancer cell growth.

\subsection{SNVs mapping to \emph{FOXA1} CREs can alter their activity}

SNVs can alter the transactivation potential of CREs \cite{baileyNoncodingSomaticInherited2016,rheinbayRecurrentFunctionalRegulatory2017,zhangIntegrativeFunctionalGenomics2012,huangHighlyRecurrentTERT2013,hornTERTPromoterMutations2013,fuxmanbassHumanGeneCenteredTranscription2015,zhouEmergenceNoncodingCancer2016,feiginRecurrentNoncodingRegulatory2017,khuranaRoleNoncodingSequence2016,cowper-sal*lariBreastCancerRisk2012}.
In total, we found 10 SNVs called from 9 out of the 200 tumours that map to the six FOXA1 plexus CREs (\Cref{fig:FOXA1_fig6}a).
To assess the impact of these noncoding SNVs, we conducted luciferase assays comparing differential reporter activity between the variant and the wild-type allele of each CRE (\Cref{fig:FOXA1_fig6}b-k).
We found that the variant alleles of 6 of the 10 SNVs displayed significantly greater luciferase reporter activity when compared to the wild-type alleles (Mann-Whitney U test, $p<0.05$).
Specifically, we observed the following fold-changes: chr14:37,887,005 A $>$ G (1.65-fold), chr14:37,904,343 A $>$ T (1.35-fold), chr14:37,905,854 A $>$ G (1.28-fold), chr14:37,906,009 T $>$ C (1.71-fold), chr14:38,036,543 A $>$ G (1.44-fold), chr14:38,055,269 C $>$ G (1.39-fold) (\Cref{fig:FOXA1_fig6}b, d-h).
These results indicate that these SNVs can alter the transactivation potential of FOXA1 plexus CREs in prostate cancer cells.

\newfigure{Fig6.png}{A subset of noncoding SNVs mapping to the \emph{FOXA1} CREs are gain-of-function}{\textbf{a.} Matrix showcasing the patients from the CPC-GENE dataset that harbour SNVs at the \emph{FOXA1} CREs, exons, introns, and the 3' UTR of \emph{FOXA1}. \textbf{b-k.} Luciferase assays are conducted in LNCaP cells. Bar plot showcases the mean firefly luciferase activity normalized by \emph{renilla} luciferase activity. Error bars indicate $\pm$ s.d. $n = 5$ independent experiments for all CREs except for chr14:38,127,842 T $>$ C where $n = 3$. Each diamond represents an independent experiment. Hypothesis testing done with Mann-Whitney U test. \textbf{l-q.} Allele-specific ChIP-qPCR conducted on plasmids carrying the WT or variant sequence upon transient transfection in prostate cancer cells. Data is presented as $\log_2$ fold-change of variant sequence upon comparison to WT sequence ($n = 3$ independent experiments per ChIP). Hypothesis testing don with Student's t-test, n.s. not significant, * $p < 0.05$, ** $p < 0.01$, *** $p < 0.001$.}{fig:FOXA1_fig6}

\subsection{SNVs mapping to \emph{FOXA1} CREs can modulate the binding of TFs}

We next assessed if the changes in transactivation potential induced by noncoding SNVs related to changes in TF binding to CREs by allele-specific ChIP-qPCR \cite{baileyNoncodingSomaticInherited2016,zhangIntegrativeFunctionalGenomics2012,cowper-sal*lariBreastCancerRisk2012} in LNCaP prostate cancer cells.
We observed differential binding of FOXA1, AR, HOXB13, GATA2 and FOXP1 for the chr14:37887005 (A $>$ G) SNV found in CRE1; the chr14:37904343 (A $>$ T), chr14:37905854 (A $>$ G) and chr14:37906009 (T $>$ C) SNVs found in CRE2; and the chr14:38055269 (C $>$ G) SNV found in CRE4 (Student's t-test, $p<0.05$, \Cref{fig:FOXA1_fig6}l-p).
In contrast, SOX9 and NKX3.1 binding was unaffected by these SNVs (Figure  3.6l-q).
Compared to the wild-type sequence, chr14:37,887,005 A $>$ G significantly increased AR binding (1.31-fold increase), GATA2 binding (1.25-fold increase) and FOXP1 binding (1.23-fold increase); chr14:37,904,343 A $>$ T significant increased AR binding (1.30-fold increase), GATA2 (1.25-fold increase) and FOXP1 (1.33-fold increase); chr14:37,905,854 A $>$ G significantly increased FOXA1 binding (1.41-fold increase) and AR binding (1.33-fold increase); chr14:37,906,009 T $>$ C significantly increased the binding of FOXA1 (1.29-fold increase), AR (1.31-fold increase), HOXB13 (1.13-fold increase) and FOXP1 (1.25-fold increase); and chr14:38,055,269 C $>$ G significantly increased FOXA1 binding (1.20-fold increase).
Notably all six SNVs increased the binding of the TFs known to bind at these CREs.
In contrast, none of the SNVs significantly decreased the binding of these TFs.
Our observations suggest that gain-of-function SNVs populate the FOXA1 plexus CREs.
