\subsection{Discussion}

Genetic alterations that subvert the higher-order chromatin organization to allow for aberrant focal interactions may be more common in cancer than previously recognized.
In this work we demonstrated that \gls{cre} hijacking by \glspl{sv} is often associated with opposing gene expression changes at \gls{sv} antipodes, whereby genes on one flank of the breakpoint are up-regulated while genes on the other flank are repressed.
Complex \glspl{sv}, such as chromoplexy and chromothripsis, are found in numerous cancer types \cite{bacaPunctuatedEvolutionProstate2013,liPatternsSomaticStructural2020}, providing many opportunities for widespread effects on gene expression and \gls{cre} hijacking.
This is in addition to many known cancer drivers that alter \gls{cre} interactions, including the \gls{ar} and FOXA1 enhancer amplifications in primary and metastatic prostate tumours \cite{paroliaDistinctStructuralClasses2019,quigleyGenomicHallmarksStructural2018,takedaSomaticallyAcquiredEnhancer2018,zhouNoncodingMutationsTarget2020,kronTMPRSS2ERGFusion2017,viswanathanStructuralAlterationsDriving2018}.
More recent findings also fit this model, such as accumulation of extra-chromosomal circular DNA activating oncogenes that would otherwise be constrained by chromatin topology \cite{wuCircularEcDNAPromotes2019,kumarATACseqIdentifiesThousands2020,mortonFunctionalEnhancersShape2019,shoshaniChromothripsisDrivesEvolution2021}.
These insights stress the importance of investigating all ends of an \gls{sv} to assess the biological impact of these mutations on the \emph{cis}-regulatory landscape as a whole, as opposed to focusing on \glspl{cre} or \gls{sv} breakpoints as single entities.

Changes to the three-dimensional genome reported in disease onset or development are often inferred from alterations in \gls{tad} boundaries \cite{oudelaarRelationshipGenomeStructure2020,akdemirDisruptionChromatinFolding2020}.
For instance, CTCF activity is targeted by somatic mutations that enrich at its binding sites in colorectal, esophageal, and liver cancers \cite{katainenCTCFCohesinbindingSites2015,guoMutationHotspotsCTCF2018}.
Furthermore, gains in \gls{dname} at CTCF binding sites are linked to altered \gls{tad} structures in gliomas \cite{flavahanInsulatorDysfunctionOncogene2016}.
In primary \gls{pca} 97\% of differentially methylated regions genome-wide in primary \gls{pca} are losses of \gls{dname} \cite{zhaoDNAMethylationLandscape2020,yuWholeGenomeMethylationSequencing2013}, an epigenetic process previously shown to have limited impact on CTCF chromatin binding \cite{mauranoRoleDNAMethylation2015}.
This suggests that aberrant CTCF binding at \gls{tad} boundaries is not a hallmark of prostate oncogenesis.
Our observation of stable chromatin organization supports this model.
Notably, stable \gls{tad} structures observed in these primary tissues contrast previous reports of chromatin organization in cell lines derived from prostate cells \cite{taberlayThreedimensionalDisorganizationCancer2016,rhieHighresolution3DEpigenomic2019}, highlighting the necessity of low-input protocols and primary tissues \cite{diazChromatinConformationAnalysis2018}.
Our findings further support recent reports of shared higher-order chromatin organization among phenotypically distinct cell types in model organisms \cite{rao3DMapHuman2014,dixonTopologicalDomainsMammalian2012,ing-simmonsIndependenceChromatinConformation2021,ghavi-helmHighlyRearrangedChromosomes2019,iyyankiSubtypeassociatedEpigenomicLandscape2021,akdemirDisruptionChromatinFolding2020}.
Taken together, this body of evidence suggests that large disruptions to \glspl{tad} and compartments may constrain the transformation of normal to cancer cells or the divergent subtyping within prostate tumours.
Instead, changes to focal chromatin interactions seem to reflect alterations in the genetic architecture leading to cancer development.
Investigating these focal chromatin interactions may provide insights on the relationship between \glspl{cre}, such as between enhancers and their target gene promoter \cite{gasperiniComprehensiveCatalogueValidated2020,nasserGenomewideEnhancerMaps2021} to better understand the etiology of disease.

In conclusion, by bypassing technical limitations to characterize the three-dimensional genome organization across benign and tumour prostate tissue, our work reveals the predominant stable nature of genome topology across prostate oncogenesis.
Instead, alterations to discrete chromatin interactions populate the \gls{pca} genome.
These impact the function of \glspl{cre}, such as we report for \gls{sv}-mediated \gls{cre} hijacking events.
Considering the contribution of \glspl{sv} across human cancers \cite{hanahanHallmarksCancerNext2011}, our collective work presents a framework inclusive of genetics, chromatin state, and three-dimensional genome organization to understand the genetic architecture across individual primary tumours.
