\section{Introduction}

The human genome is organized into hubs of chromatin interactions within the nucleus, setting its three-dimensional topology \cite{finnMolecularBasisBiological2019}.
Two classes of higher-order topology, \glspl{tad} and compartments, define clusters of contacts between DNA elements that are linearly distant from each other, such as \glspl{cre} and their target gene promoters \cite{dixonTopologicalDomainsMammalian2012,noraSpatialPartitioningRegulatory2012}.
Insulating these hubs to prevent ectopic interactions are \gls{tad} boundaries, maintained by CCCTC-binding Factor (CTCF) and the cohesin complex \cite{pomboThreedimensionalGenomeArchitecture2015}.
Disruption of \gls{tad} boundaries through genetic or epigenetic variants can activate oncogenes, as observed in medulloblastoma \cite{northcottEnhancerHijackingActivates2014}, acute myeloid leukemia \cite{groschelSingleOncogenicEnhancer2014}, gliomas \cite{flavahanInsulatorDysfunctionOncogene2016}, and salivary gland acinic cell carcinoma \cite{hallerEnhancerHijackingActivates2019}.
However, recent studies depleting CTCF or the cohesin complex produced little effect on gene expression despite global changes to the three-dimensional chromatin organization \cite{oudelaarRelationshipGenomeStructure2020,despangFunctionalDissectionSox92019,williamsonDevelopmentallyRegulatedShh2019}.
In contrast, CRE hijacking caused by genetic alterations can result in large changes to gene expression, despite having little impact on the higher-order chromatin organization \cite{northcottEnhancerHijackingActivates2014,zhouEmergenceNoncodingCancer2016}.
These contrasting observations raise questions about the interplay between components of the genetic architecture, namely, how genetic alterations, chromatin states, and the three-dimensional genome cooperate to misregulate genes in disease.
Understanding the roles that chromatin organization and cis-regulatory interactions play in gene regulation is crucial for understanding how their disruption can promote oncogenesis.

The roles of noncoding mutations targeting \glspl{cre} in cancer are becoming increasingly clear \cite{zhouEmergenceNoncodingCancer2016,rheinbayAnalysesNoncodingSomatic2020,liPatternsSomaticStructural2020}.
Mutations to the TERT promoter, for example, lead to its over-expression and telomere elongation in multiple cancer types \cite{vinagreFrequencyTERTPromoter2013,huangHighlyRecurrentTERT2013,sternMutationTERTPromoter2015}.
Similarly, mutations targeting \glspl{cre} of the ESR1 and FOXA1 oncogenes in breast and prostate cancers, respectively, lead to their sustained over-expression \cite{baileyNoncodingSomaticInherited2016,zhouNoncodingMutationsTarget2020,paroliaDistinctStructuralClasses2019}, which is associated with resistance to hormonal therapies \cite{jeselsohnESR1MutationsMechanism2015,robinsonFoxA1KeyMediator2012,fuFOXA1OverexpressionMediates2016,fuFOXA1UpregulationPromotes2019}.
Point mutations have the potential to alter three-dimensional chromatin organization, albeit indirectly, by modifying transcription factor or CTCF binding sites \cite{mauranoLargescaleIdentificationSequence2015,guoMutationHotspotsCTCF2018}.
\glspl{sv}, on the other hand, are large rearrangements of chromatin that can directly impact its structure \cite{dixonIntegrativeDetectionAnalysis2018,akdemirDisruptionChromatinFolding2020}.
This can establish novel CRE interactions from separate \glspl{tad} or chromosomes, as has been observed in leukemia \cite{hniszActivationProtooncogenesDisruption2016} and multiple developmental diseases \cite{lupianezDisruptionsTopologicalChromatin2015,allouNoncodingDeletionsIdentify2021}.
But how prevalent and to what extent these rearrangements affect the surrounding chromatin remains largely unstudied in primary tumours \cite{akdemirDisruptionChromatinFolding2020,liPatternsSomaticStructural2020,iyyankiSubtypeassociatedEpigenomicLandscape2021}.
Hence, to understand gene misregulation in cancer, it is critical to understand how \glspl{sv} impact three-dimensional chromatin organization and CRE interactions in primary tumours.

\glspl{sv} play an important role in prostate cancer (PCa), both for oncogenesis and progression.
An estimated 97\% of primary tumours contain \glspl{sv} \cite{liPatternsSomaticStructural2020,fraserGenomicHallmarksLocalized2017}, and translocations and duplications of \glspl{cre} for oncogenes such as AR \cite{takedaSomaticallyAcquiredEnhancer2018}, ERG \cite{rosenClinicalPotentialERG2012}, FOXA1 \cite{quigleyGenomicHallmarksStructural2018,paroliaDistinctStructuralClasses2019} and MYC \cite{paroliaDistinctStructuralClasses2019} are highly recurrent.
While coding mutations of FOXA1 are found in \\textapprox 10\% of metastatic castration-resistant PCa patients, \glspl{sv} that target FOXA1 \glspl{cre} are found in over 25\% of metastatic prostate tumours \cite{paroliaDistinctStructuralClasses2019}.
In addition to oncogenic activation, \glspl{sv} in prostate tumours disrupt and inactivate key tumour suppressor genes including PTEN, BRCA2, CDK12, and TP53 \cite{quigleyGenomicHallmarksStructural2018,abeshouseMolecularTaxonomyPrimary2015}.
Furthermore, over 90\% of prostate tumours contain complex \glspl{sv}, including chromothripsis and chromoplexy events 38, making it a prime model to study the effects of \glspl{sv}.
However, despite large-scale tumour sequencing efforts, investigating the impact of \glspl{sv} on three-dimensional prostate genome remains difficult, owing to constraints from chromatin conformation capture (i.e. Hi-C) assays.
In this work, we build on recent technological advances in Hi-C protocols to investigate the three-dimensional chromatin organization of the prostate from primary benign and tumour tissues.
Using patient-matched whole genome sequencing (WGS), RNA sequencing (RNA-seq), and chromatin immunoprecipitation (ChIP-seq) data, we show that \glspl{sv} in PCa repeatedly hijacking \glspl{cre} to disrupt the expression of multiple genes with minimal impact to higher-order three-dimensional chromatin organization.
