\section{Results}

\subsection{Three-dimensional genome organization is stable over oncogenesis}

\Gls{3c} technologies enable the measurement of three-dimensional genome organization.
These assays, however, are often limited to cell lines, animal models and liquid tumours due to the amount of input required \cite{liebermanaidenComprehensiveMappingLongRange2009}.
Here, we optimized and conducted low-input Hi-C \cite{diazChromatinConformationAnalysis2018} on 10 $\mu$m thick cryosections from 12 primary prostate tumours and 5 primary benign prostate sections (see \Cref{sec:3D_methods}, \Cref{fig:3D_fig1}a, \Cref{fig:3D_figs1}a).
The 12 tumours were selected from the \gls{cpcgene} cohort previously assessed for \gls{wgs} \cite{fraserGenomicHallmarksLocalized2017}, \gls{rnaseq} \cite{chenWidespreadFunctionalRNA2019}, and H3K27ac \gls{chipseq} \cite{kronTMPRSS2ERGFusion2017,mazrooeiCistromePartitioningReveals2019} (Supplementary Table 1).
All 12 of these PCa patients previously underwent radical prostatectomies and 6 of our 12 samples (50\%) harbour the \gls{t2e} fusion found in approximately half of the primary \gls{pca} patients \cite{fraserGenomicHallmarksLocalized2017}.
The total percent of genome altered ranges from $0.99\%-18.78\%$ (Supplementary Table 1) \cite{fraserGenomicHallmarksLocalized2017}.
The 12 tumour samples were histopathologically assessed to have $\ge$ 70\% prostate cellularity and $\ge$ 60\% for our group of 5 normal prostate samples.
Upon Hi-C sequencing, we reached an average of $9.90 \times 10^8$  read pairs per sample (range $5.84 \times 10^8 - 1.49 \times 10^9$ read pairs) with minimal sequencing duplication rates (range $10.6\% - 20.8\%$) (Supplementary Table 2).
Pre-processing resulted in an average of $6.23 \times 10^8$ (96.13\%) valid read pairs per sample (range $3.95 \times 10^8 - 9.01 \times 10^8$, or $82.42\% - 99.22\%$; Supplementary Table 2).
Hence, we produced a high depth, high quality Hi-C library on 17 primary prostate tissue slices.

\newfigure{chapter3/figure_1_noheader.png}{\Glsentrylongpl{tad} are stable over prostate oncogenesis}{\textbf{a.} The sample collection and data usage of primary prostate samples in this study. 10 $\mu$m sections from 6 tumours previously identified as \gls{t2e}+ and 6 \gls{t2e}$-$ were used for Hi-C sequencing. 5 additional 10 $\mu$m sections were collected from benign prostate specimens in the \gls{uhn} BioBank. \textbf{b-c.} A comparison of the number of \glspl{tad} detected at multiple window sizes (\textbf{b}) and boundary persistence (\textbf{c}) in each patient sample, with inset schematics. Boxplots show the first, second, and third quartiles. Whiskers extend to the median $\pm 1.5 \times$ the \gls{iqr}. \textbf{d.} \gls{ice}-normalized contact matrices around the \emph{\gls{ar}} gene in primary samples and cell lines. Hi-C data for 22Rv1 and RWPE1 cell lines obtained from \cite[REF][]{rhieHighresolution3DEpigenomic2019}. Coordinates are listed according to the GRCh38 reference genome.}{fig:3D_fig1}

To characterize the higher-order organization of the primary prostate genome, we first identified \glspl{tad}.
Across the 17 primary tissue samples, we observed an average of 2,305 \glspl{tad} with a median size of 560 kbp (TopDom \cite{shinTopDomEfficientDeterministic2016}, window size $w = 20$, $FDR < 0.05$; Supplementary Tables 3-4).
However, when considering all hierarchical levels of \gls{tad} organization, we did not observe significant differences in the number of \glspl{tad} identified across length scales (\Cref{fig:3D_fig1}b), nor in the persistence of their boundaries, the number of window sizes the boundary is identified within (\Cref{fig:3D_fig1}c).
This suggests few, if any, differences in three-dimensional genome organization at the \gls{tad} level between benign and tumour tissue.
Notably, we observed differences in organization around essential genes for \gls{pca} between primary tissue and previously profiled cell lines.
For example, chromatin around the \gls{ar} gene that was previously found enriched in the 22Rv1 compared to RWPE1 prostate cell lines \cite{rhieHighresolution3DEpigenomic2019} were not recapitulated in either benign or tumour primary samples (\Cref{fig:3D_fig1}d).
Moreover, when compared to other Hi-C datasets, the primary prostate samples clustered separately from cell lines (\Cref{fig:3D_figs1}b), despite similar enrichment of CTCF binding sites near \gls{tad} boundaries (\Cref{fig:3D_figs1}c).
These results suggest that \glspl{tad} are constrained over oncogenesis and that cell line models may not harbour disease-relevant three-dimensional genome organization.

We next investigated compartmentalization changes, the second class of higher-order three-dimensional genome organization.
Recurrent changes to segments nearly the size of chromosome arms showed differential compartmentalization in multiple tumour samples compared to benign samples, such as compartment B-to-A transitions on 19q and A-to-B transitions on chromosome Y (\Cref{fig:3D_figs2}a-c).
Only two transcripts on chromosome 19 were differentially expressed between the 8 tumours with benign-like compartmentalization and the other 4 (\Cref{fig:3D_figs2}d).
Similarly, no genes on chromosome Y were differentially expressed between the 4 tumours with benign-like compartmentalization and the remaining samples (\Cref{fig:3D_figs2}e).
Both arms on chromosome 3 show differential mean compartmentalization, but this appears to be driven by one tumour sample (CPCG0255) and one benign sample (BP1664855) for each arm and is not recurrent (\Cref{fig:3D_figs2}f).
Collectively, these results suggest that phenotypic differences between benign and tumour tissues do not stem from differences in higher-order three-dimensional genome organization alone.

\subsection{Focal chromatin interactions shift over oncogenesis}

Changes to focal chromatin interactions have been observed in the absence of higher-order chromatin changes \cite{takayamaTransitionQuiescentActivated2021,johnstoneLargeScaleTopologicalChanges2020}, and we hypothesized that this may be the case in \gls{pca}.
We detected chromatin interactions, identifying a median of 4,395 interactions per sample (range $1,286 - 6,993$; \Cref{fig:3D_figs3}a, Supplementary Table 5).
Among these detected interactions, we identified known contacts in \gls{pca} such as those between two distal \glspl{cre} on chromosome 14 and the \emph{FOXA1} promoter \cite{zhouNoncodingMutationsTarget2020} (\Cref{fig:3D_figs3}b), and \glspl{cre} upstream of \emph{MYC} on chromosome 8 that are frequently duplicated in metastatic disease \cite{quigleyGenomicHallmarksStructural2018} (Supplementary Table 5).
16,474 unique chromatin interactions were identified in at least one sample (\Cref{fig:3D_fig2}a), reaching an estimated \textapprox 80\% saturation of detection (\Cref{fig:3D_figs3}c) in the entire cohort and a median of 57 \% saturation per sample (\Cref{fig:3D_figs3}c-d).
This suggests that identification of focal interactions may be boosted when assessed in a cohort of samples, rather than single samples.
Restricting our analysis to the 8,486 interactions present in at least two samples (51.5\% of all interactions) yielded 1,405 tumour- and 273 benign-specific interactions, suggesting focal changes in three-dimensional genome organization occur over oncogenesis.
Aggregate peak analysis revealed Hi-C contact enrichment at all detected interactions in all samples (\Cref{fig:3D_fig2}b-c), demonstrating that tumors- and benign-specific interactions are not binary.
Rather, the contacts at "tumour-specific" loci are more enriched than those at "benign-specific" loci in tumour samples (\Cref{fig:3D_fig2}b).
Similarly, the contacts at "benign-specific" loci are more enriched than those at "tumour-specific" loci in benign samples (\Cref{fig:3D_fig2}c).
Together, these results suggest that more focal changes to chromatin interactions are present in prostate oncogenesis despite the stable higher-order organization.

\newfigure{chapter3/figure_2_noheader.png}{Focal chromatin interactions display subtle differences between benign and tumour tissue}{\textbf{a.} Stacked bar plots of the number of samples that chromatin interactions were identified in. \textbf{b-c.} \Gls{apa} of tumour (\textbf{b}) or benign (\textbf{c}) contacts in tumour-specific, benign-specific, and shared interactions identified in two or more samples. Regions plotted are $\pm$ 300 kbp around the centre of each identified focal interaction. Inset numbers are the mean $\log_2$(obs/exp) contact frequencies within the 100 kbp $\times$ 100 kbp black boxes.}{fig:3D_fig2}

\subsection{Cataloguing \glsentrylongpl{sv} from Hi-C data}

In prostate tumours, \glspl{sv} populate the genome to aid disease onset and progression \cite{fraserGenomicHallmarksLocalized2017,quigleyGenomicHallmarksStructural2018}.
Advances in computational methods now enable the identification of \glspl{sv} from Hi-C datasets \cite{dixonIntegrativeDetectionAnalysis2018,hoStructuralVariationSequencing2020}.
Applying Hi-C Breakfinder, an \gls{sv} caller that uses Hi-C data \cite{dixonIntegrativeDetectionAnalysis2018}, to our primary prostate tumour Hi-C dataset, yielded a total of 317 unique breakpoints with a median of 15 unique breakpoints per tumour (range $3-95$; \Cref{fig:3D_fig3}a; Supplementary Table 6).
As an example, we found evidence of the \gls{t2e} fusion spanning the 21q22.2-3 locus in 6/12 (50\%) patients (CPCG0258, CPCG0324, CPCG0331, CPCG0336, CPCG0342, and CPCG0366) (\Cref{fig:3D_fig3}b), in accordance with previous \gls{wgs} findings \cite{fraserGenomicHallmarksLocalized2017}.
Combining unique breakpoint pairs into rearrangement events yielded 7.5 total events on average per patient (range $1 - 36$, \Cref{fig:3D_figs4}a-b).
We also identified more inter-chromosomal breakpoint pairs with the Hi-C data in 11 of 12 tumours (\Cref{fig:3D_fig3}b), including a novel translocation event that encompasses the deleted region between \emph{TMPRSS2} and \emph{ERG} into chromosome 14 (\Cref{fig:3D_fig4}c).
Few loci contained \gls{sv} breakpoints recurrent between patients (\Cref{fig:3D_figs4}c) and fewer inversions were detected by Hi-C (\Cref{fig:3D_fig4}a).
These numbers are smaller than previously reported from matched \gls{wgs} data \cite{fraserGenomicHallmarksLocalized2017}; however, the median distance between breakpoints on the same chromosome was much larger at 31.6 Mbp for Hi-C-identified breakpoints, compared to 1.47 Mbp from \gls{wgs}-identified breakpoints (\Cref{fig:3D_fig3}c).
This is consistent with the inherent nature and resolution of the Hi-C method to detect larger, inter-chromosomal events \cite{dixonIntegrativeDetectionAnalysis2018}.
No \glspl{sv} were detected in the 5 primary benign prostate tissue samples from Hi-C data.
While this does not rule out the presence of small rearrangements undetectable by Hi-C limited by its resolution, the absence of large and inter-chromosomal \glspl{sv} further supports a difference in genome stability between benign and tumour tissues \cite{fraserGenomicHallmarksLocalized2017,bergerGenomicComplexityPrimary2011,bacaPunctuatedEvolutionProstate2013,mazrooeiCistromePartitioningReveals2019}.
Collectively, Hi-C defines a valid method to interrogate for the presence of \gls{sv} in tumour samples, compatible with the detection of intra- and inter-chromosomal interactions otherwise missed in \gls{wgs} analyses.

\newfigure{chapter3/figure_3_noheader.png}{\glspl{sv} are identified in primary tissue through \glsentrylong{3c}}{\textbf{a.} Bar plot of \gls{sv} breakpoint pairs identified by Hi-C and \gls{wgs} on matched samples. \textbf{b.} Contact matrices of the chr21:37-42 Mbp locus harbouring the \emph{TMPRSS2} and \emph{ERG} genes. Circles indicate increased contact between \emph{TMPRSS2} and \emph{ERG} in the \gls{t2e}+ tumours. \textbf{c.} Histogram showing the distance between breakpoints on the same chromosome detected by Hi-C (left) versus \gls{wgs} (right). The inset Venn diagram shows the number of unique or shared breakpoint calls from both methods. \textbf{d.} An example of a complex set of rearrangements spanning both arms of chromosome 3 in a single patient. Contact matrices are \gls{ice}-normalized. Coordinates are listed according to the GRCh38 reference genome.}{fig:3D_fig3}

Among \glspl{sv} detected in primary prostate tumours, we identified both simple and complex chains of breakpoints.
While simple \glspl{sv} correspond to fusion between two distal DNA sequences, complex chains are evidence of chromothripsis and chromoplexy \cite{bacaPunctuatedEvolutionProstate2013}.
These genomic aberrations affecting multiple regions of the genome are known to occur in both primary and metastatic PCa \cite{bacaPunctuatedEvolutionProstate2013,fraserGenomicHallmarksLocalized2017,liPatternsSomaticStructural2020}.
The chains can be pictured as paths connecting breakpoints in the contact matrix (\Cref{fig:3D_figs4}d).
8 of the 12 (66.7\%) tumour samples contained these chains, including one patient (CPCG0331) harbouring 11 complex events and three patients (CPCG0246, CPCG0345, and CPCG0365) each harbouring a single complex event.
We observed a median of 1 complex event per patient (range $0-11$) consisting of a median of 3 breakpoints (range $3-7$) spanning a median of 2 chromosomes per event (range $1-4$, Supplementary Table 7, \Cref{fig:3D_figs4}b).
Patient CPCG0331 had 11 complex events, including a 6-breakpoint event spanning 3 chromosomes (\Cref{fig:3D_figs4}b).
A highly rearranged chromosome 3 was also found in the same patient (\Cref{fig:3D_fig3}d).
The most common type of complex event involved 3 breakpoints and spanned 2 chromosomes, occurring 9 times across 5 of the 8 patients with complex events.
In summary, using Hi-C, we detected both simple and complex \glspl{sv} in primary prostate tumours not previously identified using \gls{wgs}-based methods.
We were able to identify known observations, such as a highly mutated region on chromosome 3 and subtype-specific differences in abundance, as well as find novel inter-chromosomal events not previously reported.

\subsection{\Glsentryshortpl{sv} alter gene expression independently of intra-\glsentryshort{tad} contacts}

Using combined \gls{wgs} called \glspl{sv} with those from Hi-C data, we next systematically examined the impact of \glspl{sv} on \gls{tad} structure.
This led us to look at the intra-\gls{tad} and inter-\gls{tad} interactions around each breakpoint.
We observed that only 18 of the 260 (6.9\%) \glspl{tad} containing \gls{sv} breakpoints were associated with decreased intra-\gls{tad} or increased inter-\gls{tad} interactions (\Cref{fig:3D_fig4}a).
12 of 18 (66.7\%) occurrences were within \gls{t2e}+ tumours.
We found no evidence that simple versus complex \glspl{sv} were a factor in determining whether a \gls{tad} was altered (Pearson's $\chi^2$ test, $\chi^2 = 0.0166$, $p = 0.897$, $df = 1$).
Similarly, the type of \gls{sv} (deletion, inversion, duplication, or translocation) was not predictive of whether the \gls{tad} would be altered (Pearson's $\chi^2$ test, $\chi^2 = 4.7756$, $p = 0.3111$, $df = 4$).
Overall, we find that \glspl{sv} are associated with higher-order topological changes in a small percentage of cases and that the presence of an \gls{sv} breakpoint is not predictive alone of an altered \gls{tad}.

\newlongfigure{chapter3/figure_4_noheader.png}{\glspl{sv} can alter \glspl{tad} or gene expression around breakpoints, but rarely alters both}{\textbf{a.} A count of the number of \gls{sv} breakpoints associated with altered \gls{tad} boundaries. \textbf{b.} Bar plot showing the number of genes differentially expressed around \gls{sv} breakpoints. \textbf{c-f.} An example of an \gls{sv} that alters intra-\gls{tad} contacts without significantly affecting the expression of the nearby genes. \textbf{c.} Contact matrix showing a translocation of the \emph{TMPRSS2}-\emph{ERG} locus into chromosome 14 in the \emph{RALGAPA1} gene. \textbf{d.} Differential contact matrix between the tumour containing this translocation and another tumour without it (two-sample unpaired $t$-test). \textbf{e.} Gene expression scatterplot and boxplot of genes within the affected \gls{tad} for each sample. Boxplots show the first, second, and third quartiles of \gls{wt} tumours. Whiskers extend to the median $\pm 1.5 \times$ the \gls{iqr}. Red dots represent the tumour with the example \gls{sv}, grey dots represent the tumours without. \textbf{f.} A schematic representation of the translocation and its affect on \glspl{tad}. \textbf{g-j.} An example of an \gls{sv} that does not alter intra-\gls{tad} contacts but does alter the expression of a nearby gene. \textbf{g.} Contact matrix showing a complex rearrangement around the \emph{RIMBP2} gene. \textbf{h.} Differential contact matrix between the tumour containing this rearrangement and another tumour without it (two-sample unpaired $t$-test). \textbf{e.} Gene expression scatterplot and boxplot of genes within the affected \gls{tad} for each sample. \textbf{j.} A schematic representation of the rearrangement. Coordinates are listed according to the GRCh38 reference genome.}{fig:3D_fig4}

Despite the evidence that \glspl{sv} rarely impact higher-order chromatin topology, we evaluated whether \glspl{sv} affected the expression of genes within the \glspl{tad} surrounding the breakpoint using patient-matched \gls{rnaseq} data \cite{chenWidespreadFunctionalRNA2019}.
We found that 23 of 260 breakpoints (8.8\%) are associated with significant changes to local gene expression (\Cref{fig:3D_fig4}b).
Notably, only one of these breakpoints with local gene expression appeared within an altered \gls{tad} (\Cref{fig:3D_fig4}b).
Complex events can have opposite effects at each breakpoint.
For example, while the \gls{t2e} fusion across all tumours leads to over-expression of \emph{ERG} and under-expression of \emph{TMPRSS2} \cite{fraserGenomicHallmarksLocalized2017,kronTMPRSS2ERGFusion2017}, the deleted locus between these two genes was inserted into chromosome 14 as part of a complex translocation event in one patient (\Cref{fig:3D_fig4}c-f).
This inserted fragment positions \emph{ERG} towards the 5$^\prime$ end of the \emph{RALGAPA1} gene and \emph{TMPRSS2} towards the 3$^\prime$ end (\Cref{fig:3D_fig4}c) resulting in a significant drop in intra-\gls{tad} contacts at the \emph{RALGAPA1} locus on chromosome 14 (two-sample unpaired $t$-test, $t = 6.38$, $p = 1.04 \times 10^{-9}$; \Cref{fig:3D_fig4}d).
Despite the significant topological change on chromosome 14, no significant changes to expression was detectable across genes within the same \gls{tad} on chromosome 14 (\Cref{fig:3D_fig4}e).
Conversely, \gls{tad} alterations are not required changes to gene expression.
As part of a complex \gls{sv} involving the \emph{RIMBP2} gene (\Cref{fig:3D_fig4}g-j), both ends of the gene contain breakpoints (\Cref{fig:3D_fig4}g).
This rearrangement is not associated with changes to intra-\gls{tad} contacts (two-sample unpaired $t$-test, $t = 0.8101$, $p = 0.4183$; \Cref{fig:3D_fig4}h).
However, RIMBP2 is over-expressed in this patient (\Cref{fig:3D_fig4}i).
More generally, only a single breakpoint was observed with both \gls{tad} contact and gene expression changes, although we did not find evidence to suggest these are dependent events (Pearson's $\chi^2$ test, $\chi^2 = 6.31 \times 10^3$, $p = 0.9367$, $df = 1$).
For \glspl{tad} where at least one gene was differentially expressed, 19 (83\%) of them had at least one gene with doubled or halved expression.
Notably, we found that inter-chromosomal translocations are associated with altering the expression of genes nearby their breakpoints compared to intra-chromosomal breakpoints (Pearson's $\chi^2$ test, $\chi^2 = 7.01$, $p = 8.11 \times 10^{-3}$, $df = 1$; \Cref{fig:3D_figs5}).
Taken together, these results suggest that while \glspl{sv} can alter contacts within \glspl{tad}, this is neither necessary nor sufficient to alter gene expression.

\subsection{\Glsentryshortpl{sv} alter focal chromatin interactions to hijack \glsentryshortpl{cre} and alter antipode gene expression}

Mutations in \gls{pca} have previously been found to converge on active \glspl{cre} \cite{mazrooeiCistromePartitioningReveals2019}.
To assess if \glspl{sv} function in a similar fashion, we investigated the convergence of \gls{sv} breakpoints in active \glspl{cre}.
We find that \gls{sv} breakpoints are enriched in the catalogue of \glspl{cre} captured by H3K27ac \gls{chipseq} from our 12 primary prostate tumours compared to the rest of the genome (one-sided permutation $z$-test, $z = 25.591$, $p = 0.0099$, $n = 100$; \Cref{fig:3D_fig5}a-b).
This is similar to the enrichment of point mutations in \glspl{cre} active in \gls{pca} \cite{mazrooeiCistromePartitioningReveals2019}, suggesting that \glspl{sv} which alter gene expression may do so by recurrently targeting \glspl{cre}.
Since individual \glspl{cre} can regulate multiple genes \cite{gasperiniGenomewideFrameworkMapping2019}, we suspected that \glspl{sv} that do alter gene expression may predominantly affect multiple genes at the same time, instead of single genes.
In agreement, when considering all \glspl{sv} associated with altered gene expression near a breakpoint we find 16 of the 22 (72.7\%) \glspl{sv} are associated with altered expression of multiple genes (\Cref{fig:3D_fig5}c-d).
Notably, 15 of these 16 \glspl{sv} (93.8\%) are associated with both over- and under-expression of genes, instead of genes all being either over-expressed or under-expressed (\Cref{fig:3D_fig5}d).
12 of these 15 (80\%) \glspl{sv} are associated with expression changes at \gls{sv} antipodes, opposite ends of a breakpoint pair (\Cref{fig:3D_figs6}).
The recurrent targeting of active \glspl{cre}, combined with the opposite gene expression changes at \gls{sv} antipodes, suggests that \glspl{sv} may repeatedly alter expression by CRE hijacking.

\newfigure{chapter3/figure_5_noheader.png}{\gls{sv} breakpoints are enriched in active \glspl{cre} and repeatedly alter the expression of multiple genes}{\textbf{a.} Schematic of permutation testing for the overlap between \gls{sv} breakpoints in all \gls{cpcgene} prostate tumours and the catalogue of active \glspl{cre} in the 12 tumour samples in this study. Peaks above the genome represent putative active \glspl{cre}, identified by H3K27ac \gls{chipseq} data. \textbf{b.} Histogram of permutation test results in grey. The vertical black and green bars refer to the expected and observed overlap of \gls{sv} breakpoints and \glspl{cre}, respectively. P-value is obtained from the permutation test, $n = 100$. \textbf{c.} Schematic of how the expression of genes within \glspl{tad} containing \gls{sv} breakpoints are compared between mutant and \gls{wt} tumours. \textbf{d.} Scatterplot of FDR values obtained from differential gene expression analysis as outlined in \textbf{c.} Red dots are differentially expressed genes ($FDR < 0.05$), grey dots are genes not differentially expressed between the mutant and \gls{wt} tumours.}{fig:3D_fig5}

The fusion of \emph{PMEPA1} and \emph{ZNF516} is an example of CRE hijacking resulting in opposite differential gene expression.
Specifically, the fusion results in the \emph{PMEPA1} promoter being hijacked to the 5$^\prime$ end of the \emph{ZNF516} gene.
This is concomitant with the over-expression of \emph{ZNF516} and under-expression of \emph{PMEPA1} (\Cref{fig:3D_fig6}a-c).
In addition to hijacking the \emph{PMEPA1} promoter to the \emph{ZNF516} gene, this fusion also coincides with gains in H3K27ac over the \emph{ZNF516} gene body and of H3K27ac histone hypo-acetylation over the 3$^\prime$ end of \emph{PMEPA1}'s gene body.
This mirrors the creation of a \gls{core} reported for the \gls{t2e} fusion, reflective of new \glspl{cre} enabling \emph{ERG} over-expression and the concomitant under-expression of \emph{TMPRSS2} (\Cref{fig:3D_figs6}) \cite{kronTMPRSS2ERGFusion2017,tomlinsRecurrentFusionTMPRSS22005,tomlinsDistinctClassesChromosomal2007}.
Putative \gls{cre} hijacking is also observed with inter-chromosomal rearrangements such as seen at the \gls{sv} connecting chromosomes 7 and 19, creating 2 fusion products (termed C2B and B2C; \Cref{fig:3D_fig6}d).
This \gls{sv} separates the 3$^\prime$ end of \emph{BRAF} from its promoter and upstream enhancers on chromosome 7 (C2B; \Cref{fig:3D_fig6}d), fusing it to the 3$^\prime$ end of \emph{CYP4F11} (\Cref{fig:3D_fig6}e).
Focal chromatin interactions between \emph{BRAF} and multiple active \glspl{cre} are only observed in the fusion on chromosome 19 (\Cref{fig:3D_fig6}e).
Using matched \gls{rnaseq} data, we observe an estimated 5 fold increase in expression for the 3$^\prime$ exons of \emph{BRAF} in the mutated tumour compared to others (fold change = 4.976, FDR = 0.0181; \Cref{fig:3D_fig6}f).
Collectively, over-expression of the oncogenes, such as \emph{ERG} and \emph{BRAF}, and suppression of the tumour suppressor \emph{PMEPA1} demonstrates recurrence of \gls{cre} hijacking mediated by \glspl{sv} and the disease-relevance of this recurrent mechanism in primary prostate tumours.
These \glspl{sv} result in changes to focal chromatin interactions and overshadow the effect on higher-order topology in primary prostate cancer.

\newfigure{chapter3/figure_6_noheader.png}{\glspl{sv} altering gene expression by rewiring focal chromatin interactions}{\textbf{a.} Contact matrix of the deletion between \emph{PMEPA1} and \emph{ZNF516}. \textbf{b.} Genome tracks of H3K27ac \gls{chipseq} signal around the \emph{ZNF516} and \emph{PMEPA1} genes with the rearrangement. Grey regions are loci brought into contact by the \gls{sv}. \textbf{c.} Gene expression of \emph{PMEPA1} and \emph{ZNF516} in all tumour samples. Boxplots represent the first, second, and third quartiles of \gls{wt} patients (grey dots). Red dots are the gene expression for the mutated patient. \textbf{d.} The contact matrix of an inter-chromosomal break between chromosome 7 and chromosome 19. Upper left contacts correspond to the C2B fusion, and lower-left contacts correspond to the B2C fusion. \textbf{e.} Contact frequencies of the \emph{BRAF} promoter on chromosome 7 (left) and the 3$^\prime$ end of \emph{BRAF} on chromosome 19 (right). \gls{sv}-associated contacts between the 3$^\prime$ end of \emph{BRAF} on chromosome 19 (right) are focally enriched at H3K27ac peaks downstream of \emph{CYF4P11}. Bar plot of \glspl{sv} categorized by how differentially expressed genes are altered. Contact matrices are \gls{ice}-normalized. Coordinates are listed according to the GRCh38 reference genome.}{fig:3D_fig6}
