\chapter{Supplementary Material for Chapter 2}

% don't include the tables themselves, just their names, labels, and descriptions
\begin{enumerate}[{Table \thechapter.}1]
  \item Prostate cancer \glspl{snv} within the \emph{FOXA1} \gls{tad}\label{tab:FOXA1_tab1}
  \item \gls{grna} for clonal and transient CRISPR/Cas9 and dCas9-KRAB experiments\label{tab:FOXA1_tab2}
  \item CRISPR/Cas9 Deletion PCR Validation Primers\label{tab:FOXA1_tab3}
  \item RT-PCR \gls{mrna} Expression Primers\label{tab:FOXA1_tab4}
  \item \gls{grna} for lentiviral-based CRISPR/Cas9 deletion proliferation assays\label{tab:FOXA1_tab5}
  \item Primers for MAMA ChIP-qPCR\label{tab:FOXA1_tab6}
\end{enumerate}

\newfigure{appendix1/figs1.png}{\emph{FOXA1} \gls{mrna} expression in prostate tumours}{\textbf{a.} The ranking of \emph{FOXA1} \gls{mrna} expression across 497 primary prostate tumours profiled in \gls{tcga}. \textbf{b.} \gls{mrna} expression of all genes coding for FOX \glspl{tf} across 497 primary prostate tumours profiled in \gls{tcga}.}{fig:FOXA1_figs1}

\newfigure{appendix1/figs2.png}{\emph{FOXA1} \gls{mrna} expression across \gls{pca} cell lines}{\textbf{a.} \emph{FOXA1} \gls{mrna} expression across all cancer cell lines from \gls{depmap}, profiled by \gls{rnaseq} (see \Cref{sec:FOXA1_methods}). UAT = Upper Aerodigestive Tract, CNS = Central Nervous System, H\&L Tissues = Hematopoietic and Lymphoid Tissues. \textbf{b.} \emph{FOXA1} \gls{mrna} expression across eight \gls{pca} cell lines from \gls{depmap}, profiled by \gls{rnaseq} (see \Cref{sec:FOXA1_methods}). Red dots indicate \emph{FOXA1}.}{fig:FOXA1_figs2}

\newfigure{appendix1/figs3.png}{Essentiality of \emph{FOXA1} across cancer cell lines of various cancer types}{\textbf{a.} Gene essentiality screen mediated through \gls{shrna}/\gls{mrna} across various cancer cell lines ($n = 707$). Higher score indicates less essential, and lower score indicates more essential for cell proliferation. Red dot indicates \emph{FOXA1}. \textbf{b.} \emph{FOXA1} \gls{mrna} expression normalized to housekeeping TBP \gls{mrna} expression upon \gls{sirna}-mediated knockdown, five days post-transfection ($n=3$ independent experiments). Error bars indicate $\pm$ s.d, Student's $t$-test, *** $p<0.001$.}{fig:FOXA1_figs3}

\newfigure{appendix1/figs4.png}{Visualization of the functional annotation of the six \emph{FOXA1} \glspl{cre}}{\textbf{a-f.} Visualization of Functional annotation of the six FOXA1 \glspl{cre} using public and in-house \gls{chipseq} datasets.}{fig:FOXA1_figs4}

\newfigure{appendix1/figs5.png}{Validation of clonal Cas-mediated deletions of \glspl{cre}}{\textbf{a-f}. Representative agarose gels from LNCaP clonal CRISPR/Cas9-mediated deletion products or \gls{wt} product from PCR amplification of intended CRE, followed by T7 Endonuclease I assay.}{fig:FOXA1_figs5}

\newfigure{appendix1/figs6.png}{Genome editing efficiency (\%) is inversely correlated with \emph{FOXA1} \gls{mrna} expression}{\textbf{a.} Pearson's correlation to investigate the relationship between genome editing efficiency mediated by CRISPR/Cas9 and \emph{FOXA1} \gls{mrna} expression in LNCaP cells. The Pearson's correlation here is across all of the \glspl{cre}. \textbf{b.} Pearson's correlation based on each individual \gls{cre}, correlation between genome editing efficiency mediated by CRISPR/Cas9 and \emph{FOXA1} \gls{mrna} expression in LNCaP cells.}{fig:FOXA1_figs6}

\newfigure{appendix1/figs7.png}{Intra-\gls{tad} genes and \emph{FOXA1} downstream genes are significantly changed upon deletion of \glspl{cre}}{\textbf{a.} \emph{MIPOL1} \gls{mrna} expression normalized to housekeeping gene \emph{TBP} upon deletion of each region of interest. \textbf{b.} \emph{TTC6} \gls{mrna} expression normalized to housekeeping gene \emph{TBP} upon deletion of each CRE. \textbf{c.} Zoom-in view of the \emph{FOXA1} and \emph{TTC6} locus. \textbf{d-f.} \gls{mrna} expression of \emph{GRIN3A}, \emph{SNAI2} and \emph{ACPP} normalized to housekeeping gene \emph{TBP} upon deletion of each region of interest. $\Delta$ indicates CRISPR/Cas9-mediated deletion ($n=5$ independent experiments, each dot represents an independent clone). Error bars indicate $\pm$ s.d. Student's $t$-test, * $p<0.05$, ** $p<0.01$, *** $p<0.001$.}{fig:FOXA1_figs7}

\newfigure{appendix1/figs8.png}{Validation of transient Cas9-mediated single deletion of \glspl{cre}}{\textbf{a-f.} Agarose gel of transient transfection RNP-based Cas9-mediated deletion product from PCR amplification of intended CRE followed by T7 Endonuclease I assay.}{fig:FOXA1_figs8}

\newfigure{appendix1/figs9.png}{Validation of transient Cas9-mediated double deletion of \glspl{cre}}{\textbf{a-f.} Agarose gel of transient transfection RNP-based Cas9-mediated deletion product from PCR amplification of intended \glspl{cre} followed by T7 Endonuclease I assay.}{fig:FOXA1_figs9}

\newfigure{appendix1/figs10.png}{Comparison of \emph{FOXA1} \gls{mrna} expression upon double versus single deletion of CRE(s)}{\emph{FOXA1} \gls{mrna} expression normalized to housekeeping gene \emph{TBP} upon single or double deletion of target \glspl{cre}. $\Delta$ indicates CRISPR/Cas9-mediated deletion ($n=5$ independent experiments). Error bars indicate $\pm$ s.d., Student's $t$-test, * $p<0.05$, ** $p<0.01$, *** $p<0.001$.}{fig:FOXA1_figs10}

\newfigure{appendix1/figs11.png}{Validation of Cas9-mediated deletion of \glspl{cre} from lentiviral system expressing both Cas9 protein and \gls{grna} for cell proliferation assays}{\textbf{a-f.} Agarose gel of lentiviral-based (expression of Cas9 protein and two \gls{grna}) Cas9-mediated deletion product from PCR amplification of intended \glspl{cre} followed by T7 Endonuclease I assay.}{fig:FOXA1_figs11}
