\chapter{Supplementary Material for Chapter 3}

\begin{enumerate}[{Table \thechapter.}1]
  \item Clinical information of samples involved in this study.\label{tab:3D_tab1}
  \item Sequencing metrics as calculated by HiCUP for all Hi-C libraries generated in this study.\label{tab:3D_tab2}
  \item Summary statistics for \gls{tad} counts in all 12 tumour and 5 benign samples, across multiple window sizes.\label{tab:3D_tab3}
  \item Individual \gls{tad} calls in all 12 tumour and 5 benign samples.\label{tab:3D_tab4}
  \item Detected chromatin interactions in all 12 tumour and 5 benign samples.\label{tab:3D_tab5}
  \item \gls{sv} breakpoints detected by Hi-C in each tumour sample.\label{tab:3D_tab6}
  \item Simple and complex \glspl{sv} reconstructed from \gls{sv} breakpoints.\label{tab:3D_tab7}
  \item H3K27ac peaks identified in each of the 12 primary PCa patients.\label{tab:3D_tab8}
\end{enumerate}

\newfigure{appendix2/figure_s1_noheader.png}{Sample processing and \glsfmtshort{tad} similarity between samples}{\textbf{a.} Schematic representation of the protocol and data pre-processing pipeline used in this study to obtain Hi-C sequencing data. \textbf{b.} Heatmap of \glsfmtshort{tad} similarities between primary prostate samples, prostate cell lines, and non-prostate cell lines. Median similarity scores between \glsfmtshortpl{tad} in primary prostate tissues and cell lines is 72.1\%, 66.9\% between prostate and non-prostate cell lines, and 63.5\% between primary prostate and non-prostate lines. \textbf{c.} Local enrichment of CTCF binding sites from the 22Rv1 PCa cell line around \glsfmtshort{tad} boundaries identified in the primary samples.}{fig:3D_figs1}

\newlongfigure{appendix2/figure_s2_noheader.png}{Compartmentalization changes in tumours is not associated with widespread differential gene expression}{\textbf{a.} Bland-Altman plot of the mean compartmentalization score between tumour and benign samples. Chromosomes 3, 19, and Y are highlighted for their consistent deviation between the tissue types. \textbf{b-c.} Compartmentalization genome tracks across chromosomes 19 (\textbf{b}) and Y (\textbf{c}) in all primary samples. \textbf{d-e.} Volcano plot of differential transcript expression between the tumour samples with benign-like compartmentalization and altered compartmentalization in chromosomes 19 (\textbf{d}) and Y (\textbf{e}). Grey dots are transcripts without significant differential expression, blue dots are differentially expressed transcripts ($FDR < 0.05$) that are under-expressed in the altered compartment samples. \textbf{f.} Compartmentalization genome tracks across chromosome 3. Coordinates are listed according to the GRCh38 reference genome.}{fig:3D_figs2}

\newlongfigure{appendix2/figure_s3_noheader.png}{Characterization of chromatin interactions in benign and tumour tissue}{\textbf{a.} Bar plot of the number of significant chromatin interactions identified in each of the primary prostate samples. \textbf{b.} A snapshot of significant chromatin interactions called around the \emph{FOXA1} gene. Identified interactions are highlighted as circles. The interaction marked by the solid border contains two \glsfmtshortpl{cre} of \emph{FOXA1} identified in \cite[REF][]{zhouNoncodingMutationsTarget2020} (listed in that publication as CRE1 and CRE2). The interactions marked by the dashed border indicate regions of increased contact that may contain more distal \glsfmtshortpl{cre} of \emph{FOXA1}. \textbf{c.} Saturation analysis of chromatin interactions detected in our cohort of prostate samples versus the theoretical estimation obtained through asymptotic estimation from bootstraps. Boxplots show the first, second, and third quartiles of the identified interactions across the bootstrap iterations. The dashed black line corresponds to the asymptotic model of estimated mean unique interactions obtained from an increasing number of samples. Horizontal blue dashed lines indicate the number of observed unique interactions and theoretical maximum. Vertical green dashed lines indicate the number of samples required to reach as estimated 50\%, 90\%, 95\%, and 99\% of the theoretical maximum. \textbf{d.} Saturation analysis as in (\textbf{c}), but performed via downsampling sequencing reads from each individual sample.}{fig:3D_figs3}

\newfigure{appendix2/figure_s4_noheader.png}{Structural variant detection from Hi-C data}{\textbf{a.} Circos plots of \glsfmtshortpl{sv} identified in the 12 primary prostate tumours. \textbf{b.} Graph reconstructions of the simple and complex \glsfmtshortpl{sv} in all 12 tumours. The node colour corresponds to the chromosome of origin. \textbf{c.} Bar plot of the number of 1 Mbp bins with \glsfmtshort{sv} breakpoints from multiple patients. The previously-reported highly-mutated regions on chr3 and T2E fusion are highlighted. \textbf{d.} Correspondence between the breakpoint representation in the contact matrices and a graph representation. Each node represents a breakpoint and each edge determines whether the breakpoints were directly in contact, as identified by the Hi-C contact matrix.}{fig:3D_figs4}

\newfigure{appendix2/figure_s5_noheader.png}{Relationship between inter-chromosomal rearrangements and differential gene expression}{\textbf{a.} Bar plot of the number of differentially expressed genes and whether they are involved in \glsfmtshortpl{sv} spanning multiple chromosomes. Pearson's $\chi^2$ test, $\chi^2 = 7.0088$, $p = 8.11 \times 10^{-3}$, $df = 1$. \textbf{b.} Bar plot of all 15 \glsfmtshortpl{sv} associated with both over- and under-expression, categorized by which breakpoints the differentially expressed genes flank.}{fig:3D_figs5}

\newfigure{appendix2/figure_s6_noheader.png}{Assembly of \glsentrylongpl{sv} involving enhancer-hijacking events}{\textbf{a.} Re-assembled contact matrix from the \emph{ZNF516}-\emph{PMEPA1} fusion using NeoLoopFinder \cite{wangGenomewideDetectionEnhancerhijacking2021}. Dashed lines in the \glsfmtshort{ice}-normalized contact matrix indicate \glsfmtshort{sv} breakpoints and black arrows indicate the increased contact frequency between newly-fused segments. \textbf{b.} Re-assembled contact matrix from the C2B fusion. \textbf{c.} Re-assembled contact matrix of the \glsfmtshort{t2e} fusion. \textbf{d.} Original contact matrix of the \glsfmtshort{t2e} fusion from a single patient. \textbf{e.} Genome tracks of H3K27ac \glsfmtshort{chipseq} signal in \glsfmtshort{t2e}+ and \glsfmtshort{t2e}$-$ patients. The grey region highlights the loci that come into contact as a result of the deletion. \textbf{f.} Expression of \emph{TMPRSS2} and \emph{ERG} genes. Boxplots represent first, second, and third expression quartiles of \glsfmtshort{t2e}$-$ patients (grey dots). \glsfmtshort{t2e}+ patients are represented by red dots.}{fig:3D_figs6}
