\section{Abstract}

Differential analysis in \glsentrylong{rnaseq} assays requires controlling biological and technical sources of noise to make accurate statistical inferences.
The use of biological replicates in experiments can help reduce the impact these sources of noise, but are not always readily available, such as in patients with complex mutations or rare diseases.
These non-ideal experimental designs increase the uncertainty of differential analysis of \glsentrylong{rnaseq} data.
To combat this uncertainty, we introduce a statistical method for suboptimal experimental designs using the \glsentrylong{js} estimator for a normal distribution.
Compared to conventional methods, we find a 7-23 \% error reduction in gene expression fold change estimates.
We propose that this approach to reducing noise in differential expression analysis can be useful in single-cell analyses and studying rare diseases.
