\section{Derivation of the \glsfmtlong{js} fold change estimator}
\label{sec:JS_derivation}

For a $p$-variate normal distribution, denoted as $Z \sim \mathcal{N}_p(\mu, \Sigma)$, where the mean $\mu$ is unknown and the covariance matrix $\Sigma$ is known, the following theorem holds \cite{steinInadmissibilityUsualEstimator1956}:

\begin{theorem}
  The estimator $\hat{\mu}^{(OLS)} = Z$, for any mean $\mu$, does not minimize the \gls{mse}, $\expect{(\mu - \hat{\mu}^{(OLS)})^2}$ for a single observation of $Z$ when $p \ge 3$ and $\Sigma = I_p$, the $p \times p$ identity matrix.
  Namely, the estimator $\hat{\mu}^{(JS)} = \left( 1 - \frac{b}{a + \Vert Z \Vert ^2}\right) Z$ has a smaller \gls{mse} than $\hat{\mu}^{(0)}$ for a single observation for sufficiently small coefficient $b$ and sufficiently large coefficient $a$.
\end{theorem}

This result was generalized to non-singular covariance matrices that were not necessarily the identity matrix (Theorem 2 of \cite[REF][]{bockMinimaxEstimatorsMean1975}):

\begin{theorem}
  Let $Z \sim \mathcal{N}_p \left(\mu, \Sigma \right)$ where the mean, $\mu$, is unknown and the covariance matrix, $\Sigma$, is known.
  Let $\hat{\mu}^{(JS)} = \left( 1 - \frac{c}{Z^\tran \Sigma^{-1} Z}\right) Z$ where $c$ is some scaling coefficient.
  If $p \ge 3$, $\trace{\Sigma} \ge 2 \lambda$ (where $\trace{\cdot}$ is the trace of a matrix, $\lambda$ is the largest eigenvalue of the covariance matrix, $\Sigma$), and $0 \le c \le 2 \left( \frac{\trace{\Sigma}}{\lambda} - 2 \right)$, then $\hat{\mu}^{(JS)}$ is the estimator for the mean, $\mu$, that minimizes the \gls{mse} for a single observation of $Z$.
  \label{thm:js}
\end{theorem}

Consider the differential analysis model used in the Sleuth R package \cite{pimentelDifferentialAnalysisRNAseq2017,yiGenelevelDifferentialAnalysis2018} for a single transcript, $s$, with the simple experimental design of $n_\mathrm{WT}$ \gls{wt} samples and 1 mutant sample.
For the $n_\mathrm{WT}$ \gls{wt} samples, this can be expressed as:
%
\begin{equation}
  D_s \sim \mathcal{N}_{n_\mathrm{WT}} \left( \beta_{s,0}, (\sigma_s^2 + \tau_s^2)I_{n_\mathrm{WT}} \right)
  \label{eqn:wt_model}
\end{equation}
%
where $D_s$ is the $n_\mathrm{WT}$-dimensional count of reads compatible with transcript $s$ (i.e. the read counts mapping to transcript $s$ after pseudo-alignment); $\beta_{s,0}$ is the mean read count for transcript $s$; $\sigma_s^2$ is the biological variance of counts for transcript $s$; $\tau_s^2$ is the inferential variance for transcript $s$; and $I_{n_\mathrm{WT}}$ is the ${n_\mathrm{WT}} \times {n_\mathrm{WT}}$ identity matrix.
The model for the single mutated sample is slightly modified, with a parameter $\beta_{s,1}$ representing the fold change effect of the mutation:
%
\begin{equation}
  D_s \sim \mathcal{N} \left( \beta_{s, 0} + \beta_{s, 1}, \sigma_s^2 + \tau_s^2 \right)
  \label{eqn:single_mut_model}
\end{equation}
%
For a given transcript, $s$, the variance is the same between the mutated and \gls{wt} samples (namely, $\sigma_s^2 + \tau_s^2$).
While the mean $\beta_{s, 0}$ can be estimated from the \gls{wt} samples, the fold change $\beta_{s, 1}$ is unknown and can only be determined through the single mutated sample.
By re-parameterizing this model to consider every transcript, \Cref{eqn:wt_model} can be re-written as:
%
\begin{equation}
  \Delta^{(\mathrm{WT})} \sim \mathcal{N}_{|S|}(\Beta_0, \Sigma) \\
  \label{eqn:delta_wt}
\end{equation}
%
and \Cref{eqn:single_mut_model} can be re-written as:
%
\begin{equation}
  \Delta^{(\mathrm{Mut})} \sim \mathcal{N}_{|S|}(\Beta_0 + \Beta_1, \Sigma) \\
  \label{eqn:delta_mut}
\end{equation}
%
where $\Delta^{(\mathrm{WT})}$ is the $|S|$-dimensional vector of read counts for all transcripts in a \gls{wt} sample; $\Delta^{(\mathrm{Mut})}$ is the same but for the mutated sample; $S$ is the set of all transcripts; $\Beta_0$ is the $|S|$-dimensional vector of mean counts for each transcript; $\Beta_1$ is the $|S|$-dimensional vector of fold change effects associated with the mutation; and $\Sigma$ is the covariance matrix.
Mathematically, we can express these parameters in terms of parameters from \Cref{eqn:single_mut_model}:
%
\begin{align*}
  \Beta_{0,s} & = \beta_{s,0} \forall s \in S \\
  \Beta_{1,s} & = \beta_{s,1} \forall s \in S \\
  \Sigma      & = \begin{bmatrix}
    \sigma_1^2 + \tau_1^2 &        & 0                             \\
                          & \ddots                                 \\
    0                     &        & \sigma_{|S|}^2 + \tau_{|S|}^2 \\
  \end{bmatrix}
\end{align*}

All quantities in $\Beta_0$, $\Beta_1$, and $\Sigma$ are initially unknown.
The statistical model presented in \cite[REF][]{pimentelDifferentialAnalysisRNAseq2017} can be used derive estimates for $\Beta_0$ and $\Sigma$ solely from the \gls{wt} samples (hereafter denoted with the $\hat{}$ symbol):
%
\begin{align*}
  \hat{\Beta}_0 & = \frac{1}{n_\mathrm{WT}} \sum_{i = 1}^{n_\mathrm{WT}} \Delta^{(i)} \\
  \hat{\Sigma}  & = \begin{bmatrix}
    \max\{\hat{\sigma}_1^2, \tilde{\sigma}_1^2\} + \hat{\tau}_1^2 &        & 0                                                                         \\
                                                                  & \ddots                                                                             \\
    0                                                             &        & \max\{\hat{\sigma}_{|S|}^2, \tilde{\sigma}_{|S|}^2\} + \hat{\tau}_{|S|}^2 \\
  \end{bmatrix}
\end{align*}
%
where $\Delta^{(i)}$ is the read counts for \gls{wt} sample $i$; $\hat{\sigma}_s^2$ is the raw estimate of the biological variance for transcript $s$; $\tilde{\sigma}_i^2$ is the shrunken estimate of the biological variance for transcript $s$ made through aggregating data across transcripts; and $\hat{\tau}_s^2$ is the estimate of the inferential variance for transcript $s$ obtained from bootstrapping (see \cite[REF][]{pimentelDifferentialAnalysisRNAseq2017} for details).
However, if we treat the estimates $\hat{\Beta}_0$ and $\hat{\Sigma}$ derived from the \gls{wt} samples as known inputs for the mutated model, then \Cref{eqn:single_mut_model} for a single mutated sample meets the criteria for the \gls{js} estimators.
This is not necessarily an accurate assumption, since these estimates may be biased, inaccurate at small samples sizes, or affected by confounding factors, like batch effects.
But with sufficiently large $n_\mathrm{WT}$, this may not be a practical concern.

A \gls{js} estimator for the unknown fold change, $\Beta_1$, can be constructed according to \Cref{thm:js}:
%
\begin{equation}
  \hat{\Beta}_1^{(JS)} = \left( 1 - \frac{c}{(\Delta^{(\mathrm{Mut})} - \hat{\Beta}_0)^\tran \hat{\Sigma}^{-1} (\Delta^{(\mathrm{Mut})} - \hat{\Beta}_0)} \right)(\Delta^{(\mathrm{Mut})} - \hat{\Beta}_0)
  \label{eqn:js_defn}
\end{equation}
%
where $\hat{\Beta}_0$ is the estimate obtained from the non-mutated samples for all transcripts $s \in S$, and $\Delta^{(\mathrm{Mut})}$ is the read counts for all transcripts in the single mutated sample.
By \Cref{eqn:delta_mut}, the following relationships can be derived to satisfy \Cref{thm:js}:
%
\begin{align}
  \trace{\hat{\Sigma}} & = \sum_{s \in S} \max\{ \hat{\sigma}_s^2, \tilde{\sigma}_s^2 \} + \hat{\tau}_s^2                                                                                                                                      \\
  \lambda              & = \max_{s \in S} \left\{ \max\{ \hat{\sigma}_s^2, \tilde{\sigma}_s^2 \} + \hat{\tau}_s^2 \right\}                                                                                                                     \\
  0                    & \le c \le 2 \left( \frac{\sum_{s \in S} \max\{ \hat{\sigma}_s^2, \tilde{\sigma}_s^2 \} + \hat{\tau}_s^2}{\max_{s \in S} \left\{ \max\{ \hat{\sigma}_s^2, \tilde{\sigma}_s^2 \} + \hat{\tau}_s^2 \right\}} - 2 \right)
  \label{eqn:js_scaling_bounds}
\end{align}

\subsection{Comparison between the \glsfmtshort{ols} and \glsfmtlong{js} estimators}

In the previous section, we derived the \gls{js} estimator, $\hat{\Beta}_1^{(JS)}$.
To demonstrate how the \gls{js} estimator, $\hat{\Beta}_1^{(JS)}$, compares to conventional approaches for estimating gene expression fold change that do not make use of \Cref{thm:js}, we can consider the \gls{ols} estimator.
For the experimental design described above, the \gls{ols} estimator for $\Beta_1$, $\hat{\Beta}_1^{(OLS)}$, is given by:
%
\begin{equation}
  \hat{\Beta}_1^{(OLS)} = \Delta^{(\mathrm{Mut})} - \hat{\Beta}_0
  \label{eqn:ols_defn}
\end{equation}
%
Substituting this into \Cref{eqn:js_defn} yields a simplified form of the \gls{js} estimator for $\Beta_1$:
%
\begin{equation}
  \hat{\Beta}_1^{(JS)} = \left( 1 - \frac{c}{\left( \hat{\Beta}_1^{(OLS)} \right)^\tran \hat{\Sigma}^{-1} \hat{\Beta}_1^{(OLS)}} \right) \hat{\Beta}_1^{(OLS)}
  \label{eqn:js_defn_ols}
\end{equation}
%
From this definition, one can see that the \gls{js} estimate is colinear with the \gls{ols} estimate but uniformly shrunk towards 0.
We can summarize the above with the following theorem.

\begin{theorem}
  For an experiment containing $n_\mathrm{WT}$ \gls{wt} samples and a single mutated sample, an estimate for the expression fold change of $|S|$ transcripts can be given by the \gls{js} estimator \Cref{eqn:js_defn_ols} where $\hat{\Sigma}$ is the covariance matrix for all $|S|$ transcripts estimated from all $n_\mathrm{WT}$ \gls{wt} samples; $\hat{\Beta}_1^{(OLS)}$ is the \gls{ols} estimator for expression fold change given by \Cref{eqn:ols_defn}; and $c$ is some scaling coefficient.
  When $|S| \ge 3$, $\trace{\hat{\Sigma}} \ge 2\lambda$ (where $\lambda$ is the largest eigenvalue of $\hat{\Sigma}$), and $c$ satisfies \Cref{eqn:js_scaling_bounds}, then the \gls{mse} of the \gls{js} estimator, $\hat{\Beta}_1^{(JS)}$, is smaller than the \gls{mse} of the \gls{ols} estimator, $\hat{\Beta}_1^{(OLS)}$.
\end{theorem}

It can be shown that the \gls{js} estimator is biased towards 0 with a smaller variance than the \gls{ols} estimator (see \Cref{sec:JS_moments}).
In theory, this may increase the error of some transcripts, but will decrease the \gls{mse} for a set of transcripts in aggregate (see \Cref{sec:JS_moments}).

There are two parameters of this model that will affect the amount of biasing: the scaling coefficient, $c$, and the transcripts being considered for differential expression, $S$.
Firstly, the scaling coefficient can be manually specified, and the largest biasing occurs when $c$ is its maximum value, $2 \left( \frac{\trace{\Sigma}}{\lambda} - 2 \right)$.
Secondly, the transcripts under consideration can also be manually specified, which will affect the value of the denominator, $\left( \hat{\Beta}_1^{(OLS)} \right)^\tran \hat{\Sigma}^{-1} \hat{\Beta}_1^{(OLS)}$, and thus the amount of biasing.
The more transcripts under consideration, the larger the denominator, and so the smaller the effect.
Taken together, we have produced a high-bias, low-variance fold change estimator that has a lower \gls{mse} than the \gls{ols} estimator and two tunable parameters.
