\section{Introduction}

In statistical modelling, discrepancies between model predictions and observations can be categorized into either the variance or bias of the model, known as the "bias-variance tradeoff".
When evaluating statistical models of gene expression data against high throughput \gls{rnaseq} data, many methods have prioritized reducing variance to control errors in gene expression fold changes \cite{robinsonEdgeRBioconductorPackage2010,loveModeratedEstimationFold2014,hansenRemovingTechnicalVariability2012,trapnellDifferentialAnalysisGene2013,liRSEMAccurateTranscript2011,hardcastleBaySeqEmpiricalBayesian2010,ritchieLimmaPowersDifferential2015,lawVoomPrecisionWeights2014,lengEBSeqEmpiricalBayes2013,liModelingAnalysisRNAseq2018,rissoNormalizationRNAseqData2014,bullardEvaluationStatisticalMethods2010,pimentelDifferentialAnalysisRNAseq2017,yiGenelevelDifferentialAnalysis2018}.
Using empirical Bayesian \cite{hardcastleBaySeqEmpiricalBayesian2010,lengEBSeqEmpiricalBayes2013} models, quantile normalization \cite{hansenRemovingTechnicalVariability2012}, and \glspl{glm} with negative binomial distributions have greatly reduced the false discovery rates in \gls{rnaseq} analyses.
Moreover, controlling for confounding variables such as library size \cite{bullardEvaluationStatisticalMethods2010}, batch effects \cite{leekSvaPackageRemoving2012}, and gene length \cite{bullardEvaluationStatisticalMethods2010,loveModeratedEstimationFold2014,robinsonEdgeRBioconductorPackage2010,oshlackTranscriptLengthBias2009} have also reduced errors in quantifying gene expression differences.
One of the simplest ways to reduce the impact of variance on differential analysis is to increase the number of biological replicates in each experimental condition and to  balance the size of experimental groups.
Under ideal conditions, it has been shown that \gls{rnaseq} experiments should contain 6 - 12 biological replicates of each condition \cite{schurchHowManyBiological2016}.
However, this is rarely possible due to cost limitations of \gls{rnaseq} assays and sample availability.
For example, the Orphanet Database of rare diseases estimates that \textapprox 4 500 diseases have a prevalence $< 10^{-6}$ \cite{nguengangwakapEstimatingCumulativePoint2020}, many of which are genetic in nature.
This means individuals identified with the disease will likely not have biological replicates available for differential analysis.
Similarly, in \Cref{chap:3D}, nearly all \glspl{sv} were not found in more than a single patient.
Estimating the impact of a given mutation only found in a single individual is difficult, and recapitulating complex \glspl{sv} often found in primary prostate tumours in cell line models remains challenging \cite{nakamuraCRISPRTechnologiesPrecise2021,pickar-oliverNextGenerationCRISPR2019,wangEngineering3DGenome2021}.

To address this problem of unreplicated biological samples in differential gene expression analyses, we attempt to reduce the \gls{mse} of \gls{rnaseq} models by addressing the bias (\Cref{fig:JS_fig1}a).
Using the \gls{js} estimator for the mean of normal distribution based on a single observation \cite{steinInadmissibilityUsualEstimator1956,bockMinimaxEstimatorsMean1975,steinEstimationMeanMultivariate1981}, we reduce the \gls{mse} of fold change estimates by simultaneously increasing the bias and decreasing the variance of the statistical model (\Cref{fig:JS_fig1}b).
This method works by combining variation across dimensions for a multivariate observation and uses this combined information to reduce the estimate error (\Cref{fig:JS_fig1}c).
First, we use the Sleuth model \cite{pimentelDifferentialAnalysisRNAseq2017,yiGenelevelDifferentialAnalysis2018} for differential analysis to derive the \gls{js} estimator fold gene expression fold change and relate it to the \gls{ols} estimator.
Then, using simulations from a highly replicated \gls{rnaseq} experiment \cite{gierlinskiStatisticalModelsRNAseq2015}, we compare the differences in statistical inferences between the \gls{js} and \gls{ols} estimators.
Finally, we investigate how the number of transcripts under consideration affects the reduction in \gls{mse}, suggesting how this method can be best used in practice.

\newfigure{chapter4/Figure1.png}{Reducing the bias-variance tradeoff by combining information across multiple features}{\textbf{a.} Schematic of the bias-variance tradeoff for assessing model performance. Dartboard on the left shows low bias of the darts (mean is close to the bullseye) but a large variance. Dartboard on the right shows a high bias of the darts (mean is off-centre), but a small variance. \textbf{b.} For a $p$-variate normal distribution from which a single observation is made, the naive estimator has a higher \gls{mse} than the \gls{js} estimator, defined as $\hat{\mu}^{(2)}$. \textbf{c.} A schematic showing how the \gls{js} estimators work in theory. Estimating the mean of a random variable with 3 dimensions ($\mu$) from a single person will give an estimate that is likely far from the truth ($\hat{\mu}^{(1)}$). Combining information from the three variables together can produce an estimate that is closer to the truth ($\hat{\mu}^{(2)}$).}{fig:JS_fig1}
